---
title: "Lab Notebook 2024"
author: "Callum Malcolm"
output:
  pdf_document: default
toc: true
toc_float: true
urlcolor: blue
---

```{r setup, include=FALSE, fig.retina = NULL, dpi = 300}
knitr::opts_chunk$set(echo = TRUE)
library(ggplate)
library(tidyverse)
library(readxl)
library(ggforce)
library(scales)
library(prism)
library(knitr)
library(kableExtra)
library(dplyr)
library(xtable)


knitr::opts_chunk$set(echo = TRUE)
library(ggplate)
library(tidyverse)
library(readxl)
library(ggforce)
library(scales)
library(prism)
library(knitr)
library(kableExtra)
library(dplyr)
library(xtable)
library(formatdown)

# Function to format numbers in scientific notation
format_scientific <- function(x) {
  if (is.numeric(x)) {
    formatted <- formatC(x, format = "e", digits = 2)
    parts <- strsplit(formatted, "e")[[1]]
    coef <- parts[1]
    exponent <- as.integer(parts[2])
    paste0("$", coef, " \\times 10^{", exponent, "}$")
  } else {
    x
  }
}

# Function to apply scientific notation formatting to specified columns
apply_scientific_notation <- function(df, cols) {
  df[cols] <- lapply(df[cols], function(col) {
    sapply(col, format_scientific)
  })
  return(df)
}

```

\newpage

# LN Repository

## Rack 1
- Location: Tank 2, Rack 1, Row H (Bottom)

```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/LN_Repository.xlsx", sheet = "Sheet1")

# Convert the last column (assuming it's the 4th column here) to character
#excel_data[, ncol(excel_data)] <- as.character(excel_data[, ncol(excel_data)])

# Display the data as a table
knitr::kable(excel_data)

```

# \textbf January 2024

# Tuesday 16-01-2024

## WILDseq - Mouse Experiment 1: PCR 1 Samples 1-8 / PCR 2 Sample 5
### Overview: 
- PCR tests to see if PCR would work for WS-ME1 cDNA previously made  
- Started with samples 1-8 so as not to waste sample/reagents if PCR 1 did not work

#### Sample List
| Sample ID | Treatment/Sample | ng/uL | i7 index | i5 index |
| :-----: | :---: | :----:|:-----:|:-----:| :-----:|
| 1 | Cyclophosphamide | 773.7 | N701 | S502 |
| 2 | Cyclophosphamide  | 545.3 | N702 | S502 |
| 3 | Cyclophosphamide | 903.6  | N703 | S502 |
| 4 | Cyclophosphamide | 1056.1 |N704 | S502 |
| 5 | Cyclophosphamide | 959.1 |N705 | S502 |
| 6 | Combination | 730.1 | N706 | S502 |
| 7 | Combination | 602.3 | N707 | S502 |
| 8 | Combination | 449.6  | N710 | S502 |
| 9 | Combination| 1002.1 | N701 | S503 |
| 10 | Combination | 1929.2 | N702 | S503 |
| 11| Methotrexate | 861.5 | N703 | S503 |
| 12 | Methotrexate | 1110.4  | N704 | S503 |
| 13 | Methotrexate | 1171.0 | N705 | S503 |
| 14 | Methotrexate  | 1347.8 | N706 | S503 |
| 15 | Methotrexate  | 891.4  | N707 | S503 |
| 16 | Vehicle | 374.5 | N710 | S503 |
| 17 | Vehicle | 911.2 | N701 | S505 |
| 18 | Vehicle | 829.8 | N702 | S505 |
| 19 | Vehicle | 600.1  | N703 | S505 |
| 20 | Vehicle | 750.3 | N704 | S505 |
| 21 | Baseline | 401.5 | N705 | S505 |
| 22 | Baseline | 443.3 | N706 | S505 |
| 23 | Baseline  | 373.4  | N707 | S505 |
| 24 | Baseline | 444.5  | N710 | S505 |
| 25 | Baseline | 267.6 | N701 | S506 |
| 26 | BLLW 14K | 378.7  | N702 | S506 |
| 27 | BLLW 2K | 362.7  | N703 | S506 |
| 28 | BLLW 1K | 563.8  | N704 | S506|
| 29 | Methotrexate (outlier) | 348.1 | N705 | S506|
| 30 | Combo | 380.8 | N706 | S506 |
| 31 | RBL2P 2K | 173.3 | N707 | S506 |
| 32 | RBL2P 7K | 2708.0 | N7010 | S506 |
| 33 | RBL2P 250K | 1418.5 | N701 | S507 |
| 34 |Mock_direct_1 | - | N702 | S507 |
| 35 | Mock_direct_2 | -  | N703 | S507 |
| 36 | Mock_direct_3 | - | N704 | S507 |
| 37 | Mock_culture_1 | - | N705 | S507 |
| 38 | Mock_culture_2 | - | N706 | S507 |
| 39 | Mock_culture_3 | - | N707 | S507 |

### PCR 1: Samples 1-8
- Checked [cDNA] on NanoDrop of sample 8 to make sure samples were still intact
  * Sample 8 ng/$\mu$L = 1042.3
  * It is assumed all other cDNA is of similar quality

1. Made a master mix of PCR1 reagents 
- Made enough for 31 samples
- Primer mix was made earlier

|Component       |Volume| Master Mix |
|:--------------:|:----:|:-------:|
| 10uM WS PCR1 Primer Mix | 3 $\mu$L | 93 $\mu$L |
| DNAse/RNAse H20      | 12 $\mu$L | 372 $\mu$L |
| Kapa Hifi HotStart Ready Mix (2X) | 25 $\mu$L | 775 $\mu$L |

2. Add following components to tubes

|Component       |Volume|
|:--------------:|:----:|
| PCR1 MasterMix | 40 $\mu$L | 
| cDNA      | 10 $\mu$L |

3. Performed PCR using the following parameters: 

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 60C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 1 min |
Hold Step |Step 6: 12C    | Hold |  

\center __10-25 cycles of steps 2-4__

\raggedright
### Bead Clean Up
#### Overview
- Clean PCR reaction by removing leftover primers
- This prevents false reads, cleaner indexing in later steps, and higher quality samples

#### Materials
- For 40 samples

|Component       |Expected Volume/experiment| Material ID |
|:--------------:|:----:|:---------:|
| Ampure Bead Kit | 200 $\mu$L | |

#### Protocol
1.	Equilibrate Ampure beads to room temp (available from supplies cold room)
2.	Mix well and vortex for 30sec to ensure uniform distribution.
3.	Add 90ul (1.8X volume) to PCR reaction and mix thoroughly by pipetting.
4.	Incubate 5min, room temp
5.	Place on magnet for 2 min and keep on magnet until final elution
6.	Remove supernatant and add 200ul 70% ethanol (make fresh)
7.	Incubate 30sec and remove
8.	Repeat 70% ethanol wash
9.	Air dry beads 5-10min room temp. Once dry the beads will go from shiny to matt in appearance. Avoid over drying (cracked appearance).
10.	Add 15 ul EB buffer and resuspend beads by pipetting
11.	Incubate 2min room temp
12.	Place on magnet and remove eluate
13. Assess PCR product size, contamination and concentration on tapestation.   
**Expected size = 165-172bp**

#### Tapestation PCR 1 - Samples 1-8

![2024-01-16 Tapestation PCR 1 Results Samples 1-8](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Previous/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-16 TS 1.png)
\center
Initial Test PCR 1
\raggedright

### PCR 2: Samples 5

#### Overview
- Yield from PCR1 is low (.09 - .631 ng/$\mu$L)
- Sample with lowest concentration was taken ahead to PCR 2 (E1 - Sample 5)
- PCR to index samples for sequencing by attaching UMI 

#### Materials

|Component       |Expected Volume/experiment| Material ID |
|:--------------|----|-----------------|:-------------:|
| 10uM Nxxx Nextera i7 adapter  | - |   -       |
| 10uM Sxxx Nextera i5 adapter | - |     -     |
| DNAse/RNAse H20 | 1000 $\mu$L |          |
| Kapa Hifi HotStart Ready Mix (2X) | 1200 $\mu$L |  KK2601  |


#### Protocol  
1. Prepared samples according to the following table:
  - Sample - Adaptor pairs in table above

|Component       |Volume|
|:--------------|----|
| 10uM N705 Nextera i7 adapter  | 1.5 $\mu$L |
|10uM S502 Nextera i5 adapter | 1.5 $\mu$L |
| DNAse/RNAse H20      | 21 $\mu$L |
| Kapa Hifi HotStart Ready Mix (2X) | 25 $\mu$L |
| 10ng/$\mu$L PCR1  | 1 $\mu$L | 


2. Performed PCR using the following parameters:  

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 55C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 2 min |
Hold Step |Step 6: 12C    | Hold |  
\center
 __8 cycles of steps 2-4__ 
\raggedright

3. Performed Ampure bead clean up as above with 90ul beads per 50ul PCR reaction

4. Checked size and concentration on tapestation

#### Tapestation PCR 2
![2024-01-16 Tapestation PCR 2 Test Samples 5](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Previous/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-16 TS 2.png)


#### Review
- Contamination/PCR bubble?
- Yield is adequate (assuming it is representative of the correct peak)
- Chris wants to repeat PCR1 for remaining samples, take PCR2 ahead, and re-run sames with D1000 tape (non High-Sensitivity)

#### Plan for tomorrow
- PCR 1 samples 9-30
- Bead Clean Up
- Tape Station

# Wednesday 17-01-2024

## WILDseq - Mouse Experiment 1: PCR 1 Samples 9-30

### Overview: 
- Completed PCR 1 for Samples 9-30

### PCR 1: Samples 9-30

1. Used master mix of PCR1 reagents made 2024-01-16 

2. Add following components to tubes
|Component       |Volume|
|:--------------|----|
| PCR1 MasterMix | 40 $\mu$L | 
| cDNA      | 10 $\mu$L |

3. Performed PCR using the following parameters: 

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 60C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 1 min |
Hold Step |Step 6: 12C    | Hold |  

\center
__10-25 cycles of steps 2-4__
\raggedright

### Bead Clean Up
#### Overview
- Clean PCR reaction by removing leftover primers
- This prevents false reads, cleaner indexing in later steps, and higher quality samples

#### Materials
- For 40 samples

|Component       |Expected Volume/experiment| Material ID |
|:--------------:|:----:|:---------:|
| Ampure Bead Kit | 200 $\mu$L | |

#### Protocol
1.	Equilibrate Ampure beads to room temp (available from supplies cold room)
2.	Mix well and vortex for 30sec to ensure uniform distribution.
3.	Add 90ul (1.8X volume) to PCR reaction and mix thoroughly by pipetting.
4.	Incubate 5min, room temp
5.	Place on magnet for 2 min and keep on magnet until final elution
6.	Remove supernatant and add 200ul 70% ethanol (make fresh)
7.	Incubate 30sec and remove
8.	Repeat 70% ethanol wash
9.	Air dry beads 5-10min room temp. Once dry the beads will go from shiny to matt in appearance. Avoid over drying (cracked appearance).
10.	Add 15 ul EB buffer and resuspend beads by pipetting
11.	Incubate 2min room temp
12.	Place on magnet and remove eluate
13. Assess PCR product size, contamination and concentration on tapestation.   
**Expected size = 165-172bp**

#### Tapestation PCR 1
![2024-01-17 Tapestation PCR 1 Samples 9-16](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Previous/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-17 TS 1.png)

#### Tapestation PCR 1
![2024-01-17 Tapestation PCR 1 Samples 17-30](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Previous/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-17 TS 2.png)
\\

#### Review
- Samples 10/17/18/29 don't have adequate yield
  * Will need to redo PCR 1
  * Sample 10 has acceptable PCR1 sample from previous run
- Yield is adequate for the rest to move ahead

#### Plan for tomorrow
- PCR 1 samples 17/18/29
  * Bead Clean Up
  * Tape Station
- PCR 2 for all samples
  * Double check reagents amounts
  * Running low on KAPA (re-ordered) and Nextera Primers
- Chris wants to add 2 additional samples
  * To Do - RNA Extraction / RT / PCR 1 / PCR 2
  * Wait until majority of samples have acceptable PCR 2 product
  
# Thursday 18-01-2024

## WILDseq - Mouse Experiment 1: PCR 1 Samples 17/18/29

### Overview: 
- Repeated PCR 1 for Samples 17/18/29

### PCR 1: Samples 17/18/29

1. Used master mix of PCR1 reagents made 2024-01-16 

2. Add following components to tubes
|Component       |Volume|
|:--------------:|:----:|
| PCR1 MasterMix | 40 $\mu$L | 
| cDNA      | 10 $\mu$L |

3. Performed PCR using the following parameters: 

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 60C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 1 min |
Hold Step |Step 6: 12C    | Hold | 
\center
 __10-25 cycles of steps 2-4__ 
\raggedright

### Bead Clean Up
#### Overview
- Clean PCR reaction by removing leftover primers
- This prevents false reads, cleaner indexing in later steps, and higher quality samples

#### Materials
- For 40 samples

|Component       |Expected Volume/experiment| Material ID |
|:--------------:|:----:|:---------:|
| Ampure Bead Kit | 200 $\mu$L | |

#### Protocol
1.	Equilibrate Ampure beads to room temp (available from supplies cold room)
2.	Mix well and vortex for 30sec to ensure uniform distribution.
3.	Add 90ul (1.8X volume) to PCR reaction and mix thoroughly by pipetting.
4.	Incubate 5min, room temp
5.	Place on magnet for 2 min and keep on magnet until final elution
6.	Remove supernatant and add 200ul 70% ethanol (make fresh)
7.	Incubate 30sec and remove
8.	Repeat 70% ethanol wash
9.	Air dry beads 5-10min room temp. Once dry the beads will go from shiny to matt in appearance. Avoid over drying (cracked appearance).
10.	Add 15 ul EB buffer and resuspend beads by pipetting
11.	Incubate 2min room temp
12.	Place on magnet and remove eluate
13. Assess PCR product size, contamination and concentration on tapestation.   
**Expected size = 165-172bp**

#### Tapestation PCR 1


#### Review
- Samples 10/17/18/29 don't have adequate yield
  * Will need to redo PCR 1
  * Sample 10 has acceptable PCR1 sample from previous run
- Yield is adequate for the rest to move ahead

#### Plan for tomorrow
- PCR 1 samples 17/18/29
  * Bead Clean Up
  * Tape Station
- PCR 2 for all samples
  * Double check reagents amounts
  * Running low on KAPA (re-ordered) and Nextera Primers
- Chris wants to add 2 additional samples
  * To Do - RNA Extraction / RT / PCR 1 / PCR 2
  * Wait until majority of samples have acceptable PCR 2 product
  
# Friday 19-01-2024

## WILDseq - Mouse Experiment 1: RNA Extraction/RT Samples 31-33
Overview: Extra samples given by Chris for whitelist

### RNA extraction Samples 31-33
- Performed RNA extraction of tissue/cell samples according to the following kit:

### RT Protocol Samples 31-33
1. In PCR strip tubes, prepared 5 $\mu$g of RNA in a total volume of 10 $\mu$l of RNAse/DNAase-free water.
2. Add 1 $\mu$l of WS_RT_UMI_NexteraR2 primer (2$\mu$M stock)
3. Add 1 $\mu$l dNTPs (10 mM each). 
4. Denature at 65 C for 5 mins in the PCR machine, then straight onto ice for at least 2 mins, spin briefly to get liquid to the bottom of the tube.
5. Used Master Mix previously made
6. Add 7 $\mu$l of RT MM prepared above to each sample and mix, spin briefly to get liquid to the bottom of the tube.
   - Spin briefly to get liquid to bottom of the tube 
7. In the PCR machine incubate at 53 C for 10 mins followed by 80 C for 10 mins.
8. Add 1 $\mu$l Themolabile Exonuclease I (NEB M0568) to remove excess RT primer.

|Component       |Volume|   MM volume  |
|:--------------|----|-----------------|
| Thermolabile Exonuclease I  | 1 $\mu$L |          |
| NEBuffer r3.1*        | 2 $\mu$L |          |

 **Most PCR buffers are compatible**

9. In PCR Machine: heat at 37C for 4 mins followed by 80C for 1 min  
    - Spin briefly to get liquid to bottom of the tube
10. Add 1 $\mu$l of RNAse H and incubate at 37 C for 20 mins.
11. Checked [cDNA] on NanoDrop
  - Added to Sample List

# Monday 22-01-2024

### PCR 1: Samples 31-33
- Included negative control (MM + H20)

1. Used master mix of PCR1 reagents made 2024-01-16 

2. Add following components to tubes

|Component       |Volume|
|:--------------:|:----:|
| PCR1 MasterMix | 40 $\mu$L | 
| cDNA      | 10 $\mu$L |

3. Performed PCR using the following parameters: 

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 60C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 1 min |
Hold Step |Step 6: 12C    | Hold |  

\center
 __10-25 cycles of steps 2-4__
\raggedright

### Bead Clean Up
#### Overview
- Clean PCR reaction by removing leftover primers
- This prevents false reads, cleaner indexing in later steps, and higher quality samples

#### Materials
- For 40 samples

|Component       |Expected Volume/experiment| Material ID |
|:--------------:|:----:|:---------:|
| Ampure Bead Kit | 200 $\mu$L | |

#### Protocol
1.	Equilibrate Ampure beads to room temp (available from supplies cold room)
2.	Mix well and vortex for 30sec to ensure uniform distribution.
3.	Add 90ul (1.8X volume) to PCR reaction and mix thoroughly by pipetting.
4.	Incubate 5min, room temp
5.	Place on magnet for 2 min and keep on magnet until final elution
6.	Remove supernatant and add 200ul 70% ethanol (make fresh)
7.	Incubate 30sec and remove
8.	Repeat 70% ethanol wash
9.	Air dry beads 5-10min room temp. Once dry the beads will go from shiny to matt in appearance. Avoid over drying (cracked appearance).
10.	Add 15 ul EB buffer and resuspend beads by pipetting
11.	Incubate 2min room temp
12.	Place on magnet and remove eluate
13. Assess PCR product size, contamination and concentration on tapestation.   
**Expected size = 165-172bp**


#### Tapestation PCR 1 31-33 
#### Tapestation PCR 1
![2024-01-22 Tapestation PCR 1 Samples 30-33 and Control](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Previous/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-22 TS 1.png)
\center
**!!!! Band present in - Control lane !!!!**
\raggedright

### PCR 1: Negative Control
- Remade negative control (MM + H20)

1. Used master mix of PCR1 reagents made 2024-01-16 

2. Add following components to tubes

|Component       | Volume |
|:--------------:|:----:|
| PCR1 MasterMix | 40 $\mu$L | 
| cDNA      | 10 $\mu$L |

3. Performed PCR using the following parameters: 

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 60C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 1 min |
Hold Step |Step 6: 12C    | Hold |  

\center
 __10-25 cycles of steps 2-4__
\raggedright

#### Tapestation Control Test 1
- Compared New -Control to old 
  * Wanted to determine if this was a pipetting error
![2024-01-22 Tapestation Control Test 1](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Previous/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-22 TS 1.png)


#### Tapestation Control Test 2
- Compared New -Control to H20 
  * Wanted to determine if water was contaminated
![2024-01-22 Tapestation Control Test 2](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Previous/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-22 TS 1.png)


# Tuesday 23-01-2024
## Made New Media: AR-5
1. Made new media: AR-5

| Solution | ID code | Volume| % Total volume
| :-------: | :------: | :--------:|:----------:|
| Advanced RMPI   | ---- | 500 mL     | 78% |
| Glutamax     | ---- | 6.5 mL    | 20% |
| Pen-Strep    | ---- | 6.5 mL      | 1% |
| FBS     | ---- | 128 mL    | 1% |

## Cell Culture
### A20
- Thawed A20 and seeded in T25
- Spun down CS to remove DMSO, cell pellet small
- Cells didn't look super healthy
- Monitor tomorrow

### HEK
- Got T75 flask from Chris
- 70% confluent
- Added 10mL of trypsin and tapped flask until cells were dislodged
- Added 10mL of DMEM (20% FBS) and added to 50mL tube
- Added 25mL of DMEM to 5 T75 flasks
- Mixed CS to get single cell suspension and remvoe clumps
- Transferred 4mL of CS to each flask
- Will grow up for 1-2 days and freeze down 4 T75 flasks
  * Other flask will be used to gorw WILDseq library virus for A20's

## WS-ME1 Library Prep: Control Test 3
Overview: question around contaminated primers
- Ran PCR1 with new primer/PCR 1 mix

### PCR 1: Negative Control
- Remade negative control (MM + H20)
- Remade Primer mix and PCR1 MM

#### PCR1 Primer Mix
- WILDseq protocol from Kirsty dictates 8 fwd primers mixed in equal amounts with equivilant amount of rev priemr
- Primers were reconsituted at 100 $\mu$M so needed to be diluted to 10 $\mu$


|Component                 |   Volume  |
|:------------------------:|:---------:|
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 16 $\mu$L |
| DNAse/RNAse H20          | 320 $\mu$L| 


#### PCR1 MM

|Component       |Volume| Master Mix |
|:--------------|----| ------- |
| 10uM WS PCR1 Primer Mix | 3 $\mu$L |
| DNAse/RNAse H20      | 12 $\mu$L |
| Kapa Hifi HotStart Ready Mix (2X) | 25 $\mu$L |

1. Add following components to tubes  

|Component       |Volume|
|:--------------:|:----:|
| PCR1 MasterMix | 40 $\mu$L | 
| H2O     | 10 $\mu$L |

2. Performed PCR using the following parameters: 

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 60C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 1 min |
Hold Step |Step 6: 12C    | Hold |  

\center
 __10-25 cycles of steps 2-4__
\raggedright

#### Tapestation Control Test 3 
- Compared New -Control to old -control 
  * Wanted to determine if water was contaminated
![2024-01-22 Tapestation Control Test 2](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Previous/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-23 TS 1.png)
  
  
# Wednesday 24-01-2024

## Cell Culture
### A20
- Checked cells ~10% confluent
- Cells looked healthy but sparse
- Grow up for inection
- Ask Chris about cell numbers needed for injecting into 12 mice

### HEK
- 20% confluent
- Check tomorrow and split or Friday

## WILDseq - Mouse Experiment 1: Library Prep Attempt 5 - RT
- Decided to repeat the entire library prep process using RNA extracted previously
- Everything done in PCR room RNA hood


### RT Protocol  
1. In PCR strip tubes, prepared 5 $\mu$g of RNA in a total volume of 10 $\mu$l of RNAse/DNAase-free water.
2. Add 1 $\mu$l of WS_RT_UMI_NexteraR2 primer (2$\mu$M)
- Diluted primer from stock (100$\mu$M)
- Added 1 $\mu$L RT-Primer stock into 49$\mu$L water
3. Add 1 $\mu$l dNTPs (10 mM each). 
4. Denature at 65 C for 5 mins in the PCR machine, then straight onto ice for at least 2 mins, spin briefly to get liquid to the bottom of the tube.
5. Prepare mastermix of RT enzyme and buffers. 1.1x the number of samples.

|Component       |Volume|   MM volume  |
|:--------------|----|-----------------|
| 5x SSIV Buffer | 4 $\mu$L | 134 $\mu$L      |
| SSIV RT        | 1 $\mu$L |    33.5 $\mu$L      |
| 100 mM DTT     | 1 $\mu$L |    33.5 $\mu$L      |
| RNAse Out      | 1 $\mu$L |     33.5 $\mu$L     |

6. Add 7 $\mu$l of RT MM prepared above to each sample and mix, spin briefly to get liquid to the bottom of the tube.
   - Spin briefly to get liquid to bottom of the tube 
7. In the PCR machine incubate at 53 C for 10 mins followed by 80 C for 10 mins.
8. Add 1 $\mu$l Themolabile Exonuclease I (NEB M0568) to remove excess RT primer.

|Component       |Volume|   MM volume  |
|:--------------|----|-----------------|
| Thermolabile Exonuclease I  | 1 $\mu$L |     33.5 $\mu$L     |
| NEBuffer r3.1*        | 2 $\mu$L |    67 $\mu$L      |

\center
 **Most PCR buffers are compatible**
\raggedright

9. In PCR Machine: heat at 37C for 4 mins followed by 80C for 1 min  
    - Spin briefly to get liquid to bottom of the tube
10. Add 1 $\mu$l of RNAse H and incubate at 37 C for 20 mins.
    - Stored at -4C

#### Plan for tomorrow
- PCR 1 (check for KAPA HIFI amount)
  * Bead Clean Up
  * Tape Station
- Check on cells
- Plan cell numbers needed for A20 mouse injections

# Thursday 25-01-2024

## Cell Culture
### A20
- Checked cells ~50% confluent
- Cells looked healthy but sparse
- Added 10ml of AR-5
- Grow up for injection
- Ask Chris about cell numbers needed for injecting into 12 mice

### HEK
- Collected all T175 and froze down
- Left one flask going at 20% confluency

## WS-ME1 Library Prep - Attempt 5 PCR 1
Ran PCR1 for attempt 5 

### PCR 1: Negative Control
- Remade negative control (MM + H20)
- Remade Primer mix and PCR1 MM

#### PCR1 Primer Mix
- WILDseq protocol from Kirsty dictates 8 fwd primers mixed in equal amounts with equivalent amount of rev primer
- Primers were reconsituted at 100 $\mu$M so needed to be diluted to 10 $\mu$M


|Component                 |   Volume  |
|:------------------------:|:---------:|
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 16 $\mu$L |
| DNAse/RNAse H20          | 144 $\mu$L| 


#### PCR1 MM

|Component       |Volume| Master Mix |
|:--------------|----| ------- |
| 10uM WS PCR1 Primer Mix | 3 $\mu$L |
| DNAse/RNAse H20      | 12 $\mu$L |
| Q5 Master Mix | 25 $\mu$L |

- Used Q5 instead of KAPA

1. Add following components to tubes  

|Component       |Volume|
|:--------------:|:----:|
| PCR1 MasterMix | 40 $\mu$L | 
| H2O     | 10 $\mu$L |

2. Performed PCR using the following parameters: 

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 60C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 1 min |
Hold Step |Step 6: 12C    | Hold |  

\center
 __10-25 cycles of steps 2-4__
\raggedright

### Bead Clean Up
#### Overview
- Clean PCR reaction by removing leftover primers
- This prevents false reads, cleaner indexing in later steps, and higher quality samples

#### Materials
- For 40 samples

|Component       |Expected Volume/experiment| Material ID |
|:--------------:|:----:|:---------:|
| Ampure Bead Kit | 200 $\mu$L | |

#### Protocol
1.	Equilibrate Ampure beads to room temp (available from supplies cold room)
2.	Mix well and vortex for 30sec to ensure uniform distribution.
3.	Add 90ul (1.8X volume) to PCR reaction and mix thoroughly by pipetting.
4.	Incubate 5min, room temp
5.	Place on magnet for 2 min and keep on magnet until final elution
6.	Remove supernatant and add 200ul 70% ethanol (make fresh)
7.	Incubate 30sec and remove
8.	Repeat 70% ethanol wash
9.	Air dry beads 5-10min room temp. Once dry the beads will go from shiny to matt in appearance. Avoid over drying (cracked appearance).
10.	Add 15 ul EB buffer and resuspend beads by pipetting
11.	Incubate 2min room temp
12.	Place on magnet and remove eluate
13. Assess PCR product size, contamination and concentration on tapestation.   
**Expected size = 165-172bp**

### Tapestation PCR 1 Samples 1-33

#### Tapestation PCR 1 1-16 
##### Tapestation PCR 1
![2024-01-25 Tapestation PCR 1 Samples 1-16](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Previous/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-25 TS 1.png)

#### Tapestation PCR 1 1-32 
##### Tapestation PCR 1
![2024-01-25 Tapestation PCR 1 Samples 17-32](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Previous/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-25 TS 2.png)

- Samples 18, 19, and 20 DNW

#### Tapestation PCR 1 18-20, 33 
##### Tapestation PCR 1
![2024-01-25 Tapestation PCR 1 Samples 33 and redo 18-20](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Previous/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-25 TS 3.png)

- Samples 18, 19, and 20 DNW again


# Friday 26-01-2024

## Cell Culture
### A20
- Expanded to T175
- Grow up for injection
- Ask Chris about cell numbers needed for injecting into 12 mice

### HEK
- Split 1/2
- DMEM

## WS-ME1 Library Prep - Attempt 5 PCR 1 repeat and PCR 2 prep
- Reran PCR1 for samples 18,19,20
- Prepped PCR2 primers

### PCR2 Prep
- Reconstituted PCR2 primers bought by Chris
  * Used DNAse/RNAse Free H20 and diluted to 100 $\mu$M
- Diluted into strip tubes working stock (10$\mu$M)
  * 90 $\mu$L H20 + 10 $\mu$L primer stock
  
### PCR1 Samples 18-20

#### PCR1 Protocol

- Used Primer Mix previously made on 25-01-2024

|Component       |Volume| Master Mix |
|:--------------|----| ------- |
| 10uM WS PCR1 Primer Mix | 3 $\mu$L |
| DNAse/RNAse H20      | 12 $\mu$L |
| Q5 Master Mix | 25 $\mu$L |

- Used Q5 instead of KAPA

1. Add following components to tubes  

|Component       |Volume|
|:--------------:|:----:|
| PCR1 MasterMix | 40 $\mu$L | 
| H2O     | 10 $\mu$L |

2. Performed PCR using the following parameters: 

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 60C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 1 min |
Hold Step |Step 6: 12C    | Hold |  

\center
 __10-25 cycles of steps 2-4__
\raggedright

### Bead Clean Up
#### Overview
- Clean PCR reaction by removing leftover primers
- This prevents false reads, cleaner indexing in later steps, and higher quality samples

#### Materials
- For 40 samples

|Component       |Expected Volume/experiment| Material ID |
|:--------------:|:----:|:---------:|
| Ampure Bead Kit | 200 $\mu$L | |

#### Protocol
1.	Equilibrate Ampure beads to room temp (available from supplies cold room)
2.	Mix well and vortex for 30sec to ensure uniform distribution.
3.	Add 90ul (1.8X volume) to PCR reaction and mix thoroughly by pipetting.
4.	Incubate 5min, room temp
5.	Place on magnet for 2 min and keep on magnet until final elution
6.	Remove supernatant and add 200ul 70% ethanol (make fresh)
7.	Incubate 30sec and remove
8.	Repeat 70% ethanol wash
9.	Air dry beads 5-10min room temp. Once dry the beads will go from shiny to matt in appearance. Avoid over drying (cracked appearance).
10.	Add 15 ul EB buffer and resuspend beads by pipetting
11.	Incubate 2min room temp
12.	Place on magnet and remove eluate
13. Assess PCR product size, contamination and concentration on tapestation.   
**Expected size = 165-172bp**

### Tapestation PCR 1 Samples 18-20

#### Tapestation PCR 1 18-20
##### Tapestation PCR 1
![2024-01-26 Tapestation PCR 1 Samples 18-20](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Previous/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-26 TS 1.png)


# Monday 29-01-2024

## Cell Culture
### A20
- Split 1:6
- AR-5
- Ask Chris about cell numbers needed for injecting into 12 mice

### HEK
- Split 1/6
- DMEM

## PCR Step 2 - Samples 1-24
### Overview
- PCR to index samples for sequencing by attaching UMI 
- Didn't have enough Q5 to complete all the samples

### PCR2 Protocol  
1. Prepare samples according to the following table:
  - Keep careful note of which adaptors are added to which sample

|Component       |Volume|
|:--------------|----|
| 10uM Nxxx Nextera i7 adapter  | 1.5 $\mu$L |
|10uM Sxxx Nextera i5 adapter | 1.5 $\mu$L |
| DNAse/RNAse H20      | 21 $\mu$L |
| Q5 | 25 $\mu$L |
| 10ng/$\mu$L PCR1  | 1 $\mu$L | 


2. Perform PCR using the following parameters:  

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 55C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 2 min |
Hold Step |Step 6: 12C    | Hold |

\center
__8 cycles of steps 2-4__
\raggedright

3. Perform Ampure bead clean up as above with 90ul beads per 50ul PCR reaction.
- If you don’t have a crazy number of samples you might also want to consider doing up to 4 PCR2 reactions per sample and pooling them afterwards just to make sure all your sequences are captured.

4. Check size and concentration on tapestation. Dilute and pool samples for sequencing.

### Tapestation 
![2024-01-29 Tapestation PCR 2 Samples 1-16](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Previous/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-29 TS 1.png)  
- 1-8 PCR DNW (suspected issue with bead clean-up)

![2024-01-29 Tapestation PCR 2 Samples 17-24](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Previous/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-29 TS 2.png)  -

- Last 4 samples are from Chris (seperate experiment)

# Tuesday 30-01-2024

## Cell Culture
### HEK 
- Unhealthy
- Discarded

### A20
- Split 1/6

# \textbf February 2024

# Thursday 01-02-2024

## PCR Step 2 - Samples 1-8, 25-39  
- Repeated PCR2 for samples 1-8

### Overview
- PCR to index samples for sequencing by attaching UMI 
- Didn't have enough Q5 to complete all the samples

### PCR2 Protocol  
1. Prepare samples according to the following table:
  - Keep careful note of which adaptors are added to which sample

|Component       |Volume|
|:--------------|----|
| 10uM Nxxx Nextera i7 adapter  | 1.5 $\mu$L |
|10uM Sxxx Nextera i5 adapter | 1.5 $\mu$L |
| DNAse/RNAse H20      | 21 $\mu$L |
| Q5 | 25 $\mu$L |
| 10ng/$\mu$L PCR1  | 1 $\mu$L | 


2. Perform PCR using the following parameters:  

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 55C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 2 min |
Hold Step |Step 6: 12C    | Hold |  
\center
 __8 cycles of steps 2-4__ 
\raggedright

3. Perform Ampure bead clean up as above with 90ul beads per 50ul PCR reaction.
- If you don’t have a crazy number of samples you might also want to consider doing up to 4 PCR2 reactions per sample and pooling them afterwards just to make sure all your sequences are captured.

4. Check size and concentration on tapestation. Dilute and pool samples for sequencing.


### Tapestation
![2024-02-01 Tapestation PCR 2 Samples 1-8,25-32](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Previous/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-02-01 TS 1.png)
![2024-02-01 Tapestation PCR 2 Samples 33-39](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Previous/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-02-01 TS 2.png) 


# Monday 05-02-2024

## Cell Culture
### Made DMEM
| Solution | ID code | Volume| % Total volume
| :-------: | :------: | :--------:|:----------:|
| DMEM   | ---- | 500 mL     | 78% |
| Pen-Strep    | ---- | 5.5 mL      | 1% |
| FBS     | ---- | 55 mL    | 10% |

### HEK - Seeded
- Seeded New HEK
  * HEK stock are in CM Box 1 -80  
  
1. Thawed vial
2. Added 1mL DMEM to vial
3. Transferred to 15mL falcon tube
4. Slowly added 5mL DMEM
5. Spun down at 650rpm for 4min
6. Removed supernant and resuspended vial flicking
7. Added 1mL DMEM and pipetted multiple times to create single cell suspension
8. Added to T25 flask
9. Added 14mL of DMEM
10. Placed in incubator

### A20 - Split
- Split 1/6
- Spliut T25 vial for Emily James (EJ)

# Tuesday 06-02-2024

## WS-ME1 Pooling for Submission

- Genomics core indicated that library should be submitted at 5nM-10nM in a volume of 20-30uL
- Cubit PCR2 Samples to determine concentration

```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Previous/Wildseq - Mouse Experiment 1/In_vivo_lib_prep_data.xlsx", sheet = "Pooling")

# Display the data as a table
knitr::kable(excel_data)

```

- Used IDT Library Concentration Conversion Calculator
  * Link: https://eu.idtdna.com/Calc/library-concentration-conversion
  

# Wednesday 07-02-2024

## Cell Culture

### HEK
- Decided to wait till Monday to start making virus
- Discarded HEK
- Will reseed HEK Sunday

### A20
- Split 1/6
- Keeping going for NSG/BALBc injection
  * Issue with procedure room in AMB, waiting for that to be resolved
- Also plan to incorporate WILDseq

# Friday 09-02-2024

## Cell Culture

### A20
- Split 1/6
- Keeping going for NSG/BALBc injection
  * Issue with procedure room in AMB, waiting for that to be resolved
- Also plan to incorporate WILDseq

# Monday 12-02-2024

## Cell Culture

### Made DMEM-V1

- DMEM for virus prodution
- Does not have Pen/Strep

| Solution | ID code | Volume| % Total volume
| :-------: | :------: | :--------:|:----------:|
| DMEM   | ---- | 500 mL     | 78% |
| FBS     | ---- | 128 mL    | 20% |
| Glutamax    | ---- | 6.5 mL    | 1% |

### Thawed HEK

### A20
- Split 1/6
- Keeping going for NSG/BALBc injection
  * Planning to inject with Chris tomorrow
- Also plan to incorporate WILDseq once virus is made

## Processing scRNAseq

# Tuesday 13-02-2024

## Cell Culture

### HEK
- Morphology looked weird
- 20% FBS is probably too high
- Threw away will start next week

## Processing scRNAseq

# Wednesday 14-02-2024

## Cell Culture

### A20
- Split 1/6
- Keeping going for NSG/BALBc injection
  * Planning to inject with Chris tomorrow
- Also plan to incorporate WILDseq once virus is made

## Processing scRNAseq

# Friday 16-02-2024

## A20 Xenograft EXP 1

### Cell Prep

  1. Thawed 2.5mL of Matrigel on ice ~3hours before 
  2. Transferred 12x10^6^ cells to a 15mL eppendorf
  3. Pelleted CS
  4. Resuspended in 2.5mL of PBS
  5. Transferred 400$\mu$L CS to 6 different 1mL eppendorfs
  6. Added 400$\mu$L of Matrigel, mixed gentlely and placed back on ice
  7. Brought materials over to the AMB for injection

### Injection  

- Chris Injected 
- Study Plan: SP140164  

| Mouse ID | Earmark | Genotype | Injection Location | 
| :-------: | :------: | :--------:|:----------:|
| TUAD36.2h  | NM | NSG     | IP |
| TUAD36.2i  | 1R | NSG     | IP |
| TUAD36.2k  | 2R | NSG     | IP |
| TUAD36.2a  | NM | NSG     | SC |
| TUAD36.2b  | 1L | NSG     | SC |
| TUAD36.2c  | 1R | NSG     | SC |
| TUAP3.1e  | 1L | BALB/c   | IP |
| TUAP3.1a  | NM | BALB/c   | IP |
| TUAP4.1a  | 2L | BALB/c   | IP |
| TUAP3.1b  | 1L | BALB/c   | SC |
| TUAP4.1b  | 2R | BALB/c   | SC |
| TUAP3.1c  | 1R | BALB/c   | SC |  

- Each mouse received ~1x10^6^ cells
- Mice will be monitored over the coming weeks for tumour development 

# Monday 19-02-2024

## Cell Culture
### A20
- Split 1/6
- Used AR-5

# Wednesday 21-02-2024

## Cell Culture
### A20
- Split 1/6
- Used AR-5

### HEK
- Seeded New HEK
  * HEK stock are in CM Box 1 -80 
- Used DMEM-V1

1. Thawed vial
2. Added 1mL DMEM to vial
3. Transferred to 15mL falcon tube
4. Slowly added 5mL DMEM
5. Spun down at 650rpm for 4min
6. Removed supernant and resuspended vial flicking
7. Added 1mL DMEM and pipetted multiple times to create single cell suspension
8. Added to T25 flask
9. Added 14mL of DMEM
10. Placed in incubator

# Friday 23-02-2024

## Cell Culture
### A20
- Split 1/6  
- Used AR-5

### HEK
- Split 1/6
- Used DMEM-V1

# Monday 26-02-2024

## Cell Culture

### A20
- Split into T25
- Froze down 4 vials
- 4x10^6^ cells per vial
- Used AR-5

### HEK
- Froze down 7 vials
- 6x10^6^ cells per vial

### WILDseq Virus Production Day 1
- Seeded 2 10cm dishes of HEK cells
  * Coated with Poly-D-Lysine (covered plate, removed liquid, let sit for 10 min, washed with PBS) 
  * 4.5x10^6^ cells seeded per dish (seeded at 13:00)
  
# March 2024
  

# Wednesday 06-03-2024
- Collected SC tumours
- HEK infection
  * binned A20/HEK

# Wednesday 13-03-2024
- Made media
- Seededin N2-BC4
- Froze down A20
- Collected 3 IP NSG tumours

# Thursday 14-03-2024
## Cell Culture - N2
- Changed media (N2-BC4)

# Friday 15-03-2024

## Cell Culture

### N2-BC4 RTX CDC Assay - version 1

# Monday 18-03-2024

## Cell Culture - N2 -C4
- Split: seeded 5x10^6^ cells

## N2-BC4 RTX CDC Assay - version 2
Overview: Trying to optimise RTX CDC assay
- Using 0% serum, since there are no complement factors in the media the cell live/dead reading should not change
- Next step: add varying amounts of human serum

# Friday 22-03-2024

## Cell Culture

### N2-BC4
- Cells look healthy
- Feeders in flask are also healthly
- Split 5x10^6^ back into T175 with 30mL of AR-6

## EC50 RTX N2-BC4 22324 - Seeding
- Seeded 96-well plate with N2-BC4 and treated with RTX concentration range with or with out 10% Human Serum (HS)
  * Used Heat-Inacted Serum which does not have functional complement which is why this experiment did not work
- Plate seeding protocol:
  1. Diluted cell suspension to seed 20000 cells/well in 50$\mu$L amounts
  
| Well Number | Cells total | Volume total | CS cells/mL | Flask cell count | **Flask Volume** | **Media Volume** |
| :-------: | :------: | :--------:|:----------:|:---------:|:----:|:-------:|
| 60 wells | 1.2x10^6^ cells | 3 mL     | 4x10^5^ |3.06x10^6^ | **400$\mu$L** | **2.6 mL** |

  2. Made RTX dilutions and added to respective wells in 50$\mu$L  
    - [RTX stock] = 10.3 mg/mL 
    - Drug volumes are added in triplicate
    - Drug volumes are being added consititute 1/4 of well volume: [RTX working] needs to be 4x [RTX well]  
    - 6 wells per condition, 50$\mu$L per well ~ minimum of 300$\mu$L per condition needed (recommend 500$\mu$L)  
 
```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240325_DDT.xlsx")

# Display the data as a table
knitr::kable(excel_data)

```
  
  3. Added 10% serum or media control
  - This provides complement factors to bind BL-bound RTX and initiate CDC
  - Serum/media amounts added at 100 $\mu$L/well
  - Serum volume is added in 1:4 ratio (20$\mu$L serum in 200$\mu$L final well volume)
  - Serum stock mix is 1.2mL HS : 2.4 mL media / plate
  
  4. Plate is incubated for 72 hrs at 37C

```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Write up/Lab Notebook - Table excel files/240322_PM.xlsx", sheet = "Sheet1")
title <- "EC50_RTX_N2-BC4_250322"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 96,
  plate_type = "round",
  #colour = "#2E008B",
  title = title
)

print(P1)
```

# Monday 25-03-2024

## EC50 RTX N2-BC4 22324 - Collection
- Collected plate seeded on 22-03-2025
- EC Plate collection protocol:
  1. Added 40$\mu$L Cell Titre Blue (CTB) to each conditioned well 
    * 20$\mu$L CTB/100$\mu$L of conditioned well recommended by manufacturer
  2. Incubated for 1hr at 37C
  3. Read on plate reader according to Cell Titre Blue Protocol
- Results: DNW

## EC50 RTX N2-BC4 25324 - Seeding (DNW)
- Seeded an EC50 experiement comparing the effects of RTX on N2-BC4 with or without 10% serum 
  * Used Heat-Inacted Serum which does not have functional complement which is why this experiment did not work
- Decreased overall well volume to reduce CTB being used
- Used RTX provided by Jamie
- Plate seeding protocol:
  1. Diluted cell suspension to seed 20000 cells/well in 25$\mu$L amounts
  
| Well Number | Cells total | Volume total | CS cells/mL | Flask cell count | **Flask Volume** | **Media Volume** |
| :-------: | :------: | :--------:|:----------:|:---------:|:----:|:-------:|
| 60 wells | 1.2x10^6^ cells | 1.5 mL     | 8x10^5^ | 2.92x10^6^ | **410$\mu$L** | **1.1 mL** |

  2. Made RTX dilutions and added to respective wells in 50$\mu$L   
    - [RTX stock] = 10.3 mg/mL  
    - Drug volumes are added in triplicate  
    - Drug volumes are being added consititute 1/4 of well volume: [RTX working] needs to be 4x [RTX well]    
    - 6 wells per condition, 25$\mu$L per well ~ minimum of 150$\mu$L per condition needed (recommend 200$\mu$L)   
  
```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240325_DDT.xlsx")

# Display the data as a table
knitr::kable(excel_data)

```
  
  3. Added 10% serum or media control
  - This provides complement factors to bind BL-bound RTX and initiate CDC
  - Serum/media amounts added at 50 $\mu$L/well
  - Serum volume is added in 1:4 ratio (10$\mu$L serum in 100$\mu$L final well volume)
  - Serum stock mix is 600$\mu$L HS : 1.2 mL media / plate
  
  4. Plate is incubated for 72 hrs at 37C

```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Write up/Lab Notebook - Table excel files/240322_PM.xlsx", sheet = "Sheet1")
title <- "EC50_RTX_N2-BC4_25324"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 96,
  plate_type = "round",
  #colour = "#2E008B",
  title = title
)

print(P1)
```


## Cell Culture
### N2-BC4
- Cells look healthy
- Feeders in flask are also healthly
- Split 5x10^6^ back into T175 with 30mL of AR-6

# Wednesday 27-03-2024

## Cell Culture
- Got Ramos BC 5 split from Jamie
- Cultured in T75

# Thursday 28-03-2024

## EC50 RTX N2-BC4 25325 - Collection

- Collected plate seeded on 25-03-2025
- EC Plate collection protocol:
  1. Added 40$\mu$L Cell Titre Blue (CTB) to each conditioned well 
    * 20$\mu$L CTB/100$\mu$L of conditioned well recommended by manufacturer
  2. Incubated for 1hr at 37C
  3. Read on plate reader according to Cell Titre Blue Protocol
- Results: DNW

# Friday 29-03-2024

## EC50 RTX N2-BC4/RAMOS-BC5 25329 - Seeding (DNW)
- Seeded an EC50 experiement comparing the effects of RTX on N2-BC4 and RAMOS-BC5 with or without 10% serum 
  * Used Heat-Inacted Serum which does not have functional complement which is why this experiment did not work
- Decreased overall well volume to reduce CTB being used
- Used RTX provided by Jamie
- Plate seeding protocol:
  1. Diluted cell suspension to seed 20000 cells/well in 25$\mu$L amounts
  
|Cell Line| Well Number | Cells total | Volume total | CS cells/mL | Flask cell count | **Flask Volume** | **Media Volume** |
| :-------: | :------: | :--------:|:----------:|:---------:|:----:|:-------:|
| N2-BC4| 60 wells | 1.2x10^6^ cells | 1.5 mL     | 8x10^5^ | 2.92x10^6^ | **410$\mu$L** | **1.1 mL** |
| RAMOS-BC5| 60 wells | 1.2x10^6^ cells | 1.5 mL     | 8x10^5^ | 2.92x10^6^ | **410$\mu$L** | **1.1 mL** |

  2. Made RTX dilutions and added to respective wells in 50$\mu$L  
    - [RTX stock] = 10.3 mg/mL 
    - Drug volumes are added in triplicate
    - Drug volumes are being added consititute 1/4 of well volume: [RTX working] needs to be 4x [RTX well]  
    - 6 wells per condition, 25$\mu$L per well ~ minimum of 150$\mu$L per condition needed (recommend 200$\mu$L)  
 
```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240329_DDT.xlsx")

# Display the data as a table
knitr::kable(excel_data)

```
  
  3. Added 10% serum or media control
  - This provides complement factors to bind BL-bound RTX and initiate CDC
  - Serum/media amounts added at 50 $\mu$L/well
  - Serum volume is added in 1:4 ratio (10$\mu$L serum in 100$\mu$L final well volume)
  - Serum stock mix is 600$\mu$L HS : 1.2 mL media / plate
  
  4. Plate is incubated for 72 hrs at 37C

```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Write up/Lab Notebook - Table excel files/240322_PM.xlsx", sheet = "Sheet1")
title <- "EC50_RTX_N2-BC4_250324"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 96,
  plate_type = "round",
  #colour = "#2E008B",
  title = title
)

print(P1)
```

# Sunday 31-03-2024

## Cell Culture
### N2-BC4

- Healthy, split


#### Changed FDC:

1. Thawed vial
2. Added 1mL DMEM to vial
3. Transferred to 15mL falcon tube
4. Slowly added 5mL DMEM
5. Spun down at 650rpm for 4min
6. Removed supernant and resuspended vial flicking
7. Added 1mL DMEM and pipetted multiple times to create single cell suspension
8. Added to T175 flask
9. Added 30mL of DMEM
10. Added 5x10^5^ N2 cells from previous flask
10. Placed in incubator

### Ramos-BC5
- Healthy, split
- Count: 2.18x10^6^ cells/mL
- Split: 7x10^5^ cells in 15 mL

## Ramos CDC Testing  

- Tested CDC assay adapted from [Ge et al., 2019](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6315025/#CR32)
- CDC protocol
1. Seed 250k cells 500uL volumes per well of a 12-well plate
2. Add RTX or media in 250uL volumes per well
  * [RTX] working needs to be 4x final desired well volume
3. Add Serum or media in 250uL volumes per well
  * Serum amount should be 4x final % desired

```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Write up/Lab Notebook - Table excel files/240331_PM.xlsx", sheet = "Sheet1")
title <- "EC50_RTX_N2-BC4_250331"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 12,
  plate_type = "round",
  #colour = "#2E008B",
  title = title
)

print(P1)
```

## EC50 RTX N2-BC4/RAMOS-BC5 24331 - Seeding (DNW)
- Seeded an EC50 experiement comparing the effects of RTX on N2-BC4 and RAMOS-BC5 with or without 10% serum 
- Decreased overall well volume to reduce CTB being used
- Used RTX provided by Jamie
- Only incubated for 24hr
- Plate seeding protocol:
  1. Diluted cell suspension to seed 20000 cells/well in 25$\mu$L amounts
  
|Cell Line| Well Number | Cells total | Volume total | CS cells/mL | Flask cell count | **Flask Volume** | **Media Volume** |
| :-------: | :------: | :--------:|:----------:|:---------:|:----:|:-------:|
| N2-BC4| 60 wells | 1.2x10^6^ cells | 1.5 mL     | 8x10^5^ | 2.92x10^6^ | **410$\mu$L** | **1.1 mL** |
| RAMOS-BC5| 60 wells | 1.2x10^6^ cells | 1.5 mL     | 8x10^5^ | 2.92x10^6^ | **410$\mu$L** | **1.1 mL** |

  2. Made RTX dilutions and added to respective wells in 50$\mu$L    
    - [RTX stock] = 10.3 mg/mL  
    - Drug volumes are added in triplicate  
    - Drug volumes are being added consititute 1/4 of well volume: [RTX working] needs to be 4x [RTX well]    
    - 6 wells per condition, 25$\mu$L per well ~ minimum of 150$\mu$L per condition needed (recommend 200$\mu$L)

```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240329_DDT.xlsx", sheet = "Sheet1")

# Display the data as a table
knitr::kable(excel_data)

```

  3. Added 10% serum or media control
  - This provides complement factors to bind BL-bound RTX and initiate CDC
  - Serum/media amounts added at 50 $\mu$L/well
  - Serum volume is added in 1:4 ratio (10$\mu$L serum in 100$\mu$L final well volume)
  - Serum stock mix is 600$\mu$L HS : 1.2 mL media / plate
  
  4. Plate is incubated for 72 hrs at 37C  
  
```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Write up/Lab Notebook - Table excel files/240331_PM.xlsx", sheet = "Sheet2")
title <- "EC50_RTX_N2-BC4_250331"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 96,
  plate_type = "round",
  #colour = "#2E008B",
  title = title
)

print(P1)
```


# April

# Monday 01-04-2024

## Cell Culture
### N2
- Not ready to split
- Added 5mL media

### Ramos BC 5
- Count: 1.6x10^6^ cells/mL
- Seeded: 5x10^5^ cells in 15mL

## EC50 Collection: 24329/24331
- Collected plates seeded on 24-03-2024 and 31-3-2024
- EC Plate collection protocol:
  1. Added 40$\mu$L Cell Titre Blue (CTB) to each conditioned well 
    * 20$\mu$L CTB/100$\mu$L of conditioned well recommended by manufacturer
  2. Incubated for 1hr at 37C
  3. Read on plate reader according to Cell Titre Blue Protocol

- Results:

```{r, echo=FALSE, out.width ='70%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Results/240329_EC50_CDC_RAMOS.png"

# Insert the image
knitr::include_graphics(image_path)

```

```{r, echo=FALSE, out.width ='70%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Results/240329_EC50_CDC_N2.png"

# Insert the image
knitr::include_graphics(image_path)

```

## EC50 RTX N2-BC4/RAMOS-BC5 24401 - Seeding (DNW)
- Seeded an EC50 experiement comparing the effects of RTX on N2-BC4 and RAMOS-BC5 with or without 10% serum 
- Increased [RTX] to 1mg/mL (closer to Sorcha EC50)
- Used RTX provided by Jamie
- Plate seeding protocol:
  1. Diluted cell suspension to seed 20000 cells/well in 25$\mu$L amounts
  
|Cell Line| Well Number | Required Cell / Volume total | Required cells/mL | Stock cells/mL | **Stock CS Volume** | **Media Volume** |
| :-------: | :------: | :--------:|:----------:|:---------:|:----:|
| RAMOS-BC5| 60 wells | 1.6x10^6^ cells in 2 mL     | 8x10^5^ | 1.6x10^6^ | **1 mL** | **1 mL** |

  2. Made RTX dilutions and added to respective wells in 50$\mu$L    
    - [RTX stock] = 10.3 mg/mL   
    - Drug volumes are added in triplicate  
    - Drug volumes are being added consititute 1/4 of well volume: [RTX working] needs to be 4x [RTX well]   
    - 6 wells per condition, 25$\mu$L per well ~ minimum of 150$\mu$L per condition needed (recommend 200$\mu$L)    

  
```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240401_DDT.xlsx")

# Display the data as a table
knitr::kable(excel_data)

```
  
  3. Added 10% serum or media control
  - This provides complement factors to bind BL-bound RTX and initiate CDC
  - Serum/media amounts added at 50 $\mu$L/well
  - Serum volume is added in 1:4 ratio (10$\mu$L serum in 100$\mu$L final well volume)
  - Serum stock mix is 600$\mu$L HS : 1.2 mL media / plate
  
  4. Plate is incubated for 72 hrs at 37C  


```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Write up/Lab Notebook - Table excel files/240322_PM.xlsx", sheet = "Sheet1")
title <- "EC50_RTX_N2-BC4_240401"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 96,
  plate_type = "round",
  #colour = "#2E008B",
  title = title
)

print(P1)
```


# Tuesday 02-04-2024

## N2/Ramos CD20 Flow Cytometry
- Got antibody from Chris for CD20
- Cat#: 562873

Cell Stain Protocol:  

1. Resuspend cells to 1x10^6^ cells in 1mL
2. Distribute CS into 3 wells of a 96-well plate
3. Spin down @ 1500g for 2min
4. Flick media out
5. Make Cell Stain:
  - 500uL PBS + 2uL Stain
  -Make this while cells are spinning down
6. Resuspend w/ 100uL in the first well and then resuspend the next 2 wells in the same stain volume
  - Consolidate into 1 well/sample
7. incubate in the fridge for 20min
8. Spin down @ 1500g for 2 min
9. Transfer to FACS tube and run

- Results:

```{r, echo=FALSE, out.width ='70%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Results/240402_RAMOS_N2_CD20_Flow.png"

# Insert the image
knitr::include_graphics(image_path)

```

# Thursday 04-04-2024

## Cell Culture
### N2
- Count: 2.1x10^6^ cells/mL
- Seeded: 5x10^5^ cells in 15mL

### Ramos
- Count: 1.9x10^6^ cells/mL
- Seeded: 5x10^5^ cells in 15mL

## EC50 Collection: 240401
- Collected plate seeded on 01-04-2024
- EC Plate collection protocol:
  1. Added 40$\mu$L Cell Titre Blue (CTB) to each conditioned well 
    * 20$\mu$L CTB/100$\mu$L of conditioned well recommended by manufacturer
  2. Incubated for 1hr at 37C
  3. Read on plate reader according to Cell Titre Blue Protocol

- Results:**DNW**

# Monday 08-04-2024

## Cell Culture
### N2
- Count: 1.4x10^6^ cells/mL
- Seeded: 3x10^5^ cells in 15mL

### Ramos
- Count: 2.9x10^6^ cells/mL
- Seeded: 3x10^5^ cells in 15mL

# Tuesday 09-04-2024

## Perla Drug Randomizer

- Randomized drugs for Perla Pucci mouse experiment

| Drug | Identification |
| :-------: | :------: |
| Vehicle  | B | 
| Beta 30 | F | 
| Beta 10 | D | 
| Plo 30  | A |
| Plo 10  | E | 
| TESA 4 | C | 
| TESA0,4  | G |

# Saturday 14-04-2024

## Cell Culture
### N2-BC4
- Count: 6.7x10^5^ cells/mL
- Seeded: 3x10^5^ cells in 15mL

### Ramos-BC5
- Count: 6.0x10^5^ cells/mL
- Seeded: 3x10^5^ cells in 15mL

## EC50 RTX N2-BC4/RAMOS-BC5 24414 - Seeding (DNW)
- Seeded an EC50 experiment comparing the effects of RTX on N2-BC4 and RAMOS-BC5 with or without 10% serum 
- Increased [RTX] to 1mg/mL (closer to Sorcha EC50)
- Used RTX provided by Jamie
- Looked at cell-intrisic effects of RTX (no serum)
- Plate seeding protocol:
  1. Diluted cell suspension to seed 20000 cells/well in 50$\mu$L amounts

```{r, echo=FALSE, fig.align='left', out.width='60%'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240414_DDT.xlsx", sheet = "Sheet3")

excel_data <- excel_data %>%
  mutate(across(where(is.numeric), format_scientific))

knitr::kable(excel_data, escape = FALSE, booktabs = TRUE)

```

  2. Made RTX dilutions and added to respective wells in 25$\mu$L   
    - [RTX stock] = 10.3 mg/mL   
    - Drug volumes are added in triplicate  
    - Drug volumes are being added consititute 1/4 of well volume: [RTX working] needs to be 4x [RTX well]    
    - 6 wells per condition, 25$\mu$L per well ~ minimum of 150$\mu$L per condition needed (recommend 200$\mu$L)    
 
  
```{r, echo=FALSE, fig.align='center', out.width='70%'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240414_DDT.xlsx", sheet = "Sheet2")

# Display the data as a table
knitr::kable(excel_data)

```
  
  3. Plate is incubated for 72 hrs at 37C

```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Write up/Lab Notebook - Table excel files/240322_PM.xlsx", sheet = "Sheet1")
title <- "EC50_RTX_N2-BC4_240414"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 96,
  plate_type = "round",
  #colour = "#2E008B",
  title = title
)

print(P1)
```

# Wednesday 17-04-2024 

## EC50 Collection: 240414
- Collected plate seeded on 14-04-2024
- EC Plate collection protocol:
  1. Added 40$\mu$L Cell Titre Blue (CTB) to each conditioned well 
    * 20$\mu$L CTB/100$\mu$L of conditioned well recommended by manufacturer
  2. Incubated for 1hr at 37C
  3. Read on plate reader according to Cell Titre Blue Protocol

- Results:DNW


# Monday 22-04-2024 

## Cell Culture
### N2-BC4
- Seeded: 3x10^5^ cells in 15mL

### Ramos-BC5
- Seeded: 3x10^5^ cells in 15mL

## Thawed NK-92 
- Obtained from Chris Steele
- Cultured in Advanced RPMI
- Added 200u/mL of IL-2 (#78036.1)
  * Obtained from Emily
  * Product listed as 4.1x10^4^ IU/ug
  * Stock diluted to 10ug/mL
  * Added .48 uL stock/mL of media
  
# Wednesday 24-04-2024 

## Cell Culture
### N2-BC4
- Seeded: 3x10^5^ cells in 15mL

### Ramos-BC5
- Seeded: 3x10^5^ cells in 15mL

### NK-92
- Looked very unhealthy
- Lots of dead cells/debris in the media
- Spun down, resuspended in 1mL of media
- Split into 4 wells of a 48 well plate with increasing amounts of IL-2

# Friday 26-04-2024 

## Cell Culture
### N2-BC4
- Seeded: 3x10^5^ cells in 15mL

### Ramos-BC5
- Seeded: 3x10^5^ cells in 15mL

### NK-92
- Cells still look unhealthy
- Tried to consolidate in single well of 24 well plate
- Spun down in eppendorf and resuspended in 1mL of media

# Monday 29-04-2024 

## Cell Culture
### N2-BC4
- Seeded: 3x10^5^ cells in 15mL

### Ramos-BC5
- Seeded: 3x10^5^ cells in 15mL

# Tuesday 30-04-2024 

## Cell Culture
### N2-BC4
- Count: 3x10^5^ cells/mL
- Seeded: 3x10^5^ cells in 12mL
- Reseeded FDC cells in T75

### Ramos-BC5
- Count: 4x10^5^ cells/mL
- Seeded: 4x10^5^ cells in 12mL

## Human Serum Reciept
- Serum from 3 donors ordered from Cambridge Biosciences
- Serum mixed together, aliquoted, and stored at -80

|Product | Product ID | Barcode ID | Volume |
  | :-------: | :------: |:-------: | :------: |
  | Human Serum - Fresh Frozen | SERSKF2SIL10-FSXX | PR24C441891 | 10 mL |
  | Human Serum - Fresh Frozen | SERSKF2SIL10-FSXX | PR23K435425 | 10 mL |
  | Human Serum - Fresh Frozen | SERSKF2SIL10-MSXX | PR23D435392 | 10 mL |

## Ramos/N2 CDC Testing  

- Tested CDC assay adapted from [Ge et al., 2019](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6315025/#CR32)
- CDC protocol
1. Seed 250k cells 500uL volumes per well of a 12-well plate  
2. Added RTX or media in 250uL volumes per well
  * [RTX Stock] = 10mg/mL
  * [RTX] working needs to be 4x final desired well volume
  * 1uL RTX stock added to 1mL Media
3. Add Serum or media in 250uL volumes per well
  * Serum amount should be 4x final % desired


```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Lab Notebook - Table excel files/240430_PM.xlsx", sheet = "Sheet1")
title <- "CDC Test N2 + RAMOS 240430"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 12,
  plate_type = "round",
  #colour = "#2E008B",
  title = title
)

print(P1)
```

### Results: DNW  
- No clear differences between +HIAS/+RTX and +NHS/+RTX  
- Seemed relatively healthy   
- Possible fixes:  
  * Increase RTX dose
  * Increase Serum %
  * Increase the amount of time
  * Add at the same time instead of pre-incubating with RTX
  
## EC50 RTX RAMOS-BC5 24430 - Seeding
- Seeded an EC50 experiement comparing the effects of RTX on N2-BC4 and RAMOS-BC5 with or without 10% serum 
- Increased [RTX] to 1mg/mL (closer to Sorcha EC50)
- Used RTX provided by Jamie
- Plate seeding protocol:
  1. Diluted cell suspension to seed 20000 cells/well in 50$\mu$L amounts

  2. Made RTX dilutions and added to respective wells in 25$\mu$L  
    - [RTX stock] = 10.3 mg/mL 
    - Drug volumes are added in triplicate
    - Drug volumes are being added consititute 1/4 of well volume: [RTX working] needs to be 4x [RTX well]  
    - 6 wells per condition, 25$\mu$L per well ~ minimum of 150$\mu$L per condition needed (recommend 200$\mu$L)  
    
```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240430_DDT.xlsx")

# Display the data as a table
knitr::kable(excel_data)

```
  
  3. Plate is incubated for 72 hrs at 37C

```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Lab Notebook - Table excel files/240430_PM.xlsx", sheet = "Sheet2")
title <- "EC50 CDC Test N2 + RAMOS 240430"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 96,
  plate_type = "round",
  #colour = "#2E008B",
  title = title
)

print(P1)
```

# May

# Wednesday 01-05-2024

## Cell Culture
### N2-BC4
- Count: 3x10^5^ cells/mL
- Seeded: 3x10^5^ cells in 12mL
- Reseeded FDC cells in T75

### Ramos-BC5
- Count: 4x10^5^ cells/mL
- Seeded: 4x10^5^ cells in 12mL



# June

# Monday 10-06-2024

## Cell Culture
- Split cells

|Cell Line Name | Count | Seeding Density |
  | :-------: | :------: |:-------: | 
  | Ramos BC 1 | 3.3x10^6^ cells/mL | 6.5x10^4^ cells/mL |
  | Ramos BC 3 | 3.3x10^6^ cells/mL | 6.5x10^4^ cells/mL |
  | Ramos BC 5 | 3.6x10^6^ cells/mL | 6.5x10^4^ cells/mL |
  
## EC50 RTX RAMOS-BC5 240610 - Seeding
- Seeded an EC50 experiement comparing the effects of RTX on N2-BC4 and RAMOS-BC5 with or without 10% serum 
- Seeded 3 plates each with a different RAMOS barcode population  

|Plate ID | Cell Line |
| :-------: | :------: |
| Plate 1 | RAMOS BC 1 |
| Plate 2 | RAMOS BC 3 |
| Plate 3 | RAMOS BC 5 |

- Increased [RTX] to >1mg/mL (closer to Sorcha EC50)
- Used RTX provided by Jamie
- Plate seeding protocol:
  1. Diluted cell suspension to seed 10000 cells/well in 50$\mu$L amounts
  
```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240610_DDT.xlsx", sheet = "Sheet3")

columns_to_format <- c("Cell Line", "Cell Count", "Required Cell total", "CS cells/mL")

excel_data <- apply_scientific_notation(excel_data, columns_to_format)

# Display the data as a table
knitr::kable(excel_data)
```

  2. Made RTX dilutions and added to respective wells in 25$\mu$L  
    - [RTX stock] = 10.3 mg/mL 
    - Drug volumes are added in triplicate
    - Drug volumes are being added consititute 1/4 of well volume: [RTX working] needs to be 4x [RTX well]  
    - 6 wells per condition, 25$\mu$L per well ~ minimum of 150$\mu$L per condition needed (recommend 200$\mu$L)
  
```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240610_DDT.xlsx", sheet = "Sheet2")

# Display the data as a table
knitr::kable(excel_data)

```
 
 3. Added HIAS/NHS to indicated wells
  * 25$\mu$L/well
  * Final well volume = 25% Serum (HIAS/NHS)
 4. Plate is incubated for 48 hrs at 37C

  
```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Lab Notebook - Table excel files/240610_PM.xlsx", sheet = "Sheet1")
title <- "EC50 CDC Test N2 + RAMOS 240430"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 96,
  plate_type = "round",
  #colour = "#2E008B",
  title = title
)

print(P1)
```


# Wednesday 13-06-2024

## Cell Culture
- Split cells

|Cell Line Name | Count | Seeding Density | Flask | Flask Volume | Media |
  | :-------: | :------: |:-------: | :-------: | :-------: | 
  | Ramos BC 1 | 3.3x10^6^ cells/mL | 6.5x10^4^ cells/mL | T75 | 20mL | Advanced RMPI |

## EC50 Collection: EC50_RTX_N2-BC4_10064
- Collected plates seeded on 10-06-2024
- EC Plate collection protocol:
  1. Added 40$\mu$L Cell Titre Blue (CTB) to each conditioned well 
    * 20$\mu$L CTB/100$\mu$L of conditioned well recommended by manufacturer
  2. Incubated for 1hr at 37C
  3. Read on plate reader according to Cell Titre Blue Protocol

### Results:
```{r, echo=FALSE, out.width ='70%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Results/240610_EC50_CDC_RAMOS_Results.png"

# Insert the image
knitr::include_graphics(image_path)

```
  

# Thursday 13-06-2024

## Cryopreservation - RAMOS-BC 1/3/5
- Cryopreserved 3 vials of the following cell lines:
* Ramos-BC 1
* Ramos-BC 3
* Ramos-BC 5
- ~2x10^6 cells/vial
- Freezing media: FBS + 10% DMSO

### Protocol

1. 



# Monday 24-06-2024

## Cell Culture
- Transfered RAMOS BC 3 to RPMI-1640

### Making RPMI

- RPMI_1

| Solution | ID code | Volume| % Total volume
| :-------: | :------: | :--------:|:----------:|
| RPMI 1640   | ---- | 500 mL     | 89% |
| FBS     | ---- | 56 mL    | 10% |
| Glutamax    | ---- | 5.6 mL    | 1% |

### Splitting Ramos BC 3

|Cell Line Name | Count | Seeding Density | Flask | Flask Volume | Media |
  | :-------: | :------: |:-------: | :-------: | :-------: | 
  | Ramos BC 1 | 2x10^6^ cells/mL | 1x10^5^ cells/mL | T75 | 20mL | Advanced RMPI |

1. Transferred CS to 50mL flask
2. Spun down RAMOS BC 3
3. Removed media and resuspended in 5mL of RPMI_1

## RAMOS RTX CDC Testing 240625 - Seeding

- Set up 6 well plates and incubated for various amounts of time to look at RTX-CDC in an expanded format
- Sample Plate setup:  

```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Lab Notebook - Table excel files/240624_PM.xlsx", sheet = "Sheet1")
title <- "EC50 CDC Test RAMOS 240624"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 6,
  plate_type = "round",
  #colour = "#2E008B",
  title = title
)

print(P1)
```
  

Plate Set Up:
1. Make working CS


# Tuesday 25-06-2024

## Cell Culture
- Split RAMOS
- Spun down flask and counted
  * Count: 2x10^6^ cells/mL
  * Added 1mL of CS in 19mL media (1x10^5^cells/mL)
  
## RAMOS RTX CDC Testing 240625 - Collection
- Assesed plates for cell death
- Used Trypan Blue Exclusion assay

Trypan Blue Protocol:  
1. Resuspend cells in each well
2. Take 10uL sample from each well and add to respective 0.2mL tube
3. Added 10uL Trypan blue to each 0.2mL tube and mix well
4. Added 10uL sample from tube to haemocytometer
5. Counted both live/dead in all 4 squares

### Results

```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Results/240625_CDC_6well_test_results.xlsx")

# Display the data as a table
knitr::kable(excel_data)

```

# July

# Wednesday 10-07-2024

## Cell Culture
- Split RAMOS BC 1
- Count: 1.65x10^6^
- Added 1.5mL CS into 18.5media

## RAMOS - RTX In Vitro CDC Drug Pressure Experiment
- Began RTX CDC In Vitro experiments
- Seeded RAMOS BC 1 into 2x 6 well plates
- Froze down 3 vials of RAMOS BC 1
  * These can be considered Day 0/Baseline pools
  * Marked with black mark on top of caps
  
### Freezing Down Cells Protocol
  1. Counted cells in suspension
  2. Took volume of cell suspension such that each vial would contain at least 2x10^6^ cells
  3. Spun down CS @ 300 rcf for 5min
  4. Discarded supernatant and resuspended in freezing media such that 1ml of freezing media contains 2x10^6^ cells
    - Freezing media: FBS + 10% DMSO
  5. Added 1mL CS in freezing media per cryovial
  6. Cryovials were immediately put into freezing caddy and placed in -80 freezer
  7. After 24hrs vials removed from freezing caddy and plced in Liquid Nitrogen  
    - Location: Tank 1, Rack 5, Box 6
  
  
  
### RAMOS RTX DP - Dose 1
- Began RTX CDC In Vitro dosing
- Seeded RAMOS BC 1 into 2x 6 well plates

#### Dosing Protocol

1. Count CS and dilute to 1x10^5^ cells in 1 mL
  - If cell count is below either re-culture or add required CS amount, spin down, and resuspend in 1mL
2. Add 1mL of cell suspension containing 1x10^5^ cells to respective wells of 6-well plate
3. Made RTX dilutions and added to respective Rx wells in 500$\mu$L    
    - [RTX stock] = 10.3 mg/mL  
    - Drug volumes are being added consititute 1/4 of well volume:   
      * [RTX working] needs to be 4x [RTX well]   
    - 6 wells per RTX dosing, 500$\mu$L per well ~ minimum of 3000$\mu$L per condition needed (recommend 3500$\mu$L)
    - 500$\mu$L media added to Cx wells
 3. Added NHS to all wells
  * 500$\mu$L/well
  * Final well volume = 25% Serum (NHS)
 4. Plates incubated for 24 hrs at 37C

#### Plate Layout  


```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Lab Notebook - Table excel files/240710_PM.xlsx", sheet = "Sheet1")
title <- "RAMOS RTX DP1 Control 240724"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 6,
  plate_type = "round",
  colour = "#2E008B",
  title = title
)

print(P1)
```

```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Lab Notebook - Table excel files/240710_PM.xlsx", sheet = "Sheet2")
title <- "RAMOS RTX DP1 Rituximab 240724"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 6,
  plate_type = "round",
  #colour = "#2E008B",
  title = title
)

print(P1)
```


# Wednesday 17-07-2024

## Cell Culture

### RAMOS BC 1 - Baseline
- Split 1/20 
- RPMI/10% FBS in T75

### RAMOS BC 1 - Cx/DP2
- Split 0.7x10^6^ cells/flask
- Use for RTX CDC EC50
- RPMI/10% FBS in T25



### RAMOS BC 1 - Rx/DP2
- Split 0.7x10^6^ cells/flask
- Use for RTX CDC EC50
- RPMI/10% FBS in T25




# Friday 19-07-2024

## Cell Culture

### RAMOS BC 1 - Baseline
- Split 1/20 
- RPMI/10% FBS in T75

## RAMOS - RTX In Vitro CDC Drug Pressure Experiment

### RAMOS BC 1 - Cx/DP2 and Rx/DP2
- Cells from RAMOS RTX CDC DP
- Following indicator will be used:
  * Cx - Control RAMOS lines 1-6
  * Rx - Rituximab treated RAMOS lines 1-6
  * 2nd Dose of RTX (DP2) - 10ug/mL
- Cells were expanded from 6-well plates to indvidiual T25 flasks in 6mL of media
  * Cx-DP2 were expanded by taking 1/2 of 6-well CS
  * Rx-DP2 were fully expanded
  
## EC50 RTX RAMOS-BC5 240719 - Seeding
- Seeded an EC50 experiment comparing the effects of RTX on Cx/Rx-DP1 cell lines in the prescense of 25% NHS 
- Seeded 3 plates each with a different RAMOS-DP line 
- Used Rixathon (Catalogue#: )

### Plate seeding protocol:  

  1. Diluted cell suspension to seed 10000 cells/well in 50$\mu$L amounts
  
```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240719_DDT.xlsx", sheet = "Sheet3")

columns_to_format <- c("Cell Line", "Cell Count", "Required Cell total", "CS cells/mL")

excel_data <- apply_scientific_notation(excel_data, columns_to_format)

# Display the data as a table
knitr::kable(excel_data)

```

  2. Made RTX dilutions and added to respective wells in 25$\mu$L    
    - [RTX stock] = 10.3 mg/mL    
    - Drug volumes are added in triplicate  
    - Drug volumes are being added consititute 1/4 of well volume:   
      * [RTX working] needs to be 4x [RTX well]   
    - 6 wells per condition, 25$\mu$L per well ~ minimum of 150$\mu$L per condition needed (recommend 200$\mu$L)
  
```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240719_DDT.xlsx", sheet = "Sheet2")

# Display the data as a table
knitr::kable(excel_data)

```
 
 3. Added HIAS/NHS to indicated wells
  * 25$\mu$L/well
  * Final well volume = 25% Serum (HIAS/NHS)
 4. Plate is incubated for 48 hrs at 37C

  
```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Lab Notebook - Table excel files/240610_PM.xlsx", sheet = "Sheet1")
title <- "EC50 CDC Test N2 + RAMOS 240719"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 96,
  plate_type = "round",
  #colour = "#2E008B",
  title = title
)

print(P1)
```

# Sunday 21-07-2024

## Cell Culture

### RAMOS BC 1 - Baseline
- Split 1/20 
- RPMI/10% FBS in T75

### RAMOS BC 1 - Cx/DP2
- Split 1/6
- RPMI/10% FBS in T25

### RAMOS BC 1 - Rx/DP2
- Not Growing well, still very sparse
  * Transferred to 15mL Eppendorf tubes
  * Spun down @ 300rcf for 5 min
  * Resuspended in 1mL RPMI/10% FBS
  * Added to seperate wells of 12 well plate
- RPMI/10% FBS in 12-well plate

## EC50 Collection: EC50_RTX_N2-BC4_240721
- Collected plates seeded on 21-07-2024
- EC Plate collection protocol:
  1. Added 40$\mu$L Cell Titre Blue (CTB) to each conditioned well 
    * 20$\mu$L CTB/100$\mu$L of conditioned well recommended by manufacturer
  2. Incubated for 1hr at 37C
  3. Read on plate reader according to Cell Titre Blue Protocol

### Results:
- Dosages too high
- Need to decrease amounts for next EC50
  * RTX1 should be ~20ug/mL
```{r, echo=FALSE, out.width ='70%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Results/240719_EC50_RTX_CDC_RAMOS_DP1.png"

# Insert the image
knitr::include_graphics(image_path)

```

# Tuesday 23-07-2024

### RAMOS BC 1 - Baseline
- Split 1/20 
- RPMI/10% FBS in T75

### RAMOS BC 1 - Cx/DP2
- Split 1/6
- RPMI/10% FBS in T25

### RAMOS BC 1 - Rx/DP2
- Expanded into T25
- RPMI/10% FBS in T25

# Wednesday 24-07-2024

## Cell Culture

### RAMOS BC 1 - Baseline
- Split 1/3 
- RPMI/10% FBS in T75

### RAMOS BC 1 - Cx/DP2
- Expanded 1/2 into T75
- RPMI/10% FBS in T75

### RAMOS BC 1 - Rx/DP2
- Expanded into T75
- RPMI/10% FBS in T75

### Made Media - RPMI
- Media recipe

| Solution | ID code | Volume| % Total volume
| :-------: | :------: | :--------:|:----------:|
| RPMI 1640   | ---- | 500 mL     | 89% |
| FBS     | ---- | 56 mL    | 10% |
| Glutamax    | ---- | 5.6 mL    | 1% |
| Pen/Strep   | ---- | 5.6 mL   | 1%  |


# Friday 26-07-2024

### RAMOS BC 1 - Baseline
- Split 1/6 
- RPMI/10% FBS in T75

### RAMOS BC 1 - Cx/DP2
- Split 1/6 
- RPMI/10% FBS in T75

### RAMOS BC 1 - Rx/DP2
- Added 2-3mL of fresh media

# Monday 29-07-2024

### RAMOS BC 1 - Baseline
- Split 1/3 
- RPMI/10% FBS in T75

### RAMOS BC 1 - Cx/DP2
- Add 5mL of RPMI/10% FBS 

### RAMOS BC 1 - Rx/DP2
- Cells still look unhealthy/too sparse to use
- Spun down and resuspended in 6mL of media in new T25 flask
- RPMI/10% FBS in T75

```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Cell Counts/240730_Cell_Counts.xlsx", sheet = "Sheet1")

columns_to_format <- c("Cell Count")

excel_data <- apply_scientific_notation(excel_data, columns_to_format)

# Display the data as a table
knitr::kable(excel_data, digits = 1)

```



# Tuesday 30-07-2024



## RAMOS RTX DP - Dose 3 
- Began RTX CDC In Vitro dosing
- Seeded RAMOS BC 1 into 2x 6 well plates

#### Dosing Protocol

1. Count CS and dilute to 2x10^5^ cells in 1 mL
  - If cell count is below either re-culture or add required CS amount, spin down, and resuspend in 1mL
2. Add 1mL of cell suspension containing 2x10^5^ cells to respective wells of 6-well plate
3. Made RTX dilutions and added to respective Rx wells in 500$\mu$L    
    - [RTX stock] = 10.3 mg/mL  
    - Drug volumes are being added consititute 1/4 of well volume:   
      * [RTX working] needs to be 4x [RTX well]   
    - 6 wells per RTX dosing, 500$\mu$L per well ~ minimum of 3000$\mu$L per condition needed (recommend 3500$\mu$L)
    - 500$\mu$L media added to Cx wells
 3. Added NHS to all wells
  * 500$\mu$L/well
  * Final well volume = 25% Serum (NHS)
 4. Plates incubated for 24 hrs at 37C

#### Plate Layout  

Cell Count

```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240726_DDT_DP3_Dosing.xlsx", sheet = "Sheet1")

columns_to_format <- c("Cell Count", "Final cells/well")

excel_data <- apply_scientific_notation(excel_data, columns_to_format)

# Display the data as a table
knitr::kable(excel_data, digits = 1)

```

```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Lab Notebook - Table excel files/240710_PM.xlsx", sheet = "Sheet1")
title <- "RAMOS RTX DP3 Control 240726"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 6,
  plate_type = "round",
  colour = "#2E008B",
  title = title
)

print(P1)
```

```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Lab Notebook - Table excel files/240710_PM.xlsx", sheet = "Sheet2")
title <- "RAMOS RTX DP3 Rituximab 2407246"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 6,
  plate_type = "round",
  #colour = "#2E008B",
  title = title
)

print(P1)
```


## EC50 RTX RAMOS-DP2 240730 - Seeding
- Seeded an EC50 experiment comparing the effects of RTX on Cx/Rx-DP1 cell lines in the prescense of 25% NHS 
- Seeded 3 plates each with a different RAMOS-DP line 
- Used Rixathon (Catalogue#: )

### Plate seeding protocol:  

  1. Diluted cell suspension to seed 10000 cells/well in 50$\mu$L amounts
  
```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/25_DDT_EC50.xlsx", sheet = "Sheet3")

columns_to_format <- c("Cell Line", "Cell Count", "Required Cell total", "CS cells/mL")

excel_data <- apply_scientific_notation(excel_data, columns_to_format)

# Display the data as a table
knitr::kable(excel_data)

```

  2. Made RTX dilutions and added to respective wells in 25$\mu$L    
    - [RTX stock] = 10.3 mg/mL    
    - Drug volumes are added in triplicate  
    - Drug volumes are being added consititute 1/4 of well volume:   
      * [RTX working] needs to be 4x [RTX well]   
    - 6 wells per condition, 25$\mu$L per well ~ minimum of 150$\mu$L per condition needed (recommend 200$\mu$L)
  
```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240726_DDT_EC50.xlsx", sheet = "Sheet2")

# Display the data as a table
knitr::kable(excel_data, digits = 1)

```
 
 3. Added HIAS/NHS to indicated wells
  * 25$\mu$L/well
  * Final well volume = 25% Serum (HIAS/NHS)
 4. Plate is incubated for 48 hrs at 37C

  
```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Lab Notebook - Table excel files/240726_PM_EC50.xlsx", sheet = "Sheet1")
title <- "EC50 CDC Test N2 + RAMOS 240730"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 96,
  plate_type = "round",
  #colour = "#2E008B",
  title = title
)

print(P1)
```

## RAMOS RTX CDC DP2 - CD20 Flow Cytometry
- Checked CD20 expression between Cx and Rx DP2
- CD20:
- Samples tested: 
  * Baseline RAMOS 
  * R1
  * R3 
  * R6
  * C1 
  * C3
  * C6


### CD20 Flow Protocol

- Prior to starting: Make cell stain 
  * 500$\mu$L  PBS + 2$\mu$L Stain
  * 100$\mu$L of stain used/condition
- Cell stain made: 2000 $\mu$L and 8 $\mu$L 

1. Resuspend cells and transferred to 3 wells of a 96 well plate
2. Spun down at 1500g for 2min
3. Flick media out
4. Resuspend w/ 100$\mu$L stain in well and consolidate in 1 well
  * Add 100$\mu$L to bottom well and mix until cells resuspended
  * Take 100$\mu$L CS-stain mix and add to well below and mix
  * Repeat for third well and add the total CS-Stain mix to top well
5. Incubate in fridge for 25min
  * In the dark
6. Spin down at 1500g for 2min
7. Flick out media
8. Resuspend in PBS and add to FACS tube
  * Add an additional volume of PBS to dilute cells appropriately for flow
  * Usually make it up to about 300-400$\mu$L CS-PBS in the tube
  
### Results
- No apparent change in CD20 expression
- This is interesting based on strong evidence which suggests loss of CD20 is the primary mechanism of RTX resistance
- Will need to compare to EC50_240730 results
  * Are Rx-DP2 actually resistant compared to Cx-DP2?

```{r, echo=FALSE, out.width ='100%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Results/240731_RAMOS_RTX_CDC_CD20_flow.png"

# Insert the image
knitr::include_graphics(image_path)

```

# Wednesday 31-07-2024

## RAMOS BC 1 - Baseline
- Split 1/6 
- RPMI/10% FBS in T75

## RAMOS BC 1 - Rx/DP2
- Expanded remaining Rx/DP2 into T75
- RPMI/10% FBS in T75

| DP Line | CS Volume | Media Volume|
| :-------: | :------: | :--------:|
| R1-DP2   | 10 mL | 10 mL     |
| R2-DP2    |10 mL | 10 mL    |
| R4-DP2   | 5 mL | 5 mL     |
| R6-DP2    |8 mL | 10 mL    |

## RAMOS RTX DP - Dose 3 Collection
- Decided to collect after 24hrs due to high amount of cell killing 
  * Most examples in the literature do 24hr collection

### Collection Protocol
#### Rx-DP3:

1. Well volume transferred to 15ml Eppendorf
2. Eppendorfs supn down at 200 rcf for 6 min
3. Supernatant discarded and R1-4, R6 resuspended in 1ml
  - R5 resuspended in 500$\mu$L due to small pellet
4. Collected cells added to individual wells of 12-well plate
  - R5 added to 24-well plate
  
#### Cx-DP3

- Good viability for all wells
- Entire CS transferred to individal T75s and topped up with 8mL of RPMI/10% FBS in T75

# Thursday 01-08-2024

## Rx-DP3 Culture
- Cells are growing but lots of dead cells as well
- Added 1mL to all wells
  * 500uL to R5

## Cx-RP3 Culture
- Look healthy but still growing, no split

## EC50_240730 Collection - RAMOS RTX CDC DP2

- Collected plates seeded on 21-07-2024
- EC Plate collection protocol:
  1. Added 20$\mu$L Cell Titre Blue (CTB) to each conditioned well 
    * 20$\mu$L CTB/100$\mu$L of conditioned well recommended by manufacturer
  2. Incubated for 2hr at 37C
  3. Read on plate reader according to Cell Titre Blue Protocol

### Results:
- Less response for Rx-DP2 vs Cx-DP2
- Not sure its fully "resistant", still a robust response to RTX dosing for Rx-DP2
```{r, echo=FALSE, out.width ='70%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Results/240719_EC50_RTX_CDC_RAMOS_DP2.png"

# Insert the image
knitr::include_graphics(image_path)

```

# Friday 02-08-2024

## Rx-DP3 Culture
- Cells growing, expanded to 6 well plate
- R5 unhealthy
  * Spun down and transferred to single well of 48 well plate

## Cx-DP3 Culture
- Split 1/2 and added 10ml
- Collected samples for western/barcoding
  1. Collected 1x10^6^ cells in 15 mL tube
  2. Spun down @ 300 rcf for 5min
  3. Resuspended in 500uL PBS and transferred to 1.5mL Eppendorf
  4. Spun down @ max speed for 5 min at 4C
  5. Remove supernatant
  6. Snap freeze cell pellet and transfer to -80C
  
## Ramos Baseline 
- Split 1/6 
- Gave aliquot to Chris
- Count: 3.68x10^6^ cells/mL

# Saturday 03-08-2024

## Cell Culture
- Add media to all

# Sunday 04-08-2024

## Cx-DP3 Culture
- Split 1/2 and added 10ml
- Collected samples for western/barcoding


## EC50 RTX RAMOS-DP3 240804 - Seeding
- Seeded an EC50 experiment comparing the effects of RTX + 25% NHS vs 25% HIAS on Cx-DP3 cell lines
  * 3x lines (C1, C2, C5) exposed to RTX + 25% HIAS while 1x line (C6) exposed to RTX + 25% NHS
- Seeded 2 plates each section with a different Cx-DP3 line 
- Used Rixathon (Catalogue#: )

### Plate seeding protocol:  

  1. Diluted cell suspension to seed 10000 cells/well in 50$\mu$L amounts
  
```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240804_DDT_EC50.xlsx", sheet = "Sheet3")

columns_to_format <- c("Cell Line", "Cell Count", "Required Cell total", "CS cells/mL")

excel_data <- apply_scientific_notation(excel_data, columns_to_format)

# Display the data as a table
knitr::kable(excel_data)

```

  2. Made RTX dilutions and added to respective wells in 25$\mu$L    
    - [RTX stock] = 10.3 mg/mL    
    - Drug volumes are added in triplicate  
    - Drug volumes are being added consititute 1/4 of well volume:   
      * [RTX working] needs to be 4x [RTX well]   
    - 6 wells per condition, 25$\mu$L per well ~ minimum of 150$\mu$L per condition needed (recommend 200$\mu$L)
  
```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240804_DDT_EC50.xlsx", sheet = "Sheet2")

# Display the data as a table
knitr::kable(excel_data, digits = 1)

```
 
 3. Added HIAS/NHS to indicated wells
  * 25$\mu$L/well
  * Final well volume = 25% Serum (HIAS/NHS)
 4. Plate is incubated for 48 hrs at 37C

  
```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240804_DDT_EC50.xlsx", sheet = "Sheet4")
title <- "EC50 240804 Plate 1"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 96,
  plate_type = "round",
  #colour = "#2E008B",
  title = title
)

print(P1)
```

```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240804_DDT_EC50.xlsx", sheet = "Sheet5")
title <- "EC50 240804 Plate 2"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 96,
  plate_type = "round",
  #colour = "#2E008B",
  title = title
)

print(P1)
```

# Tuesday 06-08-2024

## RAMOS Baseline Culture
- Split 1/6

## Cx-DP3 Culture
- Split 1/6 and added 10ml
 
## EC50_240804 Collection - RAMOS RTX CDC DP3

- Collected plates seeded on 06-08-2024
- EC Plate collection protocol:
  1. Added 20$\mu$L Cell Titre Blue (CTB) to each conditioned well 
    * 20$\mu$L CTB/100$\mu$L of conditioned well recommended by manufacturer
  2. Incubated for 2hr at 37C
  3. Read on plate reader according to Cell Titre Blue Protocol

### Results:
- HIAS impact on RTX CDC similar to previous experiments
- NHS impact on RTX CDC is consistent with previous experiments
```{r, echo=FALSE, out.width ='70%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Results/240804_EC50_RTX_CDC_RAMOS_DP3.png"

# Insert the image
knitr::include_graphics(image_path)

``` 


## EC50 RTX RAMOS-DP3 240806 - Seeding
- Seeded an EC50 experiment comparing the effects of RTX + 25% NHS on Rx-DP3 cell lines
  * 2x lines (R1 + R3) exposed to RTX + 25% NHS
- Seeded 1 plate, each section with a different Rx-DP3 line 
- Used Rixathon (Catalogue#: )

### Plate seeding protocol:  

  1. Diluted cell suspension to seed 10000 cells/well in 50$\mu$L amounts
  
```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240806_DDT_EC50.xlsx", sheet = "Sheet3")

columns_to_format <- c("Cell Line", "Cell Count", "Required Cell total", "CS cells/mL")

excel_data <- apply_scientific_notation(excel_data, columns_to_format)

# Display the data as a table
knitr::kable(excel_data)

```

  2. Made RTX dilutions and added to respective wells in 25$\mu$L    
    - [RTX stock] = 10.3 mg/mL    
    - Drug volumes are added in triplicate  
    - Drug volumes are being added consititute 1/4 of well volume:   
      * [RTX working] needs to be 4x [RTX well]   
    - 6 wells per condition, 25$\mu$L per well ~ minimum of 150$\mu$L per condition needed (recommend 200$\mu$L)
  
```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240806_DDT_EC50.xlsx", sheet = "Sheet2")

# Display the data as a table
knitr::kable(excel_data, digits = 1)

```
 
 3. Added HIAS/NHS to indicated wells
  * 25$\mu$L/well
  * Final well volume = 25% Serum (HIAS/NHS)
 4. Plate is incubated for 48 hrs at 37C

  
```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240806_DDT_EC50.xlsx", sheet = "Sheet4")
title <- "EC50 240806 Plate 1"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 96,
  plate_type = "round",
  #colour = "#2E008B",
  title = title
)

print(P1)
```

# Wednesday 07-08-2024

## Cell Culture
- All cells growing fine, not ready to split/use experimentally
  * RAMOS Baseline (BC 1)
  * Cx-DP3
  * Rx-DP3

# Thursday 08-08-2024

## Ramos Baseline Culture
- Split 1/2

## Cx-DP3 Culture
- Split 1/2

## Rx-DP3 Culture
- Added media to Rx-DP3 cultures
  * R1/R3/R4/R6 added 8mL media
  * R2 added 4mL

## EC50_240806 Collection - RAMOS RTX CDC DP3

- Collected plates seeded on 06-08-2024
- EC Plate collection protocol:
  1. Added 20$\mu$L Cell Titre Blue (CTB) to each conditioned well 
    * 20$\mu$L CTB/100$\mu$L of conditioned well recommended by manufacturer
  2. Incubated for 2hr at 37C
  3. Read on plate reader according to Cell Titre Blue Protocol

### Results:
- HIAS impact on RTX CDC similar to previous experiments
- NHS impact on RTX CDC is consistent with previous experiments
```{r, echo=FALSE, out.width ='70%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Results/240804_EC50_RTX_CDC_RAMOS_DP3.png"

# Insert the image
knitr::include_graphics(image_path)

``` 

## RAMOS RTX CDC DP3 - CD20 Flow Cytometry
- Checked CD20 expression between Cx and Rx DP4
- CD20:
- Samples tested: 
  * Baseline RAMOS 
  * R2
  * R3 
  * R4
  * C1 
  * C2
  * C6


### CD20 Flow Protocol

- Prior to starting: Make cell stain 
  * 500$\mu$L  PBS + 2$\mu$L Stain
  * 100$\mu$L of stain used/condition
- Cell stain made: 2000 $\mu$L and 8 $\mu$L 

1. Resuspend cells and transferred to 3 wells of a 96 well plate
2. Spun down at 1500g for 2min
3. Flick media out
4. Resuspend w/ 100$\mu$L stain in well and consolidate in 1 well
  * Add 100$\mu$L to bottom well and mix until cells resuspended
  * Take 100$\mu$L CS-stain mix and add to well below and mix
  * Repeat for third well and add the total CS-Stain mix to top well
5. Incubate in fridge for 25min
  * In the dark
6. Spin down at 1500g for 2min
7. Flick out media
8. Resuspend in PBS and add to FACS tube
  * Add an additional volume of PBS to dilute cells appropriately for flow
  * Usually make it up to about 300-400$\mu$L CS-PBS in the tube
  
### Results
- No apparent change in CD20 expression
- This is interesting based on strong evidence which suggests loss of CD20 is the primary mechanism of RTX resistance
- GFP+% is not ideal

```{r, echo=FALSE, out.width ='100%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Results/240808_RAMOS_RTX_CDC_CD20_flow.png"

# Insert the image
knitr::include_graphics(image_path)

```

```{r, echo=FALSE, out.width ='100%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Results/240808_RAMOS_RTX_CDC_CD20_flow_A.png"

# Insert the image
knitr::include_graphics(image_path)

```


# Friday 09-08-2024

## Ramos Baseline Culture
- Split 1/6

## Cx-DP3 Culture
- Split 1/6

## Rx-DP3 Culture
- Split 1/2

# Monday 12-08-2024

## Ramos Baseline Culture
- Split 1/2

## Cx-DP3 Culture
- Split 1/2

## Rx-DP3 Culture
- Split 1/2
- Froze Down R4-DP2
  1. Counted cells in suspension
  2. Took volume of cell suspension such that each vial would contain at least 2x10^6^ cells
  3. Spun down CS @ 300 rcf for 5min
  4. Discarded supernatant and resuspended in freezing media such that 1ml of freezing media contains 2x10^6^ cells
    - Freezing media: FBS + 10% DMSO
  5. Added 1mL CS in freezing media per cryovial
  6. Cryovials were immediately put into freezing caddy and placed in -80 freezer
  7. After 24hrs vials removed from freezing caddy and plced in Liquid Nitrogen  
    - Location: Tank 1, Rack 5, Box 6
    
# Wednesday 14-08-2024

## Ramos Baseline Culture
- Split 1/2

## DP3 Viable Freezing
- Viably Froze R4-DP2/Cx-DP3/Rx-DP3

### Freezing Down Cells Protocol
  1. Counted cells in suspension
  2. Took volume of cell suspension such that each vial would contain at least 2x10^6^ cells
  3. Spun down CS @ 300 rcf for 5min
  4. Discarded supernatant and resuspended in freezing media such that 1ml of freezing media contains 2x10^6^ cells
    - Freezing media: FBS + 10% DMSO
  5. Added 1mL CS in freezing media per cryovial
  6. Cryovials were immediately put into freezing caddy and placed in -80 freezer
  7. After 24hrs vials removed from freezing caddy and plced in Liquid Nitrogen  
    - Location: Tank 1, Rack 5, Box 6

## Dosing RAMOS-DP4 RTX CDC
- Began RTX CDC In Vitro dosing
- Seeded Cx/Rx-DP2 into 2x 6 well plates

#### Dosing Protocol

1. Count CS and diluted to 2x10^5^ cells in 1 mL
  - If cell count is below either re-culture or add required CS amount, spin down, and resuspend in 1mL
2. Add 1mL of cell suspension containing 2x10^5^ cells to respective wells of 6-well plate
3. Made RTX dilutions and added to respective Rx wells in 500$\mu$L    
    - [RTX stock] = 10.3 mg/mL 
    - [RTX Final] = 50$\mu$g/mL
    - Drug volumes are being added consititute 1/4 of well volume:   
      * [RTX working] needs to be 4x [RTX well]   
    - 6 wells per RTX dosing, 500$\mu$L per well ~ minimum of 3000$\mu$L per condition needed (recommend 3500$\mu$L)
    - 500$\mu$L media added to Cx wells
 3. Added NHS to all wells
  * 500$\mu$L/well
  * Final well volume = 25% Serum (NHS)
 4. Plates incubated for 24 hrs at 37C
 
#### Plate Layout  

Cell Count

```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Lab Notebook - Table excel files/240710_PM.xlsx", sheet = "Sheet3")

columns_to_format <- c("Cell Count", "Final cells/well")

excel_data <- apply_scientific_notation(excel_data, columns_to_format)

# Display the data as a table
knitr::kable(excel_data, digits = 1)

```

```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Lab Notebook - Table excel files/240710_PM.xlsx", sheet = "Sheet1")
title <- "RAMOS RTX DP3 Control 240726"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 6,
  plate_type = "round",
  colour = "#2E008B",
  title = title
)

print(P1)
```

```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Lab Notebook - Table excel files/240710_PM.xlsx", sheet = "Sheet2")
title <- "RAMOS RTX DP3 Rituximab 2407246"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 6,
  plate_type = "round",
  #colour = "#2E008B",
  title = title
)

print(P1)
```


## EC50 RTX RAMOS-DP3 240814 - Seeding
- Seeded an EC50 experiment comparing the effects of RTX + 25% NHS vs 25% HIAS on Cx-DP3 cell lines
  * 2x Rx-DP3 lines (R1 + R4) and 2x Cx-DP3 (C2 + C4) exposed to RTX + 25% HIAS
- Seeded 2 plates each section with a different Ramos conditioned lines
  * Plate 1: R1 + R4
  * Plate 2: C2 + C4
- Used Rixathon (Catalogue#: )

### Plate seeding protocol:  

  1. Diluted cell suspension to seed 10000 cells/well in 50$\mu$L amounts
  
```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240814_DDT_EC50.xlsx", sheet = "Sheet3")

columns_to_format <- c("Cell Line", "Cell Count", "Required Cell total", "CS cells/mL")

excel_data <- apply_scientific_notation(excel_data, columns_to_format)

# Display the data as a table
knitr::kable(excel_data)

```

  2. Made RTX dilutions and added to respective wells in 25$\mu$L    
    - [RTX stock] = 10.3 mg/mL    
    - Drug volumes are added in triplicate  
    - Drug volumes are being added consititute 1/4 of well volume:   
      * [RTX working] needs to be 4x [RTX well]   
    - 6 wells per condition, 25$\mu$L per well ~ minimum of 150$\mu$L per condition needed (recommend 200$\mu$L)
  
```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240814_DDT_EC50.xlsx", sheet = "Sheet2")

# Display the data as a table
knitr::kable(excel_data, digits = 1)

```
 
 3. Added HIAS/NHS to indicated wells
  * 25$\mu$L/well
  * Final well volume = 25% Serum (HIAS/NHS)
 4. Plate is incubated for 48 hrs at 37C

  
```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240814_DDT_EC50.xlsx", sheet = "Sheet4")
title <- "EC50 240814 Plate 1"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 96,
  plate_type = "round",
  #colour = "#2E008B",
  title = title
)

print(P1)
```

```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240814_DDT_EC50.xlsx", sheet = "Sheet5")
title <- "EC50 240814 Plate 2"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 96,
  plate_type = "round",
  #colour = "#2E008B",
  title = title
)

print(P1)
```

# Friday 16-08-2024

## Ramos Baseline - BC 1
- Split 1/10

## RAMOS RTX-DP4 
- Expanded Cx to T25s
- Spun down Rx-DP4 t+ added to 12 well plate

## Collect EC50 240814
- Add script here

## EC50_240814 Collection - RAMOS RTX CDC DP3

- Collected plates seeded on 14-08-2024
- EC Plate collection protocol:
  1. Added 20$\mu$L Cell Titre Blue (CTB) to each conditioned well 
    * 20$\mu$L CTB/100$\mu$L of conditioned well recommended by manufacturer
  2. Incubated for 2hr at 37C
  3. Read on plate reader according to Cell Titre Blue Protocol

### Results:
- HIAS impact on RTX CDC similar to previous experiments
- NHS impact on RTX CDC is consistent with previous experiments
```{r, echo=FALSE, out.width ='70%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Results/240814_EC50_RTX_CDC_RAMOS_DP3.png"

# Insert the image
knitr::include_graphics(image_path)

``` 

# Monday 19-08-2024

## RAMOS Rx-DP4 
- Expanded Rx to T25s

# Tuesday 20-08-2024

## RAMOS RTX-DP4 
- Expanded Rx-DP4 to T25s

## EC50 RTX RAMOS-DP3 240814 - Seeding
- Seeded an EC50 experiment comparing the effects of RTX + 25% NHS vs 25% HIAS on Cx-DP3 cell lines
  * 3x Rx-DP3 lines (R2,R3,R4) exposed to RTX + 25% HIAS
- Seeded 2 plates each section with a different Ramos conditioned lines
  * Plate 1: R2 + R3
  * Plate 2: R4
- Used Rixathon (Catalogue#: )

### Plate seeding protocol:  

  1. Diluted cell suspension to seed 10000 cells/well in 50$\mu$L amounts
  
```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240820_DDT_EC50.xlsx", sheet = "Sheet3")

columns_to_format <- c("Cell Line", "Cell Count", "Required Cell total", "CS cells/mL")

excel_data <- apply_scientific_notation(excel_data, columns_to_format)

# Display the data as a table
knitr::kable(excel_data)

```

  2. Made RTX dilutions and added to respective wells in 25$\mu$L    
    - [RTX stock] = 10.3 mg/mL    
    - Drug volumes are added in triplicate  
    - Drug volumes are being added consititute 1/4 of well volume:   
      * [RTX working] needs to be 4x [RTX well]   
    - 6 wells per condition, 25$\mu$L per well ~ minimum of 150$\mu$L per condition needed (recommend 200$\mu$L)
  
```{r, echo=FALSE, fig.align='center'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240820_DDT_EC50.xlsx", sheet = "Sheet2")

# Display the data as a table
knitr::kable(excel_data, digits = 1)

```
 
 3. Added HIAS/NHS to indicated wells
  * 25$\mu$L/well
  * Final well volume = 25% Serum (HIAS/NHS)
 4. Plate is incubated for 48 hrs at 37C

  
```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240820_DDT_EC50.xlsx", sheet = "Sheet4")
title <- "EC50 240820 Plate 1"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 96,
  plate_type = "round",
  #colour = "#2E008B",
  title = title
)

print(P1)
```

```{r, echo=FALSE}
df <- read_xlsx("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Drug Dilution Tables/240820_DDT_EC50.xlsx", sheet = "Sheet5")
title <- "EC50 240820 Plate 2"


P1 <- plate_plot(
  data = df,
  position = well,
  value = Condition,
  label = Value,
 label_size = 2,
  plate_size = 96,
  plate_type = "round",
  #colour = "#2E008B",
  title = title
)

print(P1)
```

# Wednesday 21-08-2024


## RAMOS RTX CDC DP4 - CD20 Flow Cytometry
- Checked CD20 expression between Cx and Rx DP4
- CD20:
- Samples tested: 
  * Baseline RAMOS 
  * R2
  * R3 
  * R4
  * C1 
  * C2
  * C6


### CD20 Flow Protocol

- Prior to starting: Make cell stain 
  * 500$\mu$L  PBS + 2$\mu$L Stain
  * 100$\mu$L of stain used/condition
- Cell stain made: 2000 $\mu$L and 8 $\mu$L 

1. Resuspend cells and transferred to 3 wells of a 96 well plate
2. Spun down at 1500g for 2min
3. Flick media out
4. Resuspend w/ 100$\mu$L stain in well and consolidate in 1 well
  * Add 100$\mu$L to bottom well and mix until cells resuspended
  * Take 100$\mu$L CS-stain mix and add to well below and mix
  * Repeat for third well and add the total CS-Stain mix to top well
5. Incubate in fridge for 25min
  * In the dark
6. Spin down at 1500g for 2min
7. Flick out media
8. Resuspend in PBS and add to FACS tube
  * Add an additional volume of PBS to dilute cells appropriately for flow
  * Usually make it up to about 300-400$\mu$L CS-PBS in the tube
  
### Results
- No apparent change in CD20 expression
- This is interesting based on strong evidence which suggests loss of CD20 is the primary mechanism of RTX resistance

```{r, echo=FALSE, out.width ='100%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Results/240821_RAMOS_RTX_CDC_CD20_flow.png"

# Insert the image
knitr::include_graphics(image_path)

```

- Histogram analysis is interesting
  * CD20 staining is not as strong
- This could be due to RTX exposure
  * CD20 Ab competing with bound RTX
  * Expression is not necessarily different
- Colours:
  * Rx-DP4: Green, Light blue, Dark Blue
  * Cx-DP4: Purple, Blue, Red

```{r, echo=FALSE, out.width ='100%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Results/240821_RAMOS-DP4_CD20_flow_histo.png"

# Insert the image
knitr::include_graphics(image_path)

```  

# Thursday 22-08-2024

## DP4 Collections 

### Viable Freezing
- Viably froze the following for splitseq
  * R1-DP4
  * R2-DP4
  * R3-DP4
  * R4-DP4
  * C1-DP4
  * C2-DP4
  * C3-DP4
  * C4-DP4
  
#### Protocol
- Need ~500k cells for each sample
  * Ex. one well from a 6 well plate
- Freeze down a few extra samples in order to test PCR cycles for split-seq


##### PBS + RI
- Make up fresh and store at 4C

|Component       |Volume/sample| Volume/20 samples |
|:--------------:|:----:|:---------:|
| PBS (TC Grade) | 2 mL |  42 mL   |
| RNase OUT        | 40 U/$\mu$L |  840 U/$\mu$L |
| SUPERase Inhibitor    | 20 U/$\mu$L | 420 U/$\mu$L |

##### PBS + Formaldehyde
- Make up fresh on the day and chill

|Component       |Volume/sample| Volume/20 samples |
|:--------------:|:----:|:---------:|
| PBS (TC Grade) | 2.75 mL |  57.75 mL   |
| 16% Formaldehyde (Pierce cat.28906) | 0.25 mL |  5.25 mL |

###### Protocol
1. Collect ~500k cells from cell suspension
2. Spin down at 180 x g (RCF)
3. Resuspend pellet in 1mL of PBS + RI
4.  Add 3mL of ice cold PBS + 1.33% formaldehyde and mix gently by pipetting up and down 2-3 times
5. Incubate on ice for 10min
6. Spin 300g for 3min at RT
7. Wash pellet with 1mL PBS + RI and respin 300g for 3min at RT
8. Remove supernatant and snap freeze pellet on dry ice
9. Store pellet at -80C
### Freezing Down Cells Protocol
  1. Counted cells in suspension
  2. Took volume of cell suspension such that each vial would contain at least 2x10^6^ cells
  3. Spun down CS @ 300 rcf for 5min
  4. Discarded supernatant and resuspended in freezing media such that 1ml of freezing media contains 2x10^6^ cells
    - Freezing media: FBS + 10% DMSO
  5. Added 1mL CS in freezing media per cryovial
  6. Cryovials were immediately put into freezing caddy and placed in -80 freezer
  7. After 24hrs vials removed from freezing caddy and plced in Liquid Nitrogen  
    - Location: Tank 1, Rack 5, Box 6

# Monday 10-09-2024

## RTX DP - RNA Extraction: Samples 4-24

### Overview
- Did RNA Extraction with samples 4-24
- RNeasy PLus Mini Kit - Qiagen
  * Cat #: 74134
  
### Protocol
  1. Added 600$\mu$L Buffer RLT Plus and homogenized lysate via vortexing
  2. Lysate was spun down for 3min at max speed, supernatant removed, and added to gDNA Eliminator spun column  
    - Spin column placed in 2ml Collection tube  
  3. Centrifuged for 30s at max speed
  4. Discarded columns and kept flow through
  5. 600$\mu$L 70% Ethanol added to flowthrough and mixed via pipetting
  6. Transferred 700$\mu$L of sample into an RNeasy Spin Column (placed in 2mL collection tube)
  7. Centrifuged at max speed for 30s
    - Repeated step 6-7 until all flowthrough has spun through spin column  
  8. Added 700$\mu$L Buffer RW1 to the RNedasy spin column and centrifuge at max speed for 30s
  9. Added 500$\mu$L Buffer RPE to spin column and centrifuge at max speed for 30s
  10. Added 500$\mu$L Buffer RPE to spin column and centrifuge at max speed for 30s (repeat of previous step)
  11. Placed spin column in empty tube and centrifuge at max speed for 1min
  12. Placed spin column in collection tube, added 30$\mu$L Elution Buffer and incubated at RT for 2min
  13. Centrifuge at max speed for 1min
  14. Determine concentration on nano-drop

### Results

```{r, echo=FALSE, fig.align='left'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Library_prep_tables/RTX_DP_LP_RNAextraction1.xlsx", sheet = "Sheet1")

# Display the data as a table
knitr::kable(excel_data, digits = 1)

```

# Tuesday 10-09-2024

## RTX DP - RNA Extraction: Samples 25-47

### Overview
- Did RNA Extraction with samples 25-47
- RNeasy PLus Mini Kit - Qiagen
  * Cat #: 74134
  
### Protocol
  1. Added 600$\mu$L Buffer RLT Plus and homogenized lysate via vortexing
  2. Lysate was spun down for 3min at max speed, supernatant removed, and added to gDNA Eliminator spun column  
    - Spin column placed in 2ml Collection tube  
  3. Centrifuged for 30s at max speed
  4. Discarded columns and kept flow through
  5. 600$\mu$L 70% Ethanol added to flowthrough and mixed via pipetting
  6. Transferred 700$\mu$L of sample into an RNeasy Spin Column (placed in 2mL collection tube)
  7. Centrifuged at max speed for 30s
    - Repeated step 6-7 until all flowthrough has spun through spin column  
  8. Added 700$\mu$L Buffer RW1 to the RNedasy spin column and centrifuge at max speed for 30s
  9. Added 500$\mu$L Buffer RPE to spin column and centrifuge at max speed for 30s
  10. Added 500$\mu$L Buffer RPE to spin column and centrifuge at max speed for 30s (repeat of previous step)
  11. Placed spin column in empty tube and centrifuge at max speed for 1min
  12. Placed spin column in collection tube, added 30$\mu$L Elution Buffer and incubated at RT for 2min
  13. Centrifuge at max speed for 1min
  14. Determine concentration on nano-drop

### Results

```{r, echo=FALSE, fig.align='left'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Library_prep_tables/RTX_DP_LP_RNAextraction2.xlsx", sheet = "Sheet1")

# Display the data as a table
knitr::kable(excel_data, digits = 1)

```

# Wednesday 11-09-2024

## RTX DP - RNA Extraction: Samples 1-3

### Overview
- Did RNA Extraction with samples 1-3
- RNeasy PLus Mini Kit - Qiagen
  * Cat #: 74134
  
### Protocol
  1. Added 600$\mu$L Buffer RLT Plus and homogenized lysate via vortexing
  2. Lysate was spun down for 3min at max speed, supernatant removed, and added to gDNA Eliminator spun column  
    - Spin column placed in 2ml Collection tube  
  3. Centrifuged for 30s at max speed
  4. Discarded columns and kept flow through
  5. 600$\mu$L 70% Ethanol added to flowthrough and mixed via pipetting
  6. Transferred 700$\mu$L of sample into an RNeasy Spin Column (placed in 2mL collection tube)
  7. Centrifuged at max speed for 30s
    - Repeated step 6-7 until all flowthrough has spun through spin column  
  8. Added 700$\mu$L Buffer RW1 to the RNedasy spin column and centrifuge at max speed for 30s
  9. Added 500$\mu$L Buffer RPE to spin column and centrifuge at max speed for 30s
  10. Added 500$\mu$L Buffer RPE to spin column and centrifuge at max speed for 30s (repeat of previous step)
  11. Placed spin column in empty tube and centrifuge at max speed for 1min
  12. Placed spin column in collection tube, added 30$\mu$L Elution Buffer and incubated at RT for 2min
  13. Centrifuge at max speed for 1min
  14. Determine concentration on nano-drop

### Results

```{r, echo=FALSE, fig.align='left'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Library_prep_tables/RTX_DP_LP_RNAextraction3.xlsx", sheet = "Sheet1")

# Display the data as a table
knitr::kable(excel_data, digits = 1)

```

# Friday 27-09-2024

## RTX DP - RT: Samples 1-47
### Overview
- cDNA synthesis from RNA template

### Materials

|Component       |Expected Volume/experiment| Material ID |
|:--------------:|:----:|:---------:|
| 5x SSIV Buffer | 200 $\mu$L | |
| SSIV RT        | 50 $\mu$L |   |
| 100 mM DTT     | 50 $\mu$L |    |
| RNAse Out      | 50 $\mu$L |   |
| Thermolabile Exonuclease I  | 50 $\mu$L |          |
| NEBuffer r3.1*        | 100 $\mu$L |          |


### RT Protocol  
1. In PCR strip tubes, prepare 1-5 $\mu$g of RNA in a total volume of 10 $\mu$l of RNAse/DNAase-free water.
2. Add 1 $\mu$l of WS_RT_UMI_NexteraR2 primer (2$\mu$M stock)
3. Add 1 $\mu$l dNTPs (10 mM each). 
4. Denature at 65 C for 5 mins in the PCR machine, then straight onto ice for at least 2 mins, spin briefly to get liquid to the bottom of the tube.
5. Prepare mastermix of RT enzyme and buffers. 1.1x the number of samples.

|Component       |Volume|   MM volume  |
|:--------------|----|-----------------|
| 5x SSIV Buffer | 4 $\mu$L |     208 $\mu$L     |
| SSIV RT        | 1 $\mu$L |      52 $\mu$L    |
| 100 mM DTT     | 1 $\mu$L |      52 $\mu$L   |
| RNAse Out      | 1 $\mu$L |       52 $\mu$L   |

6. Add 7 $\mu$l of RT MM prepared above to each sample and mix, spin briefly to get liquid to the bottom of the tube.
   - Spin briefly to get liquid to bottom of the tube 
7. In the PCR machine incubate at 53 C for 10 mins followed by 80 C for 10 mins.
8. Add 1 $\mu$l Themolabile Exonuclease I (NEB M0568) to remove excess RT primer.

|Component       |Volume|   MM volume  |
|:--------------|----|-----------------|
| Thermolabile Exonuclease I  | 1 $\mu$L |          |
| NEBuffer r3.1*        | 2 $\mu$L |          |

\begin{center}
\textbf{Most PCR buffers are compatible}
\end{center}

9. In PCR Machine: heat at 37C for 4 mins followed by 80C for 1 min  
    - Spin briefly to get liquid to bottom of the tube
10. Add 1 $\mu$l of RNAse H and incubate at 37 C for 20 mins.
11. Dilute final cDNA 1:2 with DNAse/RNAse-free water.



# Monday 30-09-2024

## RTX DP - PCR1: Samples 1-47

## Overview
- Aim to keep number of cycles at a minimum. 
- Forward primer is an equal mix of primers with different length spacers to generate diversity required for sequencing.

## Materials

|Component       |Expected Volume/experiment| Material ID |
|:--------------|----|-----------------|:-------------:|
| 10uM WS_RT-PCR1_Fwd  | 70 $\mu$L |   -       |
| 10uM WS_RT-PCR1_Rev | 70 $\mu$L |     -     |
| DNAse/RNAse H20 | 550 $\mu$L |          |
| Kapa Hifi HotStart Ready Mix (2X) | 1200 $\mu$L |  KK2601  |

## PCR1 Protocol  

- Set up PCR reactions on ice
- One reaction per sample plus -RT control

1. Prepare mastermix of below reagents at 1.1x the number of samples

|Component       |Volume|
|:--------------|----|
| 10uM WS_RT-PCR1_Fwd | 1.5 $\mu$L |
|10uM WS_RT-PCR1_Rev | 1.5 $\mu$L |
| DNAse/RNAse H20      | 12 $\mu$L |
| Kapa Hifi HotStart Ready Mix (2X) | 25 $\mu$L |
|||
| **Total Master Mix volume**|**40 $\mu$L** |
|||
| cDNA from above        | 10 $\mu$L | 


2. Perform PCR using the following parameters:  

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 60C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 1 min |
Hold Step |Step 6: 12C    | Hold |  

\begin{center}
\textbf{10-25 cycles of steps 2-4}
\end{center}

Clean up reaction using Ampure beads and check product size and yield on tapestation

# Tuesday 01-10-2024

## RTX DP - Bead Clean Up: Samples 1-47
### Overview
- Clean PCR reaction by removing leftover primers
- This prevents false reads, cleaner indexing in later steps, and higher quality samples

### Materials
- For 40 samples

|Component       |Expected Volume/experiment| Material ID |
|:--------------:|:----:|:---------:|
| Ampure Bead Kit | 200 $\mu$L | |

### Protocol
1.	Equilibrate Ampure beads to room temp (available from supplies cold room)
2.	Mix well and vortex for 30sec to ensure uniform distribution.
3.	Add 90ul (1.8X volume) to PCR reaction and mix thoroughly by pipetting.
4.	Incubate 5min, room temp
5.	Place on magnet for 2 min and keep on magnet until final elution
6.	Remove supernatant and add 200ul 70% ethanol (make fresh)
7.	Incubate 30sec and remove
8.	Repeat 70% ethanol wash
9.	Air dry beads 5-10min room temp. Once dry the beads will go from shiny to matt in appearance. Avoid over drying (cracked appearance).
10.	Add 15 ul EB buffer and resuspend beads by pipetting
11.	Incubate 2min room temp
12.	Place on magnet and remove eluate
13. Assess PCR product size, contamination and concentration on tapestation.   
**Expected size = 165-172bp**

## RTX DP - Tape Station: Samples 1-8
### Overview


### Materials


### Protocol
1.	Added 2$\mu$L High Sensitvity D1000 Sample Buffer with 2$\mu$L DNA sample
2.	Spun down, then vortexed using IKA vortexer and adaptor at 2000rpm for 1 min
3.	Spun down to position sample at the bottom of the tube
4.	Loaded samples into the 2200 TapeStation

### Results

```{r, echo=FALSE, out.width ='100%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/tapestation/2024-10-01 - PCR1 SAMPLES 1-8 RTX RAMOS DP 1.pdf"

# Insert the image
knitr::include_graphics(image_path)

```  


# Wednesday 02-10-2024

## RTX DP - Tape Station: Samples 9-47
### Overview
- Clean PCR reaction by removing leftover primers
- This prevents false reads, cleaner indexing in later steps, and higher quality samples

### Materials
- For 40 samples

|Component       |Expected Volume/experiment| Material ID |
|:--------------:|:----:|:---------:|
| Ampure Bead Kit | 200 $\mu$L | |

### Protocol
1.	Added 2$\mu$L High Sensitvity D1000 Sample Buffer with 2$\mu$L DNA sample
2.	Spun down, then vortexed using IKA vortexer and adaptor at 2000rpm for 1 min
3.	Spun down to position sample at the bottom of the tube
4.	Loaded samples into the 2200 TapeStation

### Results

```{r, echo=FALSE, out.width ='100%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/tapestation/2024-10-02 - PCR1 SAMPLES 9-16 RTX RAMOS TS.pdf"

# Insert the image
knitr::include_graphics(image_path)

```  

```{r, echo=FALSE, out.width ='100%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/tapestation/2024-10-02 - PCR1 SAMPLES 17-32 RTX RAMOS TS.pdf"

# Insert the image
knitr::include_graphics(image_path)

``` 

```{r, echo=FALSE, out.width ='100%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/tapestation/2024-10-02 - PCR1 SAMPLES 33-48 RTX RAMOS TS.pdf"

# Insert the image
knitr::include_graphics(image_path)

``` 

# Thursday 03-10-2024

## RTX DP - PCR1: Repeat Samples 24 + 45

## Overview
- Repeated samples 24 + 45
- Aim to keep number of cycles at a minimum. 
- Forward primer is an equal mix of primers with different length spacers to generate diversity required for sequencing.

## Materials

|Component       |Expected Volume/experiment| Material ID |
|:--------------|----|-----------------|:-------------:|
| 10uM WS_RT-PCR1_Fwd  | 70 $\mu$L |   -       |
| 10uM WS_RT-PCR1_Rev | 70 $\mu$L |     -     |
| DNAse/RNAse H20 | 550 $\mu$L |          |
| Kapa Hifi HotStart Ready Mix (2X) | 1200 $\mu$L |  KK2601  |

## PCR1 Protocol  

- Set up PCR reactions on ice
- One reaction per sample plus -RT control

1. Prepare mastermix of below reagents at 1.1x the number of samples

|Component       |Volume|
|:--------------|----|
| 10uM WS_RT-PCR1_Fwd | 1.5 $\mu$L |
|10uM WS_RT-PCR1_Rev | 1.5 $\mu$L |
| DNAse/RNAse H20      | 12 $\mu$L |
| Kapa Hifi HotStart Ready Mix (2X) | 25 $\mu$L |
|||
| **Total Master Mix volume**|**40 $\mu$L** |
|||
| cDNA from above        | 10 $\mu$L | 


2. Perform PCR using the following parameters:  

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 60C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 1 min |
Hold Step |Step 6: 12C    | Hold |  

\begin{center}
\textbf{10-25 cycles of steps 2-4}
\end{center}

Clean up reaction using Ampure beads and check product size and yield on tapestation

# Friday 04-10-2024

## RTX DP - Tapestation: Repeat Samples 24 + 45
### Overview
- Run automated gel to check bandsize/location

### Materials

|Component       |Expected Volume/experiment| Material ID |
|:--------------:|:----:|:---------:|
| DS |  | |

### Protocol
1.	Added 2$\mu$L High Sensitvity D1000 Sample Buffer with 2$\mu$L DNA sample
2.	Spun down, then vortexed using IKA vortexer and adaptor at 2000rpm for 1 min
3.	Spun down to position sample at the bottom of the tube
4.	Loaded samples into the 2200 TapeStation

### Results

```{r, echo=FALSE, out.width ='100%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/tapestation/2024-10-04 - PCR1 REDO SAMPLES 24 + 45 RTX RAMOS TS.pdf"

# Insert the image
knitr::include_graphics(image_path)

``` 


# Monday 07-10-2024

## RTX DP - PCR2: 1-47
### Overview
- PCR2 for all samples using PCR1 product post cleanup
- PCR to index samples for sequencing by attaching UMI 
- VERY IMPORTANT to accurately record which samples receive which adaptors  
  *[Guide for Index Adaptor Pooling](https://support-docs.illumina.com/SHARE/IndexAdaptersPooling/Content/SHARE/FrontPages/IndexAdapterPooling.htm) 

### Materials

|Component       |Expected Volume/experiment| Material ID |
|:--------------|----|-----------------|:-------------:|
| 10uM Nxxx Nextera i7 adapter  | - |   -       |
| 10uM Sxxx Nextera i5 adapter | - |     -     |
| DNAse/RNAse H20 | 1000 $\mu$L |          |
| Q5 | 1200 $\mu$L |  M0492S  |

### Protocol
1. Prepare samples according to the following table:
  - Keep careful note of which adaptors are added to which sample

|Component       |Volume|
|:--------------|----|
| 10uM Nxxx Nextera i7 adapter  | 1.5 $\mu$L |
|10uM Sxxx Nextera i5 adapter | 1.5 $\mu$L |
| DNAse/RNAse H20      | 21 $\mu$L |
| Kapa Hifi HotStart Ready Mix (2X) | 25 $\mu$L |
| 10ng/$\mu$L PCR1  | 1 $\mu$L | 


2. Perform PCR using the following parameters:  

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 55C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 2 min |
Hold Step |Step 6: 12C    | Hold |  

\begin{center}
\textbf{8 cycles of steps 2-4}
\end{center}

3. Perform Ampure bead clean up as above with 90ul beads per 50ul PCR reaction.
- If you don’t have a crazy number of samples you might also want to consider doing up to 4 PCR2 reactions per sample and pooling them afterwards just to make sure all your sequences are captured.

4. Check size and concentration on tapestation. Dilute and pool samples for sequencing.

# Tuesday 08-10-2024

## RTX DP - Bead Clean Up 2: Samples 1-47
### Overview
- Clean PCR reaction by removing leftover primers
- This prevents false reads, cleaner indexing in later steps, and higher quality samples

### Materials
- For 40 samples

|Component       |Expected Volume/experiment| Material ID |
|:--------------:|:----:|:---------:|
| Ampure Bead Kit | 200 $\mu$L | |

### Protocol
1.	Equilibrate Ampure beads to room temp (available from supplies cold room)
2.	Mix well and vortex for 30sec to ensure uniform distribution.
3.	Add 90ul (1.8X volume) to PCR reaction and mix thoroughly by pipetting.
4.	Incubate 5min, room temp
5.	Place on magnet for 2 min and keep on magnet until final elution
6.	Remove supernatant and add 200ul 70% ethanol (make fresh)
7.	Incubate 30sec and remove
8.	Repeat 70% ethanol wash
9.	Air dry beads 5-10min room temp. Once dry the beads will go from shiny to matt in appearance. Avoid over drying (cracked appearance).
10.	Add 15 ul EB buffer and resuspend beads by pipetting
11.	Incubate 2min room temp
12.	Place on magnet and remove eluate
13. Assess PCR product size, contamination and concentration on tapestation.   
**Expected size = 165-172bp**

## RTX DP - Tapestation: Samples 1-16
### Overview
- Run automated gel to check bandsize/location

### Materials

|Component       |Expected Volume/experiment| Material ID |
|:--------------:|:----:|:---------:|
| DS |  | |

### Protocol
1.	Added 2$\mu$L High Sensitvity D1000 Sample Buffer with 2$\mu$L DNA sample
2.	Spun down, then vortexed using IKA vortexer and adaptor at 2000rpm for 1 min
3.	Spun down to position sample at the bottom of the tube
4.	Loaded samples into the 2200 TapeStation

### Results

```{r, echo=FALSE, out.width ='100%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/tapestation/2024-10-08 - PCR2 SAMPLES 1-8 RTX RAMOS DP TS.pdf"

# Insert the image
knitr::include_graphics(image_path)

```

```{r, echo=FALSE, out.width ='100%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/tapestation/2024-10-08 - PCR2 SAMPLES 9-16 RTX RAMOS DP TS.pdf"

# Insert the image
knitr::include_graphics(image_path)

``` 

# Wednesday 09-10-2024

## RTX DP - Tapestation: Samples 17-47
### Overview
- Run automated gel to check bandsize/location

### Materials

|Component       |Expected Volume/experiment| Material ID |
|:--------------:|:----:|:---------:|
| DS |  | |

### Protocol
1.	Added 2$\mu$L High Sensitvity D1000 Sample Buffer with 2$\mu$L DNA sample
2.	Spun down, then vortexed using IKA vortexer and adaptor at 2000rpm for 1 min
3.	Spun down to position sample at the bottom of the tube
4.	Loaded samples into the 2200 TapeStation

### Results

```{r, echo=FALSE, out.width ='100%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/tapestation/2024-10-09 - PCR2 SAMPLES 17-24 RTX RAMOS TS.pdf"

# Insert the image
knitr::include_graphics(image_path)

``` 


```{r, echo=FALSE, out.width ='100%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/tapestation/2024-10-09 - PCR2 SAMPLES 25-40 RTX RAMOS TS.pdf"

# Insert the image
knitr::include_graphics(image_path)

``` 

# Thursday 10-10-2024

## RTX DP - PCR2: Repeat Samples 1-8
### Overview
- PCR2 for all samples using PCR1 product post cleanup
- PCR to index samples for sequencing by attaching UMI 
- VERY IMPORTANT to accurately record which samples receive which adaptors  
  *[Guide for Index Adaptor Pooling](https://support-docs.illumina.com/SHARE/IndexAdaptersPooling/Content/SHARE/FrontPages/IndexAdapterPooling.htm) 

### Materials

|Component       |Expected Volume/experiment| Material ID |
|:--------------|----|-----------------|:-------------:|
| 10uM Nxxx Nextera i7 adapter  | - |   -       |
| 10uM Sxxx Nextera i5 adapter | - |     -     |
| DNAse/RNAse H20 | 1000 $\mu$L |          |
| Q5 | 1200 $\mu$L |  M0492S  |

### Protocol
1. Prepare samples according to the following table:
  - Keep careful note of which adaptors are added to which sample

|Component       |Volume|
|:--------------|----|
| 10uM Nxxx Nextera i7 adapter  | 1.5 $\mu$L |
|10uM Sxxx Nextera i5 adapter | 1.5 $\mu$L |
| DNAse/RNAse H20      | 21 $\mu$L |
| Kapa Hifi HotStart Ready Mix (2X) | 25 $\mu$L |
| 10ng/$\mu$L PCR1  | 1 $\mu$L | 


2. Perform PCR using the following parameters:  

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 55C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 2 min |
Hold Step |Step 6: 12C    | Hold |  

\begin{center}
\textbf{8 cycles of steps 2-4}
\end{center}

3. Perform Ampure bead clean up as above with 90ul beads per 50ul PCR reaction.
- If you don’t have a crazy number of samples you might also want to consider doing up to 4 PCR2 reactions per sample and pooling them afterwards just to make sure all your sequences are captured.

4. Check size and concentration on tapestation. Dilute and pool samples for sequencing.

## RTX DP - Bead Clean Up 2: Repeat Samples 1-8
### Overview
- Clean PCR reaction by removing leftover primers
- This prevents false reads, cleaner indexing in later steps, and higher quality samples

### Materials
- For 40 samples

|Component       |Expected Volume/experiment| Material ID |
|:--------------:|:----:|:---------:|
| Ampure Bead Kit | 200 $\mu$L | |

### Protocol
1.	Equilibrate Ampure beads to room temp (available from supplies cold room)
2.	Mix well and vortex for 30sec to ensure uniform distribution.
3.	Add 90ul (1.8X volume) to PCR reaction and mix thoroughly by pipetting.
4.	Incubate 5min, room temp
5.	Place on magnet for 2 min and keep on magnet until final elution
6.	Remove supernatant and add 200ul 70% ethanol (make fresh)
7.	Incubate 30sec and remove
8.	Repeat 70% ethanol wash
9.	Air dry beads 5-10min room temp. Once dry the beads will go from shiny to matt in appearance. Avoid over drying (cracked appearance).
10.	Add 15 ul EB buffer and resuspend beads by pipetting
11.	Incubate 2min room temp
12.	Place on magnet and remove eluate
13. Assess PCR product size, contamination and concentration on tapestation.   
**Expected size = 165-172bp**

## RTX DP - Tapestation: Repeat Samples 1-8
### Overview
- Run automated gel to check bandsize/location

### Materials

|Component       |Expected Volume/experiment| Material ID |
|:--------------:|:----:|:---------:|
| DS |  | |

### Protocol
1.	Added 2$\mu$L High Sensitvity D1000 Sample Buffer with 2$\mu$L DNA sample
2.	Spun down, then vortexed using IKA vortexer and adaptor at 2000rpm for 1 min
3.	Spun down to position sample at the bottom of the tube
4.	Loaded samples into the 2200 TapeStation

### Results

```{r, echo=FALSE, out.width ='100%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/tapestation/2024-10-10 - PCR2 SAMPLES 1-8 RTX RAMOS TS RE-RUN.pdf"

# Insert the image
knitr::include_graphics(image_path)
```

# Friday 11-10-2024

## RTX DP - PCR2: Repeat Samples 1 + 6
### Overview
- PCR2 for all samples using PCR1 product post cleanup
- PCR to index samples for sequencing by attaching UMI 
- VERY IMPORTANT to accurately record which samples receive which adaptors  
  *[Guide for Index Adaptor Pooling](https://support-docs.illumina.com/SHARE/IndexAdaptersPooling/Content/SHARE/FrontPages/IndexAdapterPooling.htm) 

### Materials

|Component       |Expected Volume/experiment| Material ID |
|:--------------|----|-----------------|:-------------:|
| 10uM Nxxx Nextera i7 adapter  | - |   -       |
| 10uM Sxxx Nextera i5 adapter | - |     -     |
| DNAse/RNAse H20 | 1000 $\mu$L |          |
| Q5 | 1200 $\mu$L |  M0492S  |

### Protocol
1. Prepare samples according to the following table:
  - Keep careful note of which adaptors are added to which sample

|Component       |Volume|
|:--------------|----|
| 10uM Nxxx Nextera i7 adapter  | 1.5 $\mu$L |
|10uM Sxxx Nextera i5 adapter | 1.5 $\mu$L |
| DNAse/RNAse H20      | 21 $\mu$L |
| Kapa Hifi HotStart Ready Mix (2X) | 25 $\mu$L |
| 10ng/$\mu$L PCR1  | 1 $\mu$L | 


2. Perform PCR using the following parameters:  

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 55C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 2 min |
Hold Step |Step 6: 12C    | Hold |  

\begin{center}
\textbf{8 cycles of steps 2-4}
\end{center}

3. Perform Ampure bead clean up as above with 90ul beads per 50ul PCR reaction.
- If you don’t have a crazy number of samples you might also want to consider doing up to 4 PCR2 reactions per sample and pooling them afterwards just to make sure all your sequences are captured.

4. Check size and concentration on tapestation. Dilute and pool samples for sequencing.

## RTX DP - Bead Clean Up 2: Repeat Samples 1 + 6
### Overview
- Clean PCR reaction by removing leftover primers
- This prevents false reads, cleaner indexing in later steps, and higher quality samples

### Materials
- For 40 samples

|Component       |Expected Volume/experiment| Material ID |
|:--------------:|:----:|:---------:|
| Ampure Bead Kit | 200 $\mu$L | |

### Protocol
1.	Equilibrate Ampure beads to room temp (available from supplies cold room)
2.	Mix well and vortex for 30sec to ensure uniform distribution.
3.	Add 90ul (1.8X volume) to PCR reaction and mix thoroughly by pipetting.
4.	Incubate 5min, room temp
5.	Place on magnet for 2 min and keep on magnet until final elution
6.	Remove supernatant and add 200ul 70% ethanol (make fresh)
7.	Incubate 30sec and remove
8.	Repeat 70% ethanol wash
9.	Air dry beads 5-10min room temp. Once dry the beads will go from shiny to matt in appearance. Avoid over drying (cracked appearance).
10.	Add 15 ul EB buffer and resuspend beads by pipetting
11.	Incubate 2min room temp
12.	Place on magnet and remove eluate
13. Assess PCR product size, contamination and concentration on tapestation.   
**Expected size = 165-172bp**

## RTX DP - Tapestation: Repeat Samples 1 + 6
### Overview
- Run automated gel to check bandsize/location

### Materials

|Component       |Expected Volume/experiment| Material ID |
|:--------------:|:----:|:---------:|
| DS |  | |

### Protocol
1.	Added 2$\mu$L High Sensitvity D1000 Sample Buffer with 2$\mu$L DNA sample
2.	Spun down, then vortexed using IKA vortexer and adaptor at 2000rpm for 1 min
3.	Spun down to position sample at the bottom of the tube
4.	Loaded samples into the 2200 TapeStation

### Results

```{r, echo=FALSE, out.width ='100%', fig.align='center'}

# Path to your image file
image_path <- "~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/tapestation/2024-10-11 - PCR2 SAMPLES 1 + 6 RTX RAMOS TS RE-RUN.pdf"

# Insert the image
knitr::include_graphics(image_path)
```

# Monday 11-10-2024

## Qubit - RTX DP Samples 1-47

### Overview
- Quantify DNA concentration to submit library for sequencing
  
### Protocol
  1. Prepare Qubit tubes  
    - 2 tubes for standards  
    - 47 sample tubes  
  2. Add solutions in following amounts:
  
```{r, echo=FALSE, fig.align='left'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Qubit_protocol.xlsx", sheet = "Sheet1")

# Display the data as a table
knitr::kable(excel_data, digits = 1)

```
  3. Vortex for 3-5 seconds
  4. Allow tubes to incubate at RT for 2min and read samples


### Results

```{r, echo=FALSE, fig.align='left'}

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Lab Notebook/Data/Library_prep_tables/RTX_DP_LP_QUBIT.xlsx", sheet = "Sheet1")

# Display the data as a table
knitr::kable(excel_data, digits = 1)

```

## MR - Cell Culture

### MR - Making Media

RPMI:  
10% FBS = 56ml
1% Pen Strep = 5.6ml
1% Glutamax = 5.6ml

### MR - Thawing cells: RAMOS

Overview: BC we need cells to start the EC50

Protocol:  
1. Get cells from the liquid nitrogen- ask Callum, then place in ice box to thaw  
2. Put 1ml media into the cryo vial and mix slowly  
3. Take the cells and place into red capped tube  
4. Put 6ml media into tube slowly down side of tube so dont rupture cells  
5. Count cells- see below  
6. Spin down the tube of cells at 600rpm = 74 rcf, for 5 mins- get pellet   
7. Tip out the supernatant into waste tip thing  
8. Flick to dislodge pellet  
9. Add 12ml media, mix  
10. Put into flask, label flask  
11. Check under microscope - if confluent then add another 8ml media to flask  



Counting cells:  
a) Take 10 ul cells and 10ul trypan blue into the tiny tube  
b) Put glass slide ontop of the haemocytometer, then pipette the blue cells in via the top  
c) Under microscope: count live cells in the 16 middle squares with counter  
d) Number of cells = Counted number x 20,000  
e) Clean the slide  

OR to count cells:  
Use the countess- number of live cells


10/10/24 Number of cells = 780 000

# Monday 14-10-2024

## EC50 protocol

### A. Add 200ul PBS to the outside wells using multi channel

### B. Seed 10,000 cells per well

1. Count cells  
2. Dilute to 10,000 cells per well: 10,000 cells per 100ul = 1,000,000 cells per ml, for 4ml total (if not enough cells, then work out the correct number of cells, spin it down and then resuspend in the volume you need)  
3. Add 100ul (or 50ul for RTX) of the cells to each well  

### C. Make up conc of drugs

1. Add 500ul media to tubes labelled 2-9  
2. Into tube 1 add drug (20ul Methotrexate / 6.6ul vincristine / 3.88ul RTX) plus (1ml - ul of drug) Dox: 30uM, so need 60uM in 1ml media = stock 
3. take 500ul of that and add to tube 2, mix  
4. repeat for serial dilution- should end up with 1ml in tube 9  

### D. Add drugs to wells and let sit for 72 hours 

1. Take 500ul from each tube, including just 500ul media for the first control well, and put into multi channel in the order: C, 9,8,7,6,5,4,3,2,1  
2. Reverse pipette a little then add 100ul (or 25ul for RTX) drug to the wells x3 rows of each  
3. For RTX: Add 25ul serum (HIAS for the top three and NHS for the bottom three rows)


## Set up the EC50 for Methotrexate and vincristine- but seeded too many cells



# Tuesday 15-10-2024
## Set up EC50 for RTX- but added 50ul drug rather than 25ul


# Thursday 17-10-2024
## EC50 Analysis

0. Check cells under microscope  
1. Add 20ul cell titre blue from aliquotted vials to each well- must be in the dark!    
2. Incubate for 2 hrs  
3. Use plate reader (as below)    
4. Analyse data and plot using R to find EC50 value  

## Using the plate reader

a) Bring the plate covered in paper to keep in the dark  
b) Open SoftMax pro 6.3  
c) Protocols- manager- library- cell growth and viability - cell titre blue fluorescence  
d) New plate if 2 plates  
e) Open machine making sure row A of the plate is at the top (correct orientation of the plate)  
f) Close 
g) Read, enter  
h) Plate 2- repeat steps e) to g)  
i) Open excel- copy and paste data across- label sheets with drug names etc- put onto USB stick- SAVE  
j) Save the data also onto the laptop- into Callum's folder  


# Monday 21-10-2024

## Growth Curve- using a 6 well plate
1. For 3 wells: add 100,000 cells of RTX naive cells (100ul today) +3ml media
2. For the other three wells: add 100,000 cells of RTX resistant cells (170ul today) +3ml media
3. Then split cells



## EC50 seeding
### To add to tube 1: 
Methotrexate: 2ul, Vincristine: 6.66ul, Cyclophosphamide: 1.5ul, Doxorubicin: 10ul

# Tuesday 22-10-2024

## Counted growth plates for the 100,000 cell plate (naive average of 3 cells, then x20,000. RTX resistant average of 1 cell x20,000)
## Plated a new growth plate with 200,000 cells instead
## Plated two RTX EC50 plates: 1 with Ramos naive cells, and one with RTX resistant cells, both with HIAS on top and NHS on bottom



# Thursday 24-10-2024
## Counted growth plates x2

## FACs: 
- Overview: In order to see if CD20 does get downregulated in RTX resistant cells or not?

### Protocol
1. Count cells  
2. Make stain solution: 2ul stain for every 500ul PBS  
3. Add 200-300ul cells to 3 wells  
4. Spin down the plate at 1500rpm for 2 mins  
5. Flick out supernatant  
6. Take 100ul of the stain solution and add to the bottom of the three wells, mix, then add to the second well, mix, then add all of it to the top well and mix  
7. Incubate in fridge at 4 degrees for 25 mins  
8. Spin down again  
9. Resuspend cells in PBS  
10. Label FACs tubes and add the cells into them  
11. spin down some control unstained cells and then resuspend in PBS and then put into a FACs tube labelled too  

### FACs machine:
1. Live cells (FSC-A vs SSC-A)  
2. FSC-H to get singlets only  
3. 450/50 laser for the BV421 flurorochrome used to bind the CD20- adjust so cells below 10^3- to see CD20+ cells to the right of this  
4. 

#### FACs analysis: (floreader.io)
1. Use Poly tool to outline the population- gate called 'Live cells'- then drag this gate down to the pipeline under 'All files'  
2. Go into the 'live cells' gate- then in the plot of FSC-H (y axis) VS FSC-A (x axis)- us the Poly tool to capture this population- call the gate 'singlets' - then pull down into pipeline under the 'live cells' one  
3. Go into the 'singlets' gate- then in a plot of SSC-A (y axis) VS violet laser (x axis), use the Poly tool to draw a square to the right of the population seen = 'CD20+ cells' gate- drag down into pipeline under the 'singlets' one


# Monday 28-10-24
## Seeded a 6 well plate (500,000 cells/ well)
3 wells control Ramos (1.5ml) +media (1.5ml), and 3 wells Ramos (1.5ml) +RTX (1.5ml)

## Seeded two EC50 chemotherapy plates this time with lower conc of drugs
Meth: 1uM
Vin: 20nM
CycloP: 50uM
Dox: 1uM



# Tuesday 29-10-2024

- Seeded two RTX EC50 plates: one with naive and one with resistant cells 
  * 10,000 cells per well
  * 7.76ul RTX in 1ml media for serial dilution tube 1


# Thursday 31-10-24
- Analysed the 4 EC50 plates
- Did the FACs from the 6 well plate and Floread.io 


# Monday 04/11/2024
- Seeded the two drug plates again with new concs (Meth: 5uM, Vin: 20nM, CycloP: 400uM, Dox:1uM)


# Tuesday 05/11/2024
- Seeded the RTX EC50 plates x2

# Thursday 07/11/2024
- Analysed the 4 EC50 plates 

# Tuesday 12/11/2024
- Seeded the 4 EC50 plates again (2x chemo and 2x RTX plates)
- Seeded the epigenetics test 384 well plate to find the Z value
- Seeded the first Dox VS RTX synergy screen plate

# Thursday 14/11/2024
 - Analysed the 4x EC50s, 4x synergy screen plates and 1x 384 well Z value plate


# Tuesday 19/11/2023
## MR - Cell Culture
- Seeded three more EC50 plates but needed to use new stocks for all of them 
  * 2x Meth @5uM 
  * 2xCycloP @1uM (THIS IS DIFFERENT FROM BEFORE)
  * 1x Vin @20nm 
  * 1x Dox @1uM
-  Seeded Vin synergy screen plate


# Thursday 21/11/2024
## MR - Read EC50 Plates
 Analysed the 3x EC50 plates and the 2x Vin synergy plates and plotted in R
## MR - Cell Culture
- Split the cells into larger flask ready for epigenetic screen on Tuesday


# Tuesday 03/12/2024

## MR - Cell Culture
 - Seeded the methotrexate VS RTX synergy screen plate
 - Seeded and did the robot for the epigenetic drug screen plates (three plates of naive and three plates of resistant Ramos)

# Thursday 05/12/2024
## Read the 2x methotrexate synergy screen plates
## Read the 6x epigenetic drug screen plates




