---
title: "Experimental Plan"
author: "Callum Malcolm"
date: "2023-12-07"
output: pdf_document
toc: true
toc_float: true
urlcolor: blue

---

\newpage

# Thesis Plan

## Chapter 1: Clinical Data

### C1.1 Analysis of Immune cell make up of publically avaialble RNAseq BL Datasets
 - CIBERSORT
  * Immune profile
 - ECOTYPER
  * COO analysis
 - GSEA analysis of samples with unique immune profile(?)
 
###C1.2 Spatial Transcriptomics of sBL Clinical Samples
- 4 samples (maybe 6)
- Initial IHC:
  * H+E
  * CD20
  * CD10
  * Ki-67
  * MYC
- Genomic Analysis of MYC status
  * FISH
- ST
  * 10x Visium
  * Focus on Immune profile subsets
  
###C1.3 Mibiscope
- Identify proteins of interest from 1.1/1.2


## Chapter 2: Rituximab Resistance in BL PDX  

- Overview: looking at mechanisms of Rituximab resistance in BL PDX
  * CDC: Complement Dependent Cytoxicity
  * ADCC: Antibody-Dependent Ccellular Cytoxicity
- Use N2 and RBL2P

- Can incorporate publicially available dataset to compare Rituximab treatment vs not
  * refractory 

### Experiment 1: Establish Rituximab-related CDC in PDX  

Overview: Are PDX susceptible to RTX-dependent CDC?
- Previously described protocol:
  * Paper: [Analysis of changes in CD20, CD55, and CD59 expression on established rituximab-resistant B-lymphoma cell lines] (Analysis of changes in CD20, CD55, and CD59 expression on established rituximab-resistant B-lymphoma cell lines)

Naive RAMOS cells (RAMOS-N) were simultaneously exposed to the several concentrations of rituximab and human serum, and then after 1-h incubation at 37 Â°C, the percentage of lysed cells was measured. RAMOS-N cells were found to be lysed in the presence of rituximab and serum, and the percentage of lysed cells increased gradually in a dose-dependent manner. These results confirmed that RAMOS-N cells were sensitive to rituximab-related CDC and that the lysis ratio increased according to the concentration of rituximab and serum.

Outcome: Cell Lysis curves for RTX-Dependent CDC in PDX

### Experiment 2: Establishing RTX-resistant PDX

Overview: Can we generate PDX lines resistant to RTX?
- Protocol found in paper above
- Incorporate WILDseq to look at the change in clonal dynamics

Outcome: Validation of RTX-Resistant Cells Lines

### Experiment 3: Analyse changes in CD20
Overview: How does CD20 expression dynamics change in PDX-N vs PDX-R?
- Flow for CD20 (CD19 Control)
- qPCR for CD20 expression

### Experiment 4: Co-inhibitor Testing
Overview: Do co-inhibtors found in the literature promote CDC in PDX-R?

- Midostaurin: PI3K/AKT pathway
- CD59 inhibitor
- PIM1 inhibition

### Experiment 5: ADCC Testing

Overview: Does RTX CDC resistance confer ADCC resistance?




sBL PDX in Humanised Mice

### NK-92 and Rituximab
- 2x PDX?
- WILDseq GFP+ test
- Co-Culture "EC50"
  * Use GFP+ as a proxy for cell viability
  * +/- Rituximab (other Glo-BNHL treatments?)
  * You need to think about this
- Long term co-culture
  * Sort
  * RNAseq
  * Proteomics (?)
  * WILDseq Clonal Selection (?)

### C2.1 Humanised vs NSG Pilot
- IHC
- Flow 





## Project 1: Spatial Transciptomics of BL

11 Samples of BL acquired from VIVO BioBank  
- Want to do stuff similar to these studies:  
  * [PDAC Spatial Transcriptomics](https://www.nature.com/articles/s41588-022-01157-1)  
  * [Genomic/Transcriptomic analysis of Clinical BL Samples](https://www.nature.com/articles/s41467-019-08578-3)
- Whole Genome, whole exome sequencing
- Patient demographics?
- Survival outcomes?
  
### Tools  

- [NCI GDC data portal](https://portal.gdc.cancer.gov/analysis_page?app=)
  * Compare gene expression, mutations, survial analytics(?)
- Cibersortx
  * Look at predicted immune populations
- SRA
  * Publicly available BL RNAseq datasets
- GEO
  * Gene expression omnibus
- BLGSP
  * BL genome repository (need access)
- ICGC
  * BL genome repository (need access)
  * ICGC 25k argos(?) should have BL cases

- DepMap
  * Cancer dependency on various genes (skews metabolic in terms of relevant output)
- GTEx
  * Adult Genotype-Tissue Expression (GTEx) project
  * 

### Paper Plan
Based on PDAC ST paper

- Panel 1: 
  * Patient Data
    a. Correlation survival studies between immune cell infiltration and survival

- Panel 1:
  * Patient sample characterisation
    a. Work Flow (sample backgrounds, collection from sources)
    b. Overview of all cell types profiled in the scRNA-seq cohort.
    c. Overview of spatial transcriptomics cohort
- Panel 2
  * Characterize the tumour (Tumor vs stroma vs immune cells)
    a. Differential pathway enrichment case-level tumor subpopulations
    b. Tumor cluster pathway enrichment for specific case
    c. Tumor cluster pathway enrichment for another specific case
    d. UMAP of tumor subclusters for specific case
    e. UMAP of tumor subclusters for specific case
    
- Panel 3
  * Genomic Landscape and oncogenic driver heterogeneity (??)
  
- Panel 4
  * Tumour cell heterogeneity in ST data(?)

- Panel 5
  * Immune Populations in TME

- Panel 6
  * Mibiscope

## Thesis ID
Looking at characterizing the Human BL TME  
- Publically available BL Data   
- VISIUM internal samples   
- Mibiscope Internal Samples  

Publically available datasets analysis  
- Talk to Jamie again

- 10x Visium transcriptomics
  * T Cell Dynamics
    - Infiltration
    - Exhaustion
  * NK cell dynamics
  * Places to get it done:
    - CI (no)
    - [Source Bioscience](https://sourcebioscience.com/genomics/10x-visium-spatial-transcriptomics/)
    - Wellcome Sanger Institute
      * Teichmann Lab used 10x Visium
  
- MIBIScope Comparison
  * Does Protein expression map onto spatial gene expression?
  * Collaborate with Nina in Germany
 
## Publically Available RNAseq  

1. The iDEP or TACITuS pipeline for transcriptomic profiling and GSEA/pathway analysis.   
2. The CIBERSORT immune-cell deconvolution algorithm to obtain cell-fractions and gene expression/enrichment scores of tumor-infiltrating lymphocytes.   
3. The GEPIA2 database for survival analysis correlating to immune infiltration and/or differential gene expression.   



# Ideas

## Rituximab Resistance

### Generate Rituximab Resistant PDX
- Resistance to Rituximab-CDC
- Resistance to Rituximab-ADCC (NK cell dependent)
- Apply barcoding approach to try to identify signalling pathways which are depended on for evasion

#### RTX CDC Resistance
1. RTX CDC assay
- Change concentrations of RTX
- Change concentrations of serum
2. RTX Resistance
- Take barcode sample at Day 0, resistance 1, resistance 2, resistance 3, ect. 
  * Viably freeze for scRNAseq?
  *Cell pellet for bulk rna
- Increase concentrations
- Monitor by CDC assay
3. In vivo
- Compare resistant to non-resistant
- Barcode Experiment 
- In vivo CRISPR screen

Jamie's plan:




#### RTX NK-92 Assay



## BL based Macrophage polarisation

### Co-culture experiement


### Mosaic Macropahge engineering
- Activated by specific pathways which disengage them




## Humanised Mice PDX 

### Incorporate Barcodes?
- This is very undercooked

## Omo-MYC
- Waiting for regualtory approval

## (Shelved) A20 Modelling the BL Tumour Microenviroment

### Overview
- Using A20 to model BL immune TME
  * Look specifically at T-cell infiltration
  * Immune evasion mechanisms deployed by BL 
- Compare results to patient data
- Apply findings to humanised mouse models of BL PDX

#### Issues
- What is the Biological/clinical relevance of this model?
- Are you priming an infiltrating response due to the nature of cell injections causing necrosis (thereby stimulating the immune system)

#### Background
- The impact of immune TME in BL is unclear
- BL 

### Experimental Plan

#### E1: NSG vs BALB/C A20 injection
##### E1 Overview
- Compare between immunocompetent and immunocomprimised mice
- Basic actors to compare
  * Tumour growth rate
  * Tumour size

##### Injection Plan

| Group | Strain  |Location | Cell Injection| Cell Number|
|:--------:|:----:|:----:|:-------:|
| 1 | BALB/c | Sub-cut | A20 | 1x10^6^ |
| 2 |BALB/c  | IP | A20 | 1x10^6^ |
| 3 | NSG  | IP | A20 | 1x10^6^ |
| 4 | NSG  | Sub-cut | A20 | 1x10^6^ |

  
##### IHC Panel
- Compare markers between tumour types
- Burkitt IHC indentification:
  * CD10+ (B-Cell Germinal Centre)
  * Bcl-2-
  * Ki-67%hi (proliferation index)
  
  Potential IHC Panel Markers
  
| Cell Type | Marker | 
| :-------: | :------: | 
| Proliferation marker | Ki67 |
| B cell | CD20 |
|   T cells (all)   | CD3 |
| T Cells (cytotoxic) | CD8 |
| T cells (helper) | CD 4 |
| Dendritic Cells | CD11c |
| Macrophage | F4/80 |
| NK (?)| CD56 (?) | 

##### Flow Panel
- Options are:
  * Standard T Cell
  * TRegs
  * B-cells
  * DC Mono CD11c
  * Th17
  * TfH
- Check what Swetha ordered
* Box 7 antibody sheet dropbox
  
 Standard T Cell
  
| Cell Type | Marker | 
| :-------: | :------: | 
| T Memory | CXCR3 |
| Naive Immune Cells | CCR7 |
|   Naive T Cells   | CD45RA |
| Th17 | CCR6 |
| GC B-Cells | CD38 |
| TRegs (Helper) | CD4 |
| Macrophage | HLA-DR |
|  T Cells (all)   | CD3 |
| T Cells (Cytoxic)  | CD8 |

#### E2: BL Therapies (this will not work)
- Balb/c drug vs no drug vs WS-A20 drug vs WS-A20 no drug
  * Want to compare effect of GFP on immune cell infiltration
  * Look at clonal dynamics of rituximab treatment
- Glo-BNHL Trial
  * Odronextamab
    - CD20xCD3 Bispecific antibody
  * Loncastuximab tesirine
    - 
  * Rituximab
- CAR treatments?
- Bi-Specific antibodies (check that they can be applied to mice)
- Rituximab comparison
  * Clonal dynamics of Rituximab treatment
    - WILDseq


#### E3: Immune focused CRISPR Screen
- Immune compromised vs Immune competent
  * Think very carefully about specific mouse models (some still have macrophages, NK cells, ect.)
  * [JAX Lab Article](https://www.jax.org/news-and-insights/2006/March/choosing-an-immunodeficient-mouse-model#:~:text=Immunodeficient%20mouse%20models%2C%20particularly%20severe,cell%20biology%20and%20other%20research.)


## Ferritin as a drug delivery system
- Does BL have increased Tfr1 expression relative to normal cells/B cells?


\newpage

|Sample Number| EBV Status | CD20 | CD10 | MYC Translocation |
| :-------: | :------: | :------: | :--------------: | :----: |
|   | 5 Negative |           |             | 6 Positive          |
| 7 | 1 Positive | 7 Positive| 7 Positive  |  1 Not Conclusive |
|   | 1 Not tested |        |    |         |                    | 





