---
title: "Lab_Notebook_2024"
author: "Callum Malcolm"
output:
  pdf_document: default
toc: true
toc_float: true
urlcolor: blue
---
\newpage

# \textbf January 2024

# Tuesday 16-01-2024

## WILDseq - Mouse Experiment 1: PCR 1 Samples 1-8 / PCR 2 Sample 5
### Overview: 
- PCR tests to see if PCR would work for WS-ME1 cDNA previously made  
- Started with samples 1-8 so as not to waste sample/reagents if PCR 1 did not work

#### Sample List
| Sample ID | Treatment/Sample | ng/uL | i7 index | i5 index |
| :-----: | :---: | :----:|:-----:|:-----:| :-----:|
| 1 | Cyclophosphamide | 773.7 | N701 | S502 |
| 2 | Cyclophosphamide  | 545.3 | N702 | S502 |
| 3 | Cyclophosphamide | 903.6  | N703 | S502 |
| 4 | Cyclophosphamide | 1056.1 |N704 | S502 |
| 5 | Cyclophosphamide | 959.1 |N705 | S502 |
| 6 | Combination | 730.1 | N706 | S502 |
| 7 | Combination | 602.3 | N707 | S502 |
| 8 | Combination | 449.6  | N710 | S502 |
| 9 | Combination| 1002.1 | N701 | S503 |
| 10 | Combination | 1929.2 | N702 | S503 |
| 11| Methotrexate | 861.5 | N703 | S503 |
| 12 | Methotrexate | 1110.4  | N704 | S503 |
| 13 | Methotrexate | 1171.0 | N705 | S503 |
| 14 | Methotrexate  | 1347.8 | N706 | S503 |
| 15 | Methotrexate  | 891.4  | N707 | S503 |
| 16 | Vehicle | 374.5 | N710 | S503 |
| 17 | Vehicle | 911.2 | N701 | S505 |
| 18 | Vehicle | 829.8 | N702 | S505 |
| 19 | Vehicle | 600.1  | N703 | S505 |
| 20 | Vehicle | 750.3 | N704 | S505 |
| 21 | Baseline | 401.5 | N705 | S505 |
| 22 | Baseline | 443.3 | N706 | S505 |
| 23 | Baseline  | 373.4  | N707 | S505 |
| 24 | Baseline | 444.5  | N710 | S505 |
| 25 | Baseline | 267.6 | N701 | S506 |
| 26 | BLLW 14K | 378.7  | N702 | S506 |
| 27 | BLLW 2K | 362.7  | N703 | S506 |
| 28 | BLLW 1K | 563.8  | N704 | S506|
| 29 | Methotrexate (outlier) | 348.1 | N705 | S506|
| 30 | Combo | 380.8 | N706 | S506 |
| 31 | RBL2P 2K | 173.3 | N707 | S506 |
| 32 | RBL2P 7K | 2708.0 | N7010 | S506 |
| 33 | RBL2P 250K | 1418.5 | N701 | S507 |
| 34 |Mock_direct_1 | - | N702 | S507 |
| 35 | Mock_direct_2 | -  | N703 | S507 |
| 36 | Mock_direct_3 | - | N704 | S507 |
| 37 | Mock_culture_1 | - | N705 | S507 |
| 38 | Mock_culture_2 | - | N706 | S507 |
| 39 | Mock_culture_3 | - | N707 | S507 |

### PCR 1: Samples 1-8
- Checked [cDNA] on NanoDrop of sample 8 to make sure samples were still intact
  * Sample 8 ng/$\mu$L = 1042.3
  * It is assumed all other cDNA is of similar quality

1. Made a master mix of PCR1 reagents 
- Made enough for 31 samples
- Primer mix was made earlier

|Component       |Volume| Master Mix |
|:--------------:|:----:|:-------:|
| 10uM WS PCR1 Primer Mix | 3 $\mu$L | 93 $\mu$L |
| DNAse/RNAse H20      | 12 $\mu$L | 372 $\mu$L |
| Kapa Hifi HotStart Ready Mix (2X) | 25 $\mu$L | 775 $\mu$L |

2. Add following components to tubes

|Component       |Volume|
|:--------------:|:----:|
| PCR1 MasterMix | 40 $\mu$L | 
| cDNA      | 10 $\mu$L |

3. Performed PCR using the following parameters: 

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 60C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 1 min |
Hold Step |Step 6: 12C    | Hold |  

\center __10-25 cycles of steps 2-4__

\raggedright
### Bead Clean Up
#### Overview
- Clean PCR reaction by removing leftover primers
- This prevents false reads, cleaner indexing in later steps, and higher quality samples

#### Materials
- For 40 samples

|Component       |Expected Volume/experiment| Material ID |
|:--------------:|:----:|:---------:|
| Ampure Bead Kit | 200 $\mu$L | |

#### Protocol
1.	Equilibrate Ampure beads to room temp (available from supplies cold room)
2.	Mix well and vortex for 30sec to ensure uniform distribution.
3.	Add 90ul (1.8X volume) to PCR reaction and mix thoroughly by pipetting.
4.	Incubate 5min, room temp
5.	Place on magnet for 2 min and keep on magnet until final elution
6.	Remove supernatant and add 200ul 70% ethanol (make fresh)
7.	Incubate 30sec and remove
8.	Repeat 70% ethanol wash
9.	Air dry beads 5-10min room temp. Once dry the beads will go from shiny to matt in appearance. Avoid over drying (cracked appearance).
10.	Add 15 ul EB buffer and resuspend beads by pipetting
11.	Incubate 2min room temp
12.	Place on magnet and remove eluate
13. Assess PCR product size, contamination and concentration on tapestation.   
**Expected size = 165-172bp**

#### Tapestation PCR 1 - Samples 1-8

![2024-01-16 Tapestation PCR 1 Results Samples 1-8](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-16 TS 1.png)
\center
Initial Test PCR 1
\raggedright

### PCR 2: Samples 5

#### Overview
- Yield from PCR1 is low (.09 - .631 ng/$\mu$L)
- Sample with lowest concentration was taken ahead to PCR 2 (E1 - Sample 5)
- PCR to index samples for sequencing by attaching UMI 

#### Materials

|Component       |Expected Volume/experiment| Material ID |
|:--------------|----|-----------------|:-------------:|
| 10uM Nxxx Nextera i7 adapter  | - |   -       |
| 10uM Sxxx Nextera i5 adapter | - |     -     |
| DNAse/RNAse H20 | 1000 $\mu$L |          |
| Kapa Hifi HotStart Ready Mix (2X) | 1200 $\mu$L |  KK2601  |


#### Protocol  
1. Prepared samples according to the following table:
  - Sample - Adaptor pairs in table above

|Component       |Volume|
|:--------------|----|
| 10uM N705 Nextera i7 adapter  | 1.5 $\mu$L |
|10uM S502 Nextera i5 adapter | 1.5 $\mu$L |
| DNAse/RNAse H20      | 21 $\mu$L |
| Kapa Hifi HotStart Ready Mix (2X) | 25 $\mu$L |
| 10ng/$\mu$L PCR1  | 1 $\mu$L | 


2. Performed PCR using the following parameters:  

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 55C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 2 min |
Hold Step |Step 6: 12C    | Hold |  
\center
 __8 cycles of steps 2-4__ 
\raggedright

3. Performed Ampure bead clean up as above with 90ul beads per 50ul PCR reaction

4. Checked size and concentration on tapestation

#### Tapestation PCR 2
![2024-01-16 Tapestation PCR 2 Test Samples 5](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-16 TS 2.png)


#### Review
- Contamination/PCR bubble?
- Yield is adequate (assuming it is representative of the correct peak)
- Chris wants to repeat PCR1 for remaining samples, take PCR2 ahead, and re-run sames with D1000 tape (non High-Sensitivity)

#### Plan for tomorrow
- PCR 1 samples 9-30
- Bead Clean Up
- Tape Station

# Wednesday 17-01-2024

## WILDseq - Mouse Experiment 1: PCR 1 Samples 9-30

### Overview: 
- Completed PCR 1 for Samples 9-30

### PCR 1: Samples 9-30

1. Used master mix of PCR1 reagents made 2024-01-16 

2. Add following components to tubes
|Component       |Volume|
|:--------------|----|
| PCR1 MasterMix | 40 $\mu$L | 
| cDNA      | 10 $\mu$L |

3. Performed PCR using the following parameters: 

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 60C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 1 min |
Hold Step |Step 6: 12C    | Hold |  

\center
__10-25 cycles of steps 2-4__
\raggedright

### Bead Clean Up
#### Overview
- Clean PCR reaction by removing leftover primers
- This prevents false reads, cleaner indexing in later steps, and higher quality samples

#### Materials
- For 40 samples

|Component       |Expected Volume/experiment| Material ID |
|:--------------:|:----:|:---------:|
| Ampure Bead Kit | 200 $\mu$L | |

#### Protocol
1.	Equilibrate Ampure beads to room temp (available from supplies cold room)
2.	Mix well and vortex for 30sec to ensure uniform distribution.
3.	Add 90ul (1.8X volume) to PCR reaction and mix thoroughly by pipetting.
4.	Incubate 5min, room temp
5.	Place on magnet for 2 min and keep on magnet until final elution
6.	Remove supernatant and add 200ul 70% ethanol (make fresh)
7.	Incubate 30sec and remove
8.	Repeat 70% ethanol wash
9.	Air dry beads 5-10min room temp. Once dry the beads will go from shiny to matt in appearance. Avoid over drying (cracked appearance).
10.	Add 15 ul EB buffer and resuspend beads by pipetting
11.	Incubate 2min room temp
12.	Place on magnet and remove eluate
13. Assess PCR product size, contamination and concentration on tapestation.   
**Expected size = 165-172bp**

#### Tapestation PCR 1
![2024-01-17 Tapestation PCR 1 Samples 9-16](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-17 TS 1.png)

#### Tapestation PCR 1
![2024-01-17 Tapestation PCR 1 Samples 17-30](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-17 TS 2.png)
\\

#### Review
- Samples 10/17/18/29 don't have adequate yield
  * Will need to redo PCR 1
  * Sample 10 has acceptable PCR1 sample from previous run
- Yield is adequate for the rest to move ahead

#### Plan for tomorrow
- PCR 1 samples 17/18/29
  * Bead Clean Up
  * Tape Station
- PCR 2 for all samples
  * Double check reagents amounts
  * Running low on KAPA (re-ordered) and Nextera Primers
- Chris wants to add 2 additional samples
  * To Do - RNA Extraction / RT / PCR 1 / PCR 2
  * Wait until majority of samples have acceptable PCR 2 product
  
# Thursday 18-01-2024

## WILDseq - Mouse Experiment 1: PCR 1 Samples 17/18/29

### Overview: 
- Repeated PCR 1 for Samples 17/18/29

### PCR 1: Samples 17/18/29

1. Used master mix of PCR1 reagents made 2024-01-16 

2. Add following components to tubes
|Component       |Volume|
|:--------------:|:----:|
| PCR1 MasterMix | 40 $\mu$L | 
| cDNA      | 10 $\mu$L |

3. Performed PCR using the following parameters: 

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 60C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 1 min |
Hold Step |Step 6: 12C    | Hold | 
\center
 __10-25 cycles of steps 2-4__ 
\raggedright

### Bead Clean Up
#### Overview
- Clean PCR reaction by removing leftover primers
- This prevents false reads, cleaner indexing in later steps, and higher quality samples

#### Materials
- For 40 samples

|Component       |Expected Volume/experiment| Material ID |
|:--------------:|:----:|:---------:|
| Ampure Bead Kit | 200 $\mu$L | |

#### Protocol
1.	Equilibrate Ampure beads to room temp (available from supplies cold room)
2.	Mix well and vortex for 30sec to ensure uniform distribution.
3.	Add 90ul (1.8X volume) to PCR reaction and mix thoroughly by pipetting.
4.	Incubate 5min, room temp
5.	Place on magnet for 2 min and keep on magnet until final elution
6.	Remove supernatant and add 200ul 70% ethanol (make fresh)
7.	Incubate 30sec and remove
8.	Repeat 70% ethanol wash
9.	Air dry beads 5-10min room temp. Once dry the beads will go from shiny to matt in appearance. Avoid over drying (cracked appearance).
10.	Add 15 ul EB buffer and resuspend beads by pipetting
11.	Incubate 2min room temp
12.	Place on magnet and remove eluate
13. Assess PCR product size, contamination and concentration on tapestation.   
**Expected size = 165-172bp**

#### Tapestation PCR 1


#### Review
- Samples 10/17/18/29 don't have adequate yield
  * Will need to redo PCR 1
  * Sample 10 has acceptable PCR1 sample from previous run
- Yield is adequate for the rest to move ahead

#### Plan for tomorrow
- PCR 1 samples 17/18/29
  * Bead Clean Up
  * Tape Station
- PCR 2 for all samples
  * Double check reagents amounts
  * Running low on KAPA (re-ordered) and Nextera Primers
- Chris wants to add 2 additional samples
  * To Do - RNA Extraction / RT / PCR 1 / PCR 2
  * Wait until majority of samples have acceptable PCR 2 product
  
# Friday 19-01-2024

## WILDseq - Mouse Experiment 1: RNA Extraction/RT Samples 31-33
Overview: Extra samples given by Chris for whitelist

### RNA extraction Samples 31-33
- Performed RNA extraction of tissue/cell samples according to the following kit:

### RT Protocol Samples 31-33
1. In PCR strip tubes, prepared 5 $\mu$g of RNA in a total volume of 10 $\mu$l of RNAse/DNAase-free water.
2. Add 1 $\mu$l of WS_RT_UMI_NexteraR2 primer (2$\mu$M stock)
3. Add 1 $\mu$l dNTPs (10 mM each). 
4. Denature at 65 C for 5 mins in the PCR machine, then straight onto ice for at least 2 mins, spin briefly to get liquid to the bottom of the tube.
5. Used Master Mix previously made
6. Add 7 $\mu$l of RT MM prepared above to each sample and mix, spin briefly to get liquid to the bottom of the tube.
   - Spin briefly to get liquid to bottom of the tube 
7. In the PCR machine incubate at 53 C for 10 mins followed by 80 C for 10 mins.
8. Add 1 $\mu$l Themolabile Exonuclease I (NEB M0568) to remove excess RT primer.

|Component       |Volume|   MM volume  |
|:--------------|----|-----------------|
| Thermolabile Exonuclease I  | 1 $\mu$L |          |
| NEBuffer r3.1*        | 2 $\mu$L |          |

 **Most PCR buffers are compatible**

9. In PCR Machine: heat at 37C for 4 mins followed by 80C for 1 min  
    - Spin briefly to get liquid to bottom of the tube
10. Add 1 $\mu$l of RNAse H and incubate at 37 C for 20 mins.
11. Checked [cDNA] on NanoDrop
  - Added to Sample List

# Monday 22-01-2024

### PCR 1: Samples 31-33
- Included negative control (MM + H20)

1. Used master mix of PCR1 reagents made 2024-01-16 

2. Add following components to tubes

|Component       |Volume|
|:--------------:|:----:|
| PCR1 MasterMix | 40 $\mu$L | 
| cDNA      | 10 $\mu$L |

3. Performed PCR using the following parameters: 

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 60C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 1 min |
Hold Step |Step 6: 12C    | Hold |  

\center
 __10-25 cycles of steps 2-4__
\raggedright

### Bead Clean Up
#### Overview
- Clean PCR reaction by removing leftover primers
- This prevents false reads, cleaner indexing in later steps, and higher quality samples

#### Materials
- For 40 samples

|Component       |Expected Volume/experiment| Material ID |
|:--------------:|:----:|:---------:|
| Ampure Bead Kit | 200 $\mu$L | |

#### Protocol
1.	Equilibrate Ampure beads to room temp (available from supplies cold room)
2.	Mix well and vortex for 30sec to ensure uniform distribution.
3.	Add 90ul (1.8X volume) to PCR reaction and mix thoroughly by pipetting.
4.	Incubate 5min, room temp
5.	Place on magnet for 2 min and keep on magnet until final elution
6.	Remove supernatant and add 200ul 70% ethanol (make fresh)
7.	Incubate 30sec and remove
8.	Repeat 70% ethanol wash
9.	Air dry beads 5-10min room temp. Once dry the beads will go from shiny to matt in appearance. Avoid over drying (cracked appearance).
10.	Add 15 ul EB buffer and resuspend beads by pipetting
11.	Incubate 2min room temp
12.	Place on magnet and remove eluate
13. Assess PCR product size, contamination and concentration on tapestation.   
**Expected size = 165-172bp**


#### Tapestation PCR 1 31-33 
#### Tapestation PCR 1
![2024-01-22 Tapestation PCR 1 Samples 30-33 and Control](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-22 TS 1.png)
\center
**!!!! Band present in - Control lane !!!!**
\raggedright

### PCR 1: Negative Control
- Remade negative control (MM + H20)

1. Used master mix of PCR1 reagents made 2024-01-16 

2. Add following components to tubes

|Component       | Volume |
|:--------------:|:----:|
| PCR1 MasterMix | 40 $\mu$L | 
| cDNA      | 10 $\mu$L |

3. Performed PCR using the following parameters: 

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 60C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 1 min |
Hold Step |Step 6: 12C    | Hold |  

\center
 __10-25 cycles of steps 2-4__
\raggedright

#### Tapestation Control Test 1
- Compared New -Control to old 
  * Wanted to determine if this was a pipetting error
![2024-01-22 Tapestation Control Test 1](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-22 TS 1.png)


#### Tapestation Control Test 2
- Compared New -Control to H20 
  * Wanted to determine if water was contaminated
![2024-01-22 Tapestation Control Test 2](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-22 TS 1.png)


# Tuesday 23-01-2024
## Made New Media: AR-5
1. Made new media: AR-5

| Solution | ID code | Volume| % Total volume
| :-------: | :------: | :--------:|:----------:|
| Advanced RMPI   | ---- | 500 mL     | 78% |
| Glutamax     | ---- | 6.5 mL    | 20% |
| Pen-Strep    | ---- | 6.5 mL      | 1% |
| FBS     | ---- | 128 mL    | 1% |

## Cell Culture
### A20
- Thawed A20 and seeded in T25
- Spun down CS to remove DMSO, cell pellet small
- Cells didn't look super healthy
- Monitor tomorrow

### HEK
- Got T75 flask from Chris
- 70% confluent
- Added 10mL of trypsin and tapped flask until cells were dislodged
- Added 10mL of DMEM (20% FBS) and added to 50mL tube
- Added 25mL of DMEM to 5 T75 flasks
- Mixed CS to get single cell suspension and remvoe clumps
- Transferred 4mL of CS to each flask
- Will grow up for 1-2 days and freeze down 4 T75 flasks
  * Other flask will be used to gorw WILDseq library virus for A20's

## WS-ME1 Library Prep: Control Test 3
Overview: question around contaminated primers
- Ran PCR1 with new primer/PCR 1 mix

### PCR 1: Negative Control
- Remade negative control (MM + H20)
- Remade Primer mix and PCR1 MM

#### PCR1 Primer Mix
- WILDseq protocol from Kirsty dictates 8 fwd primers mixed in equal amounts with equivilant amount of rev priemr
- Primers were reconsituted at 100 $\mu$M so needed to be diluted to 10 $\mu$


|Component                 |   Volume  |
|:------------------------:|:---------:|
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 16 $\mu$L |
| DNAse/RNAse H20          | 320 $\mu$L| 


#### PCR1 MM

|Component       |Volume| Master Mix |
|:--------------|----| ------- |
| 10uM WS PCR1 Primer Mix | 3 $\mu$L |
| DNAse/RNAse H20      | 12 $\mu$L |
| Kapa Hifi HotStart Ready Mix (2X) | 25 $\mu$L |

1. Add following components to tubes  

|Component       |Volume|
|:--------------:|:----:|
| PCR1 MasterMix | 40 $\mu$L | 
| H2O     | 10 $\mu$L |

2. Performed PCR using the following parameters: 

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 60C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 1 min |
Hold Step |Step 6: 12C    | Hold |  

\center
 __10-25 cycles of steps 2-4__
\raggedright

#### Tapestation Control Test 3 
- Compared New -Control to old -control 
  * Wanted to determine if water was contaminated
![2024-01-22 Tapestation Control Test 2](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-23 TS 1.png)
  
  
# Wednesday 24-01-2024

## Cell Culture
### A20
- Checked cells ~10% confluent
- Cells looked healthy but sparse
- Grow up for inection
- Ask Chris about cell numbers needed for injecting into 12 mice

### HEK
- 20% confluent
- Check tomorrow and split or Friday

## WILDseq - Mouse Experiment 1: Library Prep Attempt 5 - RT
- Decided to repeat the entire library prep process using RNA extracted previously
- Everything done in PCR room RNA hood


### RT Protocol  
1. In PCR strip tubes, prepared 5 $\mu$g of RNA in a total volume of 10 $\mu$l of RNAse/DNAase-free water.
2. Add 1 $\mu$l of WS_RT_UMI_NexteraR2 primer (2$\mu$M)
- Diluted primer from stock (100$\mu$M)
- Added 1 $\mu$L RT-Primer stock into 49$\mu$L water
3. Add 1 $\mu$l dNTPs (10 mM each). 
4. Denature at 65 C for 5 mins in the PCR machine, then straight onto ice for at least 2 mins, spin briefly to get liquid to the bottom of the tube.
5. Prepare mastermix of RT enzyme and buffers. 1.1x the number of samples.

|Component       |Volume|   MM volume  |
|:--------------|----|-----------------|
| 5x SSIV Buffer | 4 $\mu$L | 134 $\mu$L      |
| SSIV RT        | 1 $\mu$L |    33.5 $\mu$L      |
| 100 mM DTT     | 1 $\mu$L |    33.5 $\mu$L      |
| RNAse Out      | 1 $\mu$L |     33.5 $\mu$L     |

6. Add 7 $\mu$l of RT MM prepared above to each sample and mix, spin briefly to get liquid to the bottom of the tube.
   - Spin briefly to get liquid to bottom of the tube 
7. In the PCR machine incubate at 53 C for 10 mins followed by 80 C for 10 mins.
8. Add 1 $\mu$l Themolabile Exonuclease I (NEB M0568) to remove excess RT primer.

|Component       |Volume|   MM volume  |
|:--------------|----|-----------------|
| Thermolabile Exonuclease I  | 1 $\mu$L |     33.5 $\mu$L     |
| NEBuffer r3.1*        | 2 $\mu$L |    67 $\mu$L      |

\center
 **Most PCR buffers are compatible**
\raggedright

9. In PCR Machine: heat at 37C for 4 mins followed by 80C for 1 min  
    - Spin briefly to get liquid to bottom of the tube
10. Add 1 $\mu$l of RNAse H and incubate at 37 C for 20 mins.
    - Stored at -4C

#### Plan for tomorrow
- PCR 1 (check for KAPA HIFI amount)
  * Bead Clean Up
  * Tape Station
- Check on cells
- Plan cell numbers needed for A20 mouse injections

# Thursday 25-01-2024

## Cell Culture
### A20
- Checked cells ~50% confluent
- Cells looked healthy but sparse
- Added 10ml of AR-5
- Grow up for injection
- Ask Chris about cell numbers needed for injecting into 12 mice

### HEK
- Collected all T175 and froze down
- Left one flask going at 20% confluency

## WS-ME1 Library Prep - Attempt 5 PCR 1
Ran PCR1 for attempt 5 

### PCR 1: Negative Control
- Remade negative control (MM + H20)
- Remade Primer mix and PCR1 MM

#### PCR1 Primer Mix
- WILDseq protocol from Kirsty dictates 8 fwd primers mixed in equal amounts with equivalent amount of rev primer
- Primers were reconsituted at 100 $\mu$M so needed to be diluted to 10 $\mu$M


|Component                 |   Volume  |
|:------------------------:|:---------:|
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 2 $\mu$L  |
| 100uM WS PCR1 Primer Mix | 16 $\mu$L |
| DNAse/RNAse H20          | 144 $\mu$L| 


#### PCR1 MM

|Component       |Volume| Master Mix |
|:--------------|----| ------- |
| 10uM WS PCR1 Primer Mix | 3 $\mu$L |
| DNAse/RNAse H20      | 12 $\mu$L |
| Q5 Master Mix | 25 $\mu$L |

- Used Q5 instead of KAPA

1. Add following components to tubes  

|Component       |Volume|
|:--------------:|:----:|
| PCR1 MasterMix | 40 $\mu$L | 
| H2O     | 10 $\mu$L |

2. Performed PCR using the following parameters: 

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 60C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 1 min |
Hold Step |Step 6: 12C    | Hold |  

\center
 __10-25 cycles of steps 2-4__
\raggedright

### Bead Clean Up
#### Overview
- Clean PCR reaction by removing leftover primers
- This prevents false reads, cleaner indexing in later steps, and higher quality samples

#### Materials
- For 40 samples

|Component       |Expected Volume/experiment| Material ID |
|:--------------:|:----:|:---------:|
| Ampure Bead Kit | 200 $\mu$L | |

#### Protocol
1.	Equilibrate Ampure beads to room temp (available from supplies cold room)
2.	Mix well and vortex for 30sec to ensure uniform distribution.
3.	Add 90ul (1.8X volume) to PCR reaction and mix thoroughly by pipetting.
4.	Incubate 5min, room temp
5.	Place on magnet for 2 min and keep on magnet until final elution
6.	Remove supernatant and add 200ul 70% ethanol (make fresh)
7.	Incubate 30sec and remove
8.	Repeat 70% ethanol wash
9.	Air dry beads 5-10min room temp. Once dry the beads will go from shiny to matt in appearance. Avoid over drying (cracked appearance).
10.	Add 15 ul EB buffer and resuspend beads by pipetting
11.	Incubate 2min room temp
12.	Place on magnet and remove eluate
13. Assess PCR product size, contamination and concentration on tapestation.   
**Expected size = 165-172bp**

### Tapestation PCR 1 Samples 1-33

#### Tapestation PCR 1 1-16 
##### Tapestation PCR 1
![2024-01-25 Tapestation PCR 1 Samples 1-16](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-25 TS 1.png)

#### Tapestation PCR 1 1-32 
##### Tapestation PCR 1
![2024-01-25 Tapestation PCR 1 Samples 17-32](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-25 TS 2.png)

- Samples 18, 19, and 20 DNW

#### Tapestation PCR 1 18-20, 33 
##### Tapestation PCR 1
![2024-01-25 Tapestation PCR 1 Samples 33 and redo 18-20](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-25 TS 3.png)

- Samples 18, 19, and 20 DNW again


# Friday 26-01-2024

## Cell Culture
### A20
- Expanded to T175
- Grow up for injection
- Ask Chris about cell numbers needed for injecting into 12 mice

### HEK
- Split 1/2
- DMEM

## WS-ME1 Library Prep - Attempt 5 PCR 1 repeat and PCR 2 prep
- Reran PCR1 for samples 18,19,20
- Prepped PCR2 primers

### PCR2 Prep
- Reconstituted PCR2 primers bought by Chris
  * Used DNAse/RNAse Free H20 and diluted to 100 $\mu$M
- Diluted into strip tubes working stock (10$\mu$M)
  * 90 $\mu$L H20 + 10 $\mu$L primer stock
  
### PCR1 Samples 18-20

#### PCR1 Protocol

- Used Primer Mix previously made on 25-01-2024

|Component       |Volume| Master Mix |
|:--------------|----| ------- |
| 10uM WS PCR1 Primer Mix | 3 $\mu$L |
| DNAse/RNAse H20      | 12 $\mu$L |
| Q5 Master Mix | 25 $\mu$L |

- Used Q5 instead of KAPA

1. Add following components to tubes  

|Component       |Volume|
|:--------------:|:----:|
| PCR1 MasterMix | 40 $\mu$L | 
| H2O     | 10 $\mu$L |

2. Performed PCR using the following parameters: 

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 60C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 1 min |
Hold Step |Step 6: 12C    | Hold |  

\center
 __10-25 cycles of steps 2-4__
\raggedright

### Bead Clean Up
#### Overview
- Clean PCR reaction by removing leftover primers
- This prevents false reads, cleaner indexing in later steps, and higher quality samples

#### Materials
- For 40 samples

|Component       |Expected Volume/experiment| Material ID |
|:--------------:|:----:|:---------:|
| Ampure Bead Kit | 200 $\mu$L | |

#### Protocol
1.	Equilibrate Ampure beads to room temp (available from supplies cold room)
2.	Mix well and vortex for 30sec to ensure uniform distribution.
3.	Add 90ul (1.8X volume) to PCR reaction and mix thoroughly by pipetting.
4.	Incubate 5min, room temp
5.	Place on magnet for 2 min and keep on magnet until final elution
6.	Remove supernatant and add 200ul 70% ethanol (make fresh)
7.	Incubate 30sec and remove
8.	Repeat 70% ethanol wash
9.	Air dry beads 5-10min room temp. Once dry the beads will go from shiny to matt in appearance. Avoid over drying (cracked appearance).
10.	Add 15 ul EB buffer and resuspend beads by pipetting
11.	Incubate 2min room temp
12.	Place on magnet and remove eluate
13. Assess PCR product size, contamination and concentration on tapestation.   
**Expected size = 165-172bp**

### Tapestation PCR 1 Samples 18-20

#### Tapestation PCR 1 18-20
##### Tapestation PCR 1
![2024-01-26 Tapestation PCR 1 Samples 18-20](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-26 TS 1.png)


# Monday 29-01-2024

## Cell Culture
### A20
- Split 1:6
- AR-5
- Ask Chris about cell numbers needed for injecting into 12 mice

### HEK
- Split 1/6
- DMEM

## PCR Step 2 - Samples 1-24
### Overview
- PCR to index samples for sequencing by attaching UMI 
- Didn't have enough Q5 to complete all the samples

### PCR2 Protocol  
1. Prepare samples according to the following table:
  - Keep careful note of which adaptors are added to which sample

|Component       |Volume|
|:--------------|----|
| 10uM Nxxx Nextera i7 adapter  | 1.5 $\mu$L |
|10uM Sxxx Nextera i5 adapter | 1.5 $\mu$L |
| DNAse/RNAse H20      | 21 $\mu$L |
| Q5 | 25 $\mu$L |
| 10ng/$\mu$L PCR1  | 1 $\mu$L | 


2. Perform PCR using the following parameters:  

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 55C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 2 min |
Hold Step |Step 6: 12C    | Hold |

\center
__8 cycles of steps 2-4__
\raggedright

3. Perform Ampure bead clean up as above with 90ul beads per 50ul PCR reaction.
- If you don’t have a crazy number of samples you might also want to consider doing up to 4 PCR2 reactions per sample and pooling them afterwards just to make sure all your sequences are captured.

4. Check size and concentration on tapestation. Dilute and pool samples for sequencing.

### Tapestation 
![2024-01-29 Tapestation PCR 2 Samples 1-16](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-29 TS 1.png)  
- 1-8 PCR DNW (suspected issue with bead clean-up)

![2024-01-29 Tapestation PCR 2 Samples 17-24](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-01-29 TS 2.png)  -

- Last 4 samples are from Chris (seperate experiment)

# Tuesday 30-01-2024

## Cell Culture
### HEK 
- Unhealthy
- Discarded

### A20
- Split 1/6

# \textbf February 2024

# Thursday 01-02-2024

## PCR Step 2 - Samples 1-8, 25-39  
- Repeated PCR2 for samples 1-8

### Overview
- PCR to index samples for sequencing by attaching UMI 
- Didn't have enough Q5 to complete all the samples

### PCR2 Protocol  
1. Prepare samples according to the following table:
  - Keep careful note of which adaptors are added to which sample

|Component       |Volume|
|:--------------|----|
| 10uM Nxxx Nextera i7 adapter  | 1.5 $\mu$L |
|10uM Sxxx Nextera i5 adapter | 1.5 $\mu$L |
| DNAse/RNAse H20      | 21 $\mu$L |
| Q5 | 25 $\mu$L |
| 10ng/$\mu$L PCR1  | 1 $\mu$L | 


2. Perform PCR using the following parameters:  

|Step Name |Steps |Time|
|:----:|:--------------:|:----:|
|Initial Denaturation |Step 1: 95C    | 3mins |
|Denaturation | **Step 2: 98C** | **20s**|
|Annealing | **Step 3: 55C** | **15s**| 
|Extension| **Step 4: 72C** | **15s**|
|Final Extension|Step 5: 72C    | 2 min |
Hold Step |Step 6: 12C    | Hold |  
\center
 __8 cycles of steps 2-4__ 
\raggedright

3. Perform Ampure bead clean up as above with 90ul beads per 50ul PCR reaction.
- If you don’t have a crazy number of samples you might also want to consider doing up to 4 PCR2 reactions per sample and pooling them afterwards just to make sure all your sequences are captured.

4. Check size and concentration on tapestation. Dilute and pool samples for sequencing.


### Tapestation
![2024-02-01 Tapestation PCR 2 Samples 1-8,25-32](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-02-01 TS 1.png)
![2024-02-01 Tapestation PCR 2 Samples 33-39](/Users/callum/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Wildseq - Mouse Experiment 1/Tapestation/Screenshot 2024-02-01 TS 2.png) 


# Monday 05-02-2024

## Cell Culture
### Made DMEM
| Solution | ID code | Volume| % Total volume
| :-------: | :------: | :--------:|:----------:|
| DMEM   | ---- | 500 mL     | 78% |
| Pen-Strep    | ---- | 5.5 mL      | 1% |
| FBS     | ---- | 55 mL    | 10% |

### HEK - Seeded
- Seeded New HEK
  * HEK stock are in CM Box 1 -80  
  
1. Thawed vial
2. Added 1mL DMEM to vial
3. Transferred to 15mL falcon tube
4. Slowly added 5mL DMEM
5. Spun down at 650rpm for 4min
6. Removed supernant and resuspended vial flicking
7. Added 1mL DMEM and pipetted multiple times to create single cell suspension
8. Added to T25 flask
9. Added 14mL of DMEM
10. Placed in incubator

### A20 - Split
- Split 1/6
- Spliut T25 vial for Emily James (EJ)

# Tuesday 06-02-2024

## WS-ME1 Pooling for Submission

- Genomics core indicated that library should be submitted at 5nM-10nM in a volume of 20-30uL
- Cubit PCR2 Samples to determine concentration

```{r, echo=FALSE, fig.align='center'}
knitr::opts_chunk$set(echo = TRUE)
library(readxl)

excel_data <- read_excel("~/Library/CloudStorage/OneDrive-UniversityofCambridge/Turner Lab/Data/Wildseq - Mouse Experiment 1/In_vivo_lib_prep_data.xlsx", sheet = "Pooling")

# Display the data as a table
knitr::kable(excel_data)

```

- Used IDT Library Concentration Conversion Calculator
  * Link: https://eu.idtdna.com/Calc/library-concentration-conversion
  

# Wednesday 07-02-2024

## Cell Culture

### HEK
- Decided to wait till Monday to start making virus
- Discarded HEK
- Will reseed HEK Sunday

### A20
- Split 1/6
- Keeping going for NSG/BALBc injection
  * Issue with procedure room in AMB, waiting for that to be resolved
- Also plan to incorporate WILDseq

# Friday 09-02-2024

## Cell Culture

### A20
- Split 1/6
- Keeping going for NSG/BALBc injection
  * Issue with procedure room in AMB, waiting for that to be resolved
- Also plan to incorporate WILDseq

# Monday 12-02-2024

## Cell Culture

### Made DMEM-V1

- DMEM for virus prodution
- Does not have Pen/Strep

| Solution | ID code | Volume| % Total volume
| :-------: | :------: | :--------:|:----------:|
| DMEM   | ---- | 500 mL     | 78% |
| FBS     | ---- | 128 mL    | 20% |
| Glutamax    | ---- | 6.5 mL    | 1% |

### Thawed HEK

### A20
- Split 1/6
- Keeping going for NSG/BALBc injection
  * Planning to inject with Chris tomorrow
- Also plan to incorporate WILDseq once virus is made

## Processing scRNAseq

# Tuesday 13-02-2024

## Cell Culture

### HEK
- Morphology looked weird
- 20% FBS is probably too high
- Threw away will start next week

## Processing scRNAseq

# Wednesday 14-02-2024

## Cell Culture

### A20
- Split 1/6
- Keeping going for NSG/BALBc injection
  * Planning to inject with Chris tomorrow
- Also plan to incorporate WILDseq once virus is made

## Processing scRNAseq

# Friday 16-02-2024

## A20 Xenograft EXP 1

### Cell Prep

  1. Thawed 2.5mL of Matrigel on ice ~3hours before 
  2. Transferred 12x10^6^ cells to a 15mL eppendorf
  3. Pelleted CS
  4. Resuspended in 2.5mL of PBS
  5. Transferred 400$\mu$L CS to 6 different 1mL eppendorfs
  6. Added 400$\mu$L of Matrigel, mixed gentlely and placed back on ice
  7. Brought materials over to the AMB for injection

### Injection  

- Chris Injected 
- Study Plan: SP140164  

| Mouse ID | Earmark | Genotype | Injection Location | 
| :-------: | :------: | :--------:|:----------:|
| TUAD36.2h  | NM | NSG     | IP |
| TUAD36.2i  | 1R | NSG     | IP |
| TUAD36.2k  | 2R | NSG     | IP |
| TUAD36.2a  | NM | NSG     | SC |
| TUAD36.2b  | 1L | NSG     | SC |
| TUAD36.2c  | 1R | NSG     | SC |
| TUAP3.1e  | 1L | BALB/c   | IP |
| TUAP3.1a  | NM | BALB/c   | IP |
| TUAP4.1a  | 2L | BALB/c   | IP |
| TUAP3.1b  | 1L | BALB/c   | SC |
| TUAP4.1b  | 2R | BALB/c   | SC |
| TUAP3.1c  | 1R | BALB/c   | SC |  

- Each mouse received ~1x10^6^ cells
- Mice will be monitored over the coming weeks for tumour development 

# Monday 19-02-2024

## Cell Culture
### A20
- Split 1/6
- Used AR-5

# Wednesday 21-02-2024

## Cell Culture
### A20
- Split 1/6
- Used AR-5

### HEK
- Seeded New HEK
  * HEK stock are in CM Box 1 -80 
- Used DMEM-V1

1. Thawed vial
2. Added 1mL DMEM to vial
3. Transferred to 15mL falcon tube
4. Slowly added 5mL DMEM
5. Spun down at 650rpm for 4min
6. Removed supernant and resuspended vial flicking
7. Added 1mL DMEM and pipetted multiple times to create single cell suspension
8. Added to T25 flask
9. Added 14mL of DMEM
10. Placed in incubator

# Friday 23-02-2024

## Cell Culture
### A20
- Split 1/6  
- Used AR-5

### HEK
- Split 1/6
- Used DMEM-V1

# Monday 26-02-2024

## Cell Culture

### A20
- Split into T25
- Froze down 4 vials
- 4x10^6^ cells per vial
- Used AR-5

### HEK
- Froze down 7 vials
- 6x10^6^ cells per vial

### WILDseq Virus Production Day 1
- Seeded 2 10cm dishes of HEK cells
  * Coated with Poly-D-Lysine (covered plate, removed liquid, let sit for 10 min, washed with PBS) 
  * 4.5x10^6^ cells seeded per dish (seeded at 13:00)

# Wednesday 06-03-2024
- Collected SC tumours
- HEK infection
  * binned A20/HEK

# Wednesday 13-03-2024
- Made media
- Seededin N2-BC4
- Froze down A20
- Collected 3 IP NSG tumours

# Thursday 14-03-2024
- Changed media (N2-BC4)

# Friday 15-03-2024

## Cell Culture

### N2-BC4 RTX CDC Assay - version 1

# Monday 18-03-2024

## Cell Culture - N2 -C4
- Split: seeded 5x10^6^ cells

## N2-BC4 RTX CDC Assay - version 2
Overview: Trying to optimise RTX CDC assay
- Using 0% serum, since there are no complement factors in the media the cell live/dead reading should not change
- Next step: add varying amounts of human serum

# Friday 22-03-2024

## Cell Culture

### N2-BC4
- Cells look healthy
- Feeders in flask are also healthly
- Split 5x10^6^ back into T175 with 30mL of AR-6

## EC50 RTX N2-BC4 22324 - Seeding
- Seeded 96-well plate with N2-BC4 and treated with RTX concentration range with or with out 10% Human Serum (HS)
- Plate seeding protocol:
  1. Diluted cell suspension to seed 20000 cells/well in 50$\mu$L amounts
  
| Well Number | Cells total | Volume total | CS cells/mL | Flask cell count | **Flask Volume** | **Media Volume** |
| :-------: | :------: | :--------:|:----------:|:---------:|:----:|:-------:|
| 60 wells | 1.2x10^6^ cells | 3 mL     | 4x10^5^ |3.06x10^6^ | **400$\mu$L** | **2.6 mL** |

  2. Made RTX dilutions and added to respective wells in 50$\mu$L  
    - [RTX stock] = 10.3 mg/mL 
    - Drug volumes are added in triplicate
    - Drug volumes are being added consititute 1/4 of well volume: [RTX working] needs to be 4x [RTX well]  
    - 6 wells per condition, 50$\mu$L per well ~ minimum of 300$\mu$L per condition needed (recommend 500$\mu$L)  
 
  |Dilution ID| [RTX Well] | Media Volume | Source |  [RTX WS] |
  | :-------: | :------: |:-------: | :------: | :------: |
  | RTX 1 | 100 $\mu$g/mL | 1 mL | 50$\mu$L stock | 400 $\mu$g/mL |
  | RTX 2 | 50 $\mu$g/mL | 500 mL | 500$\mu$L RTX 1 | 200 $\mu$g/mL |
  | RTX 3 | 25 $\mu$g/mL | 500 mL | 500$\mu$L RTX 2 | 100 $\mu$g/mL |
  | RTX 4 | 12.5 $\mu$g/mL | 500 mL | 500$\mu$L RTX 3 | 50 $\mu$g/mL |
  | RTX 5 | 6.25 $\mu$g/mL | 500 mL | 500$\mu$L RTX 4 | 25 $\mu$g/mL |
  | RTX 6 | 3.125 $\mu$g/mL | 500 mL | 500$\mu$L RTX 5 | 12.5 $\mu$g/mL |
  | RTX 7 | 1.56 $\mu$g/mL | 500 mL | 500$\mu$L RTX 6 | 6.25 $\mu$g/mL |
  | RTX 8 | .78 $\mu$g/mL | 500 mL | 500$\mu$L RTX 7 | 3.125 $\mu$g/mL |
  | RTX 9 | .39 $\mu$g/mL | 500 mL | 500$\mu$L RTX 8 | 1.56 $\mu$g/mL |
  | CNTR | 0 $\mu$g/mL | 500 mL | - | 0 $\mu$g/mL |
  
  3. Added 10% serum or media control
  - This provides complement factors to bind BL-bound RTX and initiate CDC
  - Serum/media amounts added at 100 $\mu$L/well
  - Serum volume is added in 1:4 ratio (20$\mu$L serum in 200$\mu$L final well volume)
  - Serum stock mix is 1.2mL HS : 2.4 mL media / plate
  
  4. Plate is incubated for 72 hrs at 37C
```{r, echo=FALSE, message=FALSE, warning=FALSE}
library(tidyverse)
library(ggforce)
library(scales)
library(prism)

pbs <- "PBS"
C1 <- "0% Serum"
C2 <- "10% Serum"
EC50_ID <- "EC50_RTX_N2-BC4_22324"

dil_ID = matrix(c(pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,
                  pbs, "CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5", "RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5", "RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs), ncol = 12, nrow = 8, byrow = TRUE)

drug_list = matrix(c(pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,
                     pbs, C1,C1,C1,C1,C1,C1,C1,C1,C1,C1,pbs,
                     pbs, C1,C1,C1,C1,C1,C1,C1,C1,C1,C1,pbs,
                     pbs,C1,C1,C1,C1,C1,C1,C1,C1,C1,C1,pbs,
                     pbs, C2,C2,C2,C2,C2,C2,C2,C2,C2,C2,pbs,
                     pbs,C2,C2,C2,C2,C2,C2,C2,C2,C2,C2,pbs,
                     pbs,C2,C2,C2,C2,C2,C2,C2,C2,C2,C2,pbs,
                     pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs), ncol = 12, nrow = 8, byrow = TRUE)



df1 <- as.data.frame(drug_list)
df2 <- as.data.frame(dil_ID)
df1 <- df1 %>% 
  mutate(row = 1:8) %>% 
  pivot_longer(-row, names_to = "col", values_to = "value") %>%
  mutate(col = as.integer(str_remove(col, "V"))) %>%
  mutate(well = paste0(LETTERS[row], col))

df2 <- df2 %>% 
  mutate(row = 1:8) %>% 
  pivot_longer(-row, names_to = "col", values_to = "value") %>%
  mutate(col = as.integer(str_remove(col, "V"))) %>%
  mutate(well = paste0(LETTERS[row], col))

#Plot
p <- ggplot(data = df1) + 
  geom_circle(aes(x0 = col, y0 = row, r = 0.45, fill = value))
p <- p + coord_equal()
p <- p + 
  scale_x_continuous(breaks = 1:12, expand = expansion(mult = c(0.01, 0.01))) +
  scale_y_continuous(breaks = 1:8, labels = LETTERS[1:8], expand = expansion(mult = c(0.01, 0.01)), trans = reverse_trans())
p <- p + geom_text(data = df2, aes(x = col, y = row, label = paste0(value)), size = 2,)
p <- p +
  labs(title = "96 Well Plate Map", subtitle = EC50_ID, x = "Col", y = "Row") + 
  #theme_prism("waves") + 
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank())
print(p, max.levels = NULL, width = 2)
```

# Monday 25-03-2024

## EC50 RTX N2-BC4 22324 - Collection
- Collected plate seeded on 22-03-2025
- EC Plate collection protocol:
  1. Added 40$\mu$L Cell Titre Blue (CTB) to each conditioned well 
    * 20$\mu$L CTB/100$\mu$L of conditioned well recommended by manufacturer
  2. Incubated for 1hr at 37C
  3. Read on plate reader according to Cell Titre Blue Protocol
- Results:

## EC50 RTX N2-BC4 25324 - Seeding (DNW)
- Seeded an EC50 experiement comparing the effects of RTX on N2-BC4 with or without 10% serum 
- Decreased overall well volume to reduce CTB being used
- Used RTX provided by Jamie
- Plate seeding protocol:
  1. Diluted cell suspension to seed 20000 cells/well in 25$\mu$L amounts
  
| Well Number | Cells total | Volume total | CS cells/mL | Flask cell count | **Flask Volume** | **Media Volume** |
| :-------: | :------: | :--------:|:----------:|:---------:|:----:|:-------:|
| 60 wells | 1.2x10^6^ cells | 1.5 mL     | 8x10^5^ | 2.92x10^6^ | **410$\mu$L** | **1.1 mL** |

  2. Made RTX dilutions and added to respective wells in 50$\mu$L  
    - [RTX stock] = 10.3 mg/mL 
    - Drug volumes are added in triplicate
    - Drug volumes are being added consititute 1/4 of well volume: [RTX working] needs to be 4x [RTX well]  
    - 6 wells per condition, 25$\mu$L per well ~ minimum of 150$\mu$L per condition needed (recommend 200$\mu$L)  
 
  |Dilution ID| [RTX Well] | Media Volume | Source |  [RTX WS] |
  | :-------: | :------: |:-------: | :------: | :------: |
  | RTX 1 | 100 $\mu$g/mL | 400$\mu$L | 15.5$\mu$L stock | 400 $\mu$g/mL |
  | RTX 2 | 50 $\mu$g/mL | 200 mL | 200$\mu$L RTX 1 | 200 $\mu$g/mL |
  | RTX 3 | 25 $\mu$g/mL | 200 mL | 200$\mu$L RTX 2 | 100 $\mu$g/mL |
  | RTX 4 | 12.5 $\mu$g/mL | 200 mL | 200$\mu$L RTX 3 | 50 $\mu$g/mL |
  | RTX 5 | 6.25 $\mu$g/mL | 200 mL | 200$\mu$L RTX 4 | 25 $\mu$g/mL |
  | RTX 6 | 3.125 $\mu$g/mL | 200 mL | 200$\mu$L RTX 5 | 12.5 $\mu$g/mL |
  | RTX 7 | 1.56 $\mu$g/mL | 200 mL | 200$\mu$L RTX 6 | 6.25 $\mu$g/mL |
  | RTX 8 | .78 $\mu$g/mL | 200 mL | 200$\mu$L RTX 7 | 3.125 $\mu$g/mL |
  | RTX 9 | .39 $\mu$g/mL | 200 mL | 200$\mu$L RTX 8 | 1.56 $\mu$g/mL |
  | CNTR | 0 $\mu$g/mL | 500 mL | - | 0 $\mu$g/mL |
  
  3. Added 10% serum or media control
  - This provides complement factors to bind BL-bound RTX and initiate CDC
  - Serum/media amounts added at 50 $\mu$L/well
  - Serum volume is added in 1:4 ratio (10$\mu$L serum in 100$\mu$L final well volume)
  - Serum stock mix is 600$\mu$L HS : 1.2 mL media / plate
  
  4. Plate is incubated for 72 hrs at 37C
```{r, echo=FALSE, message=FALSE, warning=FALSE}
library(tidyverse)
library(ggforce)
library(scales)
library(prism)

pbs <- "PBS"
C1 <- "0% Serum"
C2 <- "10% Serum"
EC50_ID <- "EC50_RTX_N2-BC4_25324"

dil_ID = matrix(c(pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,
                  pbs, "CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5", "RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5", "RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs), ncol = 12, nrow = 8, byrow = TRUE)

drug_list = matrix(c(pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,
                     pbs, C1,C1,C1,C1,C1,C1,C1,C1,C1,C1,pbs,
                     pbs, C1,C1,C1,C1,C1,C1,C1,C1,C1,C1,pbs,
                     pbs,C1,C1,C1,C1,C1,C1,C1,C1,C1,C1,pbs,
                     pbs, C2,C2,C2,C2,C2,C2,C2,C2,C2,C2,pbs,
                     pbs,C2,C2,C2,C2,C2,C2,C2,C2,C2,C2,pbs,
                     pbs,C2,C2,C2,C2,C2,C2,C2,C2,C2,C2,pbs,
                     pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs), ncol = 12, nrow = 8, byrow = TRUE)



df1 <- as.data.frame(drug_list)
df2 <- as.data.frame(dil_ID)
df1 <- df1 %>% 
  mutate(row = 1:8) %>% 
  pivot_longer(-row, names_to = "col", values_to = "value") %>%
  mutate(col = as.integer(str_remove(col, "V"))) %>%
  mutate(well = paste0(LETTERS[row], col))

df2 <- df2 %>% 
  mutate(row = 1:8) %>% 
  pivot_longer(-row, names_to = "col", values_to = "value") %>%
  mutate(col = as.integer(str_remove(col, "V"))) %>%
  mutate(well = paste0(LETTERS[row], col))

#Plot
p <- ggplot(data = df1) + 
  geom_circle(aes(x0 = col, y0 = row, r = 0.45, fill = value))
p <- p + coord_equal()
p <- p + 
  scale_x_continuous(breaks = 1:12, expand = expansion(mult = c(0.01, 0.01))) +
  scale_y_continuous(breaks = 1:8, labels = LETTERS[1:8], expand = expansion(mult = c(0.01, 0.01)), trans = reverse_trans())
p <- p + geom_text(data = df2, aes(x = col, y = row, label = paste0(value)), size = 2,)
p <- p +
  labs(title = "96 Well Plate Map", subtitle = EC50_ID, x = "Col", y = "Row") + 
  #theme_prism("waves") + 
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank())
print(p, max.levels = NULL, width = 2)
```

## Cell Culture
### N2-BC4
- Cells look healthy
- Feeders in flask are also healthly
- Split 5x10^6^ back into T175 with 30mL of AR-6

# Wednesday 27-03-2024

# Thursday 28-03-2024

# Friday 29-03-2024

## EC50 RTX N2-BC4/RAMOS-BC5 25324 - Seeding (DNW)
- Seeded an EC50 experiement comparing the effects of RTX on N2-BC4 and RAMOS-BC5 with or without 10% serum 
- Decreased overall well volume to reduce CTB being used
- Used RTX provided by Jamie
- Plate seeding protocol:
  1. Diluted cell suspension to seed 20000 cells/well in 25$\mu$L amounts
  
|Cell Line| Well Number | Cells total | Volume total | CS cells/mL | Flask cell count | **Flask Volume** | **Media Volume** |
| :-------: | :------: | :--------:|:----------:|:---------:|:----:|:-------:|
| N2-BC4| 60 wells | 1.2x10^6^ cells | 1.5 mL     | 8x10^5^ | 2.92x10^6^ | **410$\mu$L** | **1.1 mL** |
| RAMOS-BC5| 60 wells | 1.2x10^6^ cells | 1.5 mL     | 8x10^5^ | 2.92x10^6^ | **410$\mu$L** | **1.1 mL** |

  2. Made RTX dilutions and added to respective wells in 50$\mu$L  
    - [RTX stock] = 10.3 mg/mL 
    - Drug volumes are added in triplicate
    - Drug volumes are being added consititute 1/4 of well volume: [RTX working] needs to be 4x [RTX well]  
    - 6 wells per condition, 25$\mu$L per well ~ minimum of 150$\mu$L per condition needed (recommend 200$\mu$L)  
 
  |Dilution ID| [RTX Well] | Media Volume | Source |  [RTX WS] |
  | :-------: | :------: |:-------: | :------: | :------: |
  | RTX 1 | 100 $\mu$g/mL | 400$\mu$L | 15.5$\mu$L stock | 400 $\mu$g/mL |
  | RTX 2 | 50 $\mu$g/mL | 200 mL | 200$\mu$L RTX 1 | 200 $\mu$g/mL |
  | RTX 3 | 25 $\mu$g/mL | 200 mL | 200$\mu$L RTX 2 | 100 $\mu$g/mL |
  | RTX 4 | 12.5 $\mu$g/mL | 200 mL | 200$\mu$L RTX 3 | 50 $\mu$g/mL |
  | RTX 5 | 6.25 $\mu$g/mL | 200 mL | 200$\mu$L RTX 4 | 25 $\mu$g/mL |
  | RTX 6 | 3.125 $\mu$g/mL | 200 mL | 200$\mu$L RTX 5 | 12.5 $\mu$g/mL |
  | RTX 7 | 1.56 $\mu$g/mL | 200 mL | 200$\mu$L RTX 6 | 6.25 $\mu$g/mL |
  | RTX 8 | .78 $\mu$g/mL | 200 mL | 200$\mu$L RTX 7 | 3.125 $\mu$g/mL |
  | RTX 9 | .39 $\mu$g/mL | 200 mL | 200$\mu$L RTX 8 | 1.56 $\mu$g/mL |
  | CNTR | 0 $\mu$g/mL | 500 mL | - | 0 $\mu$g/mL |
  
  3. Added 10% serum or media control
  - This provides complement factors to bind BL-bound RTX and initiate CDC
  - Serum/media amounts added at 50 $\mu$L/well
  - Serum volume is added in 1:4 ratio (10$\mu$L serum in 100$\mu$L final well volume)
  - Serum stock mix is 600$\mu$L HS : 1.2 mL media / plate
  
  4. Plate is incubated for 72 hrs at 37C
```{r, echo=FALSE, message=FALSE, warning=FALSE}
library(tidyverse)
library(ggforce)
library(scales)
library(prism)

pbs <- "PBS"
C1 <- "0% Serum"
C2 <- "10% Serum"
EC50_ID <- "EC50_RTX_N2-BC4_25324"

dil_ID = matrix(c(pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,
                  pbs, "CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5", "RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5", "RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs), ncol = 12, nrow = 8, byrow = TRUE)

drug_list = matrix(c(pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,
                     pbs, C1,C1,C1,C1,C1,C1,C1,C1,C1,C1,pbs,
                     pbs, C1,C1,C1,C1,C1,C1,C1,C1,C1,C1,pbs,
                     pbs,C1,C1,C1,C1,C1,C1,C1,C1,C1,C1,pbs,
                     pbs, C2,C2,C2,C2,C2,C2,C2,C2,C2,C2,pbs,
                     pbs,C2,C2,C2,C2,C2,C2,C2,C2,C2,C2,pbs,
                     pbs,C2,C2,C2,C2,C2,C2,C2,C2,C2,C2,pbs,
                     pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs), ncol = 12, nrow = 8, byrow = TRUE)



df1 <- as.data.frame(drug_list)
df2 <- as.data.frame(dil_ID)
df1 <- df1 %>% 
  mutate(row = 1:8) %>% 
  pivot_longer(-row, names_to = "col", values_to = "value") %>%
  mutate(col = as.integer(str_remove(col, "V"))) %>%
  mutate(well = paste0(LETTERS[row], col))

df2 <- df2 %>% 
  mutate(row = 1:8) %>% 
  pivot_longer(-row, names_to = "col", values_to = "value") %>%
  mutate(col = as.integer(str_remove(col, "V"))) %>%
  mutate(well = paste0(LETTERS[row], col))

#Plot
p <- ggplot(data = df1) + 
  geom_circle(aes(x0 = col, y0 = row, r = 0.45, fill = value))
p <- p + coord_equal()
p <- p + 
  scale_x_continuous(breaks = 1:12, expand = expansion(mult = c(0.01, 0.01))) +
  scale_y_continuous(breaks = 1:8, labels = LETTERS[1:8], expand = expansion(mult = c(0.01, 0.01)), trans = reverse_trans())
p <- p + geom_text(data = df2, aes(x = col, y = row, label = paste0(value)), size = 2,)
p <- p +
  labs(title = "96 Well Plate Map", subtitle = EC50_ID, x = "Col", y = "Row") + 
  #theme_prism("waves") + 
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank())
print(p, max.levels = NULL, width = 2)
```

# Sunday 31-03-2024

## Cell Culture
### N2-BC4

- Healthy, split

- Changed FDC:

1. Thawed vial
2. Added 1mL DMEM to vial
3. Transferred to 15mL falcon tube
4. Slowly added 5mL DMEM
5. Spun down at 650rpm for 4min
6. Removed supernant and resuspended vial flicking
7. Added 1mL DMEM and pipetted multiple times to create single cell suspension
8. Added to T175 flask
9. Added 30mL of DMEM
10. Added 5x10^5^ N2 cells from previous flask
10. Placed in incubator

### Ramos-BC5
- Healthy, split
- Count: 2.18x10^6^ cells/mL
- Split: 7x10^5^ cells in 15 mL

## Ramos CDC Testing  

- Tested CDC assay adapted from [Ge et al., 2019](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6315025/#CR32)
- CDC protocol
1. Seed 250k cells 500uL volumes per well of a 12-well plate
2. Add RTX or media in 250uL volumes per well
  * [RTX] working needs to be 4x final desired well volume
3. Add Serum or media in 250uL volumes per well
  * Serum amount should be 4x final % desired
  
```{r, echo=FALSE, message=FALSE, warning=FALSE}
library(tidyverse)
library(ggforce)
library(scales)
library(prism)

pbs <- "PBS"
C1 <- "0% Serum"
C2 <- "10% Serum"
R1 <- "0ug/mL RTX"
R2 <- "5ug/mL RTX"
Plate_Name <- "CDC Testing"
Plate_ID <- "CDC_24331_Ramos"

dil_ID = matrix(c(R1,R2,R1,R2,
                  R1,R2,R1,R2,
                 R1,R2,R1,R2
                  ), ncol = 4, nrow = 3, byrow = TRUE)

drug_list = matrix(c(
  C1,C1,C2,C2,
  C1,C1,C2,C2,
  C1,C1,C2,C2
  ), ncol = 4, nrow = 3, byrow = TRUE)



df1 <- as.data.frame(drug_list)
df2 <- as.data.frame(dil_ID)
df1 <- df1 %>% 
  mutate(row = 1:3) %>% 
  pivot_longer(-row, names_to = "col", values_to = "value") %>%
  mutate(col = as.integer(str_remove(col, "V"))) %>%
  mutate(well = paste0(LETTERS[row], col))

df2 <- df2 %>% 
  mutate(row = 1:3) %>% 
  pivot_longer(-row, names_to = "col", values_to = "value") %>%
  mutate(col = as.integer(str_remove(col, "V"))) %>%
  mutate(well = paste0(LETTERS[row], col))

#Plot
p <- ggplot(data = df1) + 
  geom_circle(aes(x0 = col, y0 = row, r = 0.45, fill = value))
p <- p + coord_equal()
p <- p + 
  scale_x_continuous(breaks = 1:4, expand = expansion(mult = c(0.01, 0.01))) +
  scale_y_continuous(breaks = 1:3, labels = LETTERS[1:3], expand = expansion(mult = c(0.01, 0.01)), trans = reverse_trans())
p <- p + geom_text(data = df2, aes(x = col, y = row, label = paste0(value)), size = 2,)
p <- p +
  labs(title = Plate_ID, x = "Col", y = "Row") + 
  #theme_prism("waves") + 
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank())
print(p, max.levels = NULL, width = 2)
```

## EC50 RTX N2-BC4/RAMOS-BC5 24331 - Seeding (DNW)
- Seeded an EC50 experiement comparing the effects of RTX on N2-BC4 and RAMOS-BC5 with or without 10% serum 
- Decreased overall well volume to reduce CTB being used
- Used RTX provided by Jamie
- Only incubated for 24hr
- Plate seeding protocol:
  1. Diluted cell suspension to seed 20000 cells/well in 25$\mu$L amounts
  
|Cell Line| Well Number | Cells total | Volume total | CS cells/mL | Flask cell count | **Flask Volume** | **Media Volume** |
| :-------: | :------: | :--------:|:----------:|:---------:|:----:|:-------:|
| N2-BC4| 60 wells | 1.2x10^6^ cells | 1.5 mL     | 8x10^5^ | 2.92x10^6^ | **410$\mu$L** | **1.1 mL** |
| RAMOS-BC5| 60 wells | 1.2x10^6^ cells | 1.5 mL     | 8x10^5^ | 2.92x10^6^ | **410$\mu$L** | **1.1 mL** |

  2. Made RTX dilutions and added to respective wells in 50$\mu$L  
    - [RTX stock] = 10.3 mg/mL 
    - Drug volumes are added in triplicate
    - Drug volumes are being added consititute 1/4 of well volume: [RTX working] needs to be 4x [RTX well]  
    - 6 wells per condition, 25$\mu$L per well ~ minimum of 150$\mu$L per condition needed (recommend 200$\mu$L)  
 
  |Dilution ID| [RTX Well] | Media Volume | Source |  [RTX WS] |
  | :-------: | :------: |:-------: | :------: | :------: |
  | RTX 1 | 100 $\mu$g/mL | 400$\mu$L | 15.5$\mu$L stock | 400 $\mu$g/mL |
  | RTX 2 | 50 $\mu$g/mL | 200 mL | 200$\mu$L RTX 1 | 200 $\mu$g/mL |
  | RTX 3 | 25 $\mu$g/mL | 200 mL | 200$\mu$L RTX 2 | 100 $\mu$g/mL |
  | RTX 4 | 12.5 $\mu$g/mL | 200 mL | 200$\mu$L RTX 3 | 50 $\mu$g/mL |
  | RTX 5 | 6.25 $\mu$g/mL | 200 mL | 200$\mu$L RTX 4 | 25 $\mu$g/mL |
  | RTX 6 | 3.125 $\mu$g/mL | 200 mL | 200$\mu$L RTX 5 | 12.5 $\mu$g/mL |
  | RTX 7 | 1.56 $\mu$g/mL | 200 mL | 200$\mu$L RTX 6 | 6.25 $\mu$g/mL |
  | RTX 8 | .78 $\mu$g/mL | 200 mL | 200$\mu$L RTX 7 | 3.125 $\mu$g/mL |
  | RTX 9 | .39 $\mu$g/mL | 200 mL | 200$\mu$L RTX 8 | 1.56 $\mu$g/mL |
  | CNTR | 0 $\mu$g/mL | 500 mL | - | 0 $\mu$g/mL |
  
  3. Added 10% serum or media control
  - This provides complement factors to bind BL-bound RTX and initiate CDC
  - Serum/media amounts added at 50 $\mu$L/well
  - Serum volume is added in 1:4 ratio (10$\mu$L serum in 100$\mu$L final well volume)
  - Serum stock mix is 600$\mu$L HS : 1.2 mL media / plate
  
  4. Plate is incubated for 72 hrs at 37C
```{r, echo=FALSE, message=FALSE, warning=FALSE}
library(tidyverse)
library(ggforce)
library(scales)
library(prism)

pbs <- "PBS"
C1 <- "0% Serum"
C2 <- "10% Serum"
EC50_ID <- "EC50_RTX_N2-BC4_2431"

dil_ID = matrix(c(pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,
                  pbs, "CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5", "RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5", "RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs), ncol = 12, nrow = 8, byrow = TRUE)

drug_list = matrix(c(pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,
                     pbs, C1,C1,C1,C1,C1,C1,C1,C1,C1,C1,pbs,
                     pbs, C1,C1,C1,C1,C1,C1,C1,C1,C1,C1,pbs,
                     pbs,C1,C1,C1,C1,C1,C1,C1,C1,C1,C1,pbs,
                     pbs, C2,C2,C2,C2,C2,C2,C2,C2,C2,C2,pbs,
                     pbs,C2,C2,C2,C2,C2,C2,C2,C2,C2,C2,pbs,
                     pbs,C2,C2,C2,C2,C2,C2,C2,C2,C2,C2,pbs,
                     pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs), ncol = 12, nrow = 8, byrow = TRUE)



df1 <- as.data.frame(drug_list)
df2 <- as.data.frame(dil_ID)
df1 <- df1 %>% 
  mutate(row = 1:8) %>% 
  pivot_longer(-row, names_to = "col", values_to = "value") %>%
  mutate(col = as.integer(str_remove(col, "V"))) %>%
  mutate(well = paste0(LETTERS[row], col))

df2 <- df2 %>% 
  mutate(row = 1:8) %>% 
  pivot_longer(-row, names_to = "col", values_to = "value") %>%
  mutate(col = as.integer(str_remove(col, "V"))) %>%
  mutate(well = paste0(LETTERS[row], col))

#Plot
p <- ggplot(data = df1) + 
  geom_circle(aes(x0 = col, y0 = row, r = 0.45, fill = value))
p <- p + coord_equal()
p <- p + 
  scale_x_continuous(breaks = 1:12, expand = expansion(mult = c(0.01, 0.01))) +
  scale_y_continuous(breaks = 1:8, labels = LETTERS[1:8], expand = expansion(mult = c(0.01, 0.01)), trans = reverse_trans())
p <- p + geom_text(data = df2, aes(x = col, y = row, label = paste0(value)), size = 2,)
p <- p +
  labs(title = "96 Well Plate Map", subtitle = EC50_ID, x = "Col", y = "Row") + 
  #theme_prism("waves") + 
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank())
print(p, max.levels = NULL, width = 2)
```


# Monday 01-04-2024

## Cell Culture
### N2
- Not ready to split
- Added 5mL media

### Ramos
- Count: 1.6x10^6^ cells/mL
- Seeded: 5x10^5^ cells in 15mL

## EC50 Collection: 24329/24331
- Collected plates seeded on 24-03-2024 and 31-3-2024
- EC Plate collection protocol:
  1. Added 40$\mu$L Cell Titre Blue (CTB) to each conditioned well 
    * 20$\mu$L CTB/100$\mu$L of conditioned well recommended by manufacturer
  2. Incubated for 1hr at 37C
  3. Read on plate reader according to Cell Titre Blue Protocol

- Results:

## EC50 RTX N2-BC4/RAMOS-BC5 24401 - Seeding (DNW)
- Seeded an EC50 experiement comparing the effects of RTX on N2-BC4 and RAMOS-BC5 with or without 10% serum 
- Increased [RTX] to 1mg/mL (closer to Sorcha EC50)
- Used RTX provided by Jamie
- Plate seeding protocol:
  1. Diluted cell suspension to seed 20000 cells/well in 25$\mu$L amounts
  
|Cell Line| Well Number | Cells total | Volume total | CS cells/mL | Flask cell count | **Flask Volume** | **Media Volume** |
| :-------: | :------: | :--------:|:----------:|:---------:|:----:|:-------:|
| N2-BC4| 60 wells | 1.2x10^6^ cells | 1.5 mL     | 8x10^5^ | 2.92x10^6^ | **410$\mu$L** | **1.1 mL** |
| RAMOS-BC5| 60 wells | 1.2x10^6^ cells | 1.5 mL     | 8x10^5^ | 2.92x10^6^ | **410$\mu$L** | **1.1 mL** |

  2. Made RTX dilutions and added to respective wells in 50$\mu$L  
    - [RTX stock] = 10.3 mg/mL 
    - Drug volumes are added in triplicate
    - Drug volumes are being added consititute 1/4 of well volume: [RTX working] needs to be 4x [RTX well]  
    - 6 wells per condition, 25$\mu$L per well ~ minimum of 150$\mu$L per condition needed (recommend 200$\mu$L)  
 
  |Dilution ID| [RTX Well] | Media Volume | Source |  [RTX WS] |
  | :-------: | :------: |:-------: | :------: | :------: |
  | RTX 1 | 1 mg/mL | 400$\mu$L | 15.5$\mu$L stock | 4 mg/mL |
  | RTX 2 | 500 $\mu$g/mL | 200 mL | 200$\mu$L RTX 1 | 2 $\mu$g/mL |
  | RTX 3 | 250 $\mu$g/mL | 200 mL | 200$\mu$L RTX 2 | 1 $\mu$g/mL |
  | RTX 4 | 125 $\mu$g/mL | 200 mL | 200$\mu$L RTX 3 | 5 $\mu$g/mL |
  | RTX 5 | 62.5 $\mu$g/mL | 200 mL | 200$\mu$L RTX 4 | 2.5 $\mu$g/mL |
  | RTX 6 | 31.25 $\mu$g/mL | 200 mL | 200$\mu$L RTX 5 | 1.25 $\mu$g/mL |
  | RTX 7 | 15.6 $\mu$g/mL | 200 mL | 200$\mu$L RTX 6 | .625 $\mu$g/mL |
  | RTX 8 | 7.8 $\mu$g/mL | 200 mL | 200$\mu$L RTX 7 | .3125 $\mu$g/mL |
  | RTX 9 | 3.9 $\mu$g/mL | 200 mL | 200$\mu$L RTX 8 | .156 $\mu$g/mL |
  | CNTR | 0 $\mu$g/mL | 500 mL | - | 0 $\mu$g/mL |
  
  3. Added 10% serum or media control
  - This provides complement factors to bind BL-bound RTX and initiate CDC
  - Serum/media amounts added at 50 $\mu$L/well
  - Serum volume is added in 1:4 ratio (10$\mu$L serum in 100$\mu$L final well volume)
  - Serum stock mix is 600$\mu$L HS : 1.2 mL media / plate
  
  4. Plate is incubated for 72 hrs at 37C
```{r, echo=FALSE, message=FALSE, warning=FALSE}
library(tidyverse)
library(ggforce)
library(scales)
library(prism)

pbs <- "PBS"
C1 <- "0% Serum"
C2 <- "10% Serum"
EC50_ID <- "EC50_RTX_N2-BC4_2431"

dil_ID = matrix(c(pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,
                  pbs, "CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5", "RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5", "RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs), ncol = 12, nrow = 8, byrow = TRUE)

drug_list = matrix(c(pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,
                     pbs, C1,C1,C1,C1,C1,C1,C1,C1,C1,C1,pbs,
                     pbs, C1,C1,C1,C1,C1,C1,C1,C1,C1,C1,pbs,
                     pbs,C1,C1,C1,C1,C1,C1,C1,C1,C1,C1,pbs,
                     pbs, C2,C2,C2,C2,C2,C2,C2,C2,C2,C2,pbs,
                     pbs,C2,C2,C2,C2,C2,C2,C2,C2,C2,C2,pbs,
                     pbs,C2,C2,C2,C2,C2,C2,C2,C2,C2,C2,pbs,
                     pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs), ncol = 12, nrow = 8, byrow = TRUE)



df1 <- as.data.frame(drug_list)
df2 <- as.data.frame(dil_ID)
df1 <- df1 %>% 
  mutate(row = 1:8) %>% 
  pivot_longer(-row, names_to = "col", values_to = "value") %>%
  mutate(col = as.integer(str_remove(col, "V"))) %>%
  mutate(well = paste0(LETTERS[row], col))

df2 <- df2 %>% 
  mutate(row = 1:8) %>% 
  pivot_longer(-row, names_to = "col", values_to = "value") %>%
  mutate(col = as.integer(str_remove(col, "V"))) %>%
  mutate(well = paste0(LETTERS[row], col))

#Plot
p <- ggplot(data = df1) + 
  geom_circle(aes(x0 = col, y0 = row, r = 0.45, fill = value))
p <- p + coord_equal()
p <- p + 
  scale_x_continuous(breaks = 1:12, expand = expansion(mult = c(0.01, 0.01))) +
  scale_y_continuous(breaks = 1:8, labels = LETTERS[1:8], expand = expansion(mult = c(0.01, 0.01)), trans = reverse_trans())
p <- p + geom_text(data = df2, aes(x = col, y = row, label = paste0(value)), size = 2,)
p <- p +
  labs(title = "96 Well Plate Map", subtitle = EC50_ID, x = "Col", y = "Row") + 
  #theme_prism("waves") + 
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank())
print(p, max.levels = NULL, width = 2)
```

# Tuesday 02-04-2024

## N2/Ramos CD20 Flow Cytometry
- Got antibody from Chris for CD20
- Cat#: 562873

Cell Stain Protocol:
1. Resuspend cells to 1x10^6^ cells in 1mL
2. Distribute CS into 3 wells of a 96-well plate
3. Spin down @ 1500g for 2min
4. Flick media out
5. Make Cell Stain:
  - 500uL PBS + 2uL Stain
  -Make this while cells are spinning down
6. Resuspend w/ 100uL in the first well and then resuspend the next 2 wells in the same stain volume
  - Consolidate into 1 well/sample
7. incubate in the fridge for 20min
8. Spin down @ 1500g for 2 min
9. Transfer to FACS tube and run

- Results:

# Thursday 04-04-2024

## Cell Culture
### N2
- Count: 2.1x10^6^ cells/mL
- Seeded: 5x10^5^ cells in 15mL

### Ramos
- Count: 1.9x10^6^ cells/mL
- Seeded: 5x10^5^ cells in 15mL

## EC50 Collection: 24401
- Collected plate seeded on 01-04-2024
- EC Plate collection protocol:
  1. Added 40$\mu$L Cell Titre Blue (CTB) to each conditioned well 
    * 20$\mu$L CTB/100$\mu$L of conditioned well recommended by manufacturer
  2. Incubated for 1hr at 37C
  3. Read on plate reader according to Cell Titre Blue Protocol

- Results:DNW

# Monday 08-04-2024

## Cell Culture
### N2
- Count: 1.4x10^6^ cells/mL
- Seeded: 3x10^5^ cells in 15mL

### Ramos
- Count: 2.9x10^6^ cells/mL
- Seeded: 3x10^5^ cells in 15mL

# Tuesday 09-04-2024

## Perla Drug Randomizer

- Randomized drugs for Perla Pucci mouse experiment

| Drug | Identification |
| :-------: | :------: |
| Vehicle  | B | 
| Beta 30 | F | 
| Beta 10 | D | 
| Plo 30  | A |
| Plo 10  | E | 
| TESA 4 | C | 
| TESA0,4  | G |

# Saturday 14-04-2024

## Cell Culture
### N2-BC4
- Count: 6.7x10^5^ cells/mL
- Seeded: 3x10^5^ cells in 15mL

### Ramos-BC5
- Count: 6.0x10^5^ cells/mL
- Seeded: 3x10^5^ cells in 15mL

## EC50 RTX N2-BC4/RAMOS-BC5 24414 - Seeding (DNW)
- Seeded an EC50 experiement comparing the effects of RTX on N2-BC4 and RAMOS-BC5 with or without 10% serum 
- Increased [RTX] to 1mg/mL (closer to Sorcha EC50)
- Used RTX provided by Jamie
- Looked at cell-intrisic effects of RTX (no serum)
- Plate seeding protocol:
  1. Diluted cell suspension to seed 20000 cells/well in 25$\mu$L amounts
  
|Cell Line| Well Number | Cells total | Volume total | CS cells/mL | Flask cell count | **Flask Volume** | **Media Volume** |
| :-------: | :------: | :--------:|:----------:|:---------:|:----:|:-------:|
| N2-BC4| 60 wells | 1.2x10^6^ cells | 1.5 mL     | 8x10^5^ | 2.92x10^6^ | **410$\mu$L** | **1.1 mL** |
| RAMOS-BC5| 60 wells | 1.2x10^6^ cells | 1.5 mL     | 8x10^5^ | 2.92x10^6^ | **410$\mu$L** | **1.1 mL** |

  2. Made RTX dilutions and added to respective wells in 50$\mu$L  
    - [RTX stock] = 10.3 mg/mL 
    - Drug volumes are added in triplicate
    - Drug volumes are being added consititute 1/4 of well volume: [RTX working] needs to be 4x [RTX well]  
    - 6 wells per condition, 25$\mu$L per well ~ minimum of 150$\mu$L per condition needed (recommend 200$\mu$L)  
 
  |Dilution ID| [RTX Well] | Media Volume | Source |  [RTX WS] |
  | :-------: | :------: |:-------: | :------: | :------: |
  | RTX 1 | 1 mg/mL | 400$\mu$L | 15.5$\mu$L stock | 4 mg/mL |
  | RTX 2 | 500 $\mu$g/mL | 200 mL | 200$\mu$L RTX 1 | 2 $\mu$g/mL |
  | RTX 3 | 250 $\mu$g/mL | 200 mL | 200$\mu$L RTX 2 | 1 $\mu$g/mL |
  | RTX 4 | 125 $\mu$g/mL | 200 mL | 200$\mu$L RTX 3 | 5 $\mu$g/mL |
  | RTX 5 | 62.5 $\mu$g/mL | 200 mL | 200$\mu$L RTX 4 | 2.5 $\mu$g/mL |
  | RTX 6 | 31.25 $\mu$g/mL | 200 mL | 200$\mu$L RTX 5 | 1.25 $\mu$g/mL |
  | RTX 7 | 15.6 $\mu$g/mL | 200 mL | 200$\mu$L RTX 6 | .625 $\mu$g/mL |
  | RTX 8 | 7.8 $\mu$g/mL | 200 mL | 200$\mu$L RTX 7 | .3125 $\mu$g/mL |
  | RTX 9 | 3.9 $\mu$g/mL | 200 mL | 200$\mu$L RTX 8 | .156 $\mu$g/mL |
  | CNTR | 0 $\mu$g/mL | 500 mL | - | 0 $\mu$g/mL |
  
  3. Plate is incubated for 72 hrs at 37C

```{r, echo=FALSE, message=FALSE, warning=FALSE}
library(tidyverse)
library(ggforce)
library(scales)
library(prism)

pbs <- "PBS"
C1 <- "N2"
C2 <- "Ramos"
EC50_ID <- "EC50_RTX_N2-BC4_2431"

dil_ID = matrix(c(pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,
                  pbs, "CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5", "RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5","RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,"CNTR","RTX9","RTX8","RTX7","RTX6","RTX5", "RTX4","RTX3","RTX2","RTX1",pbs,
                  pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs), ncol = 12, nrow = 8, byrow = TRUE)

drug_list = matrix(c(pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,
                     pbs, C1,C1,C1,C1,C1,C1,C1,C1,C1,C1,pbs,
                     pbs, C1,C1,C1,C1,C1,C1,C1,C1,C1,C1,pbs,
                     pbs,C1,C1,C1,C1,C1,C1,C1,C1,C1,C1,pbs,
                     pbs, C2,C2,C2,C2,C2,C2,C2,C2,C2,C2,pbs,
                     pbs,C2,C2,C2,C2,C2,C2,C2,C2,C2,C2,pbs,
                     pbs,C2,C2,C2,C2,C2,C2,C2,C2,C2,C2,pbs,
                     pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs,pbs), ncol = 12, nrow = 8, byrow = TRUE)



df1 <- as.data.frame(drug_list)
df2 <- as.data.frame(dil_ID)
df1 <- df1 %>% 
  mutate(row = 1:8) %>% 
  pivot_longer(-row, names_to = "col", values_to = "value") %>%
  mutate(col = as.integer(str_remove(col, "V"))) %>%
  mutate(well = paste0(LETTERS[row], col))

df2 <- df2 %>% 
  mutate(row = 1:8) %>% 
  pivot_longer(-row, names_to = "col", values_to = "value") %>%
  mutate(col = as.integer(str_remove(col, "V"))) %>%
  mutate(well = paste0(LETTERS[row], col))

#Plot
p <- ggplot(data = df1) + 
  geom_circle(aes(x0 = col, y0 = row, r = 0.45, fill = value))
p <- p + coord_equal()
p <- p + 
  scale_x_continuous(breaks = 1:12, expand = expansion(mult = c(0.01, 0.01))) +
  scale_y_continuous(breaks = 1:8, labels = LETTERS[1:8], expand = expansion(mult = c(0.01, 0.01)), trans = reverse_trans())
p <- p + geom_text(data = df2, aes(x = col, y = row, label = paste0(value)), size = 2,)
p <- p +
  labs(title = "96 Well Plate Map", subtitle = EC50_ID, x = "Col", y = "Row") + 
  #theme_prism("waves") + 
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank())
print(p, max.levels = NULL, width = 2)
```
  
 
# Wednesday 17-04-2024 

## EC50 Collection: 24414
- Collected plate seeded on 14-04-2024
- EC Plate collection protocol:
  1. Added 40$\mu$L Cell Titre Blue (CTB) to each conditioned well 
    * 20$\mu$L CTB/100$\mu$L of conditioned well recommended by manufacturer
  2. Incubated for 1hr at 37C
  3. Read on plate reader according to Cell Titre Blue Protocol

- Results:DNW


# Monday 22-04-2024 

## Cell Culture
### N2-BC4
- Seeded: 3x10^5^ cells in 15mL

### Ramos-BC5
- Seeded: 3x10^5^ cells in 15mL

## Thawed NK-92 
- Obtained from Chris Steele
- Cultured in Advanced RPMI
- Added 200u/mL of IL-2 (#78036.1)
  * Obtained from Emily
  * Product listed as 4.1x10^4^ IU/ug
  * Stock diluted to 10ug/mL
  * Added .48 uL stock/mL of media
  
# Wednesday 24-04-2024 

## Cell Culture
### N2-BC4
- Seeded: 3x10^5^ cells in 15mL

### Ramos-BC5
- Seeded: 3x10^5^ cells in 15mL

### NK-92
- Looked very unhealthy
- Lots of dead cells/debris in the media
- Spun down, resuspended in 1mL of media
- Split into 4 wells of a 48 well plate with increasing amounts of IL-2

# Friday 26-04-2024 

## Cell Culture
### N2-BC4
- Seeded: 3x10^5^ cells in 15mL

### Ramos-BC5
- Seeded: 3x10^5^ cells in 15mL

### NK-92
- Cells still look unhealthy
- Tried to consolidate in single well of 24 well plate
- Spun down in eppendorf and resuspended in 1mL of media

# Monday 29-04-2024 

## Cell Culture
### N2-BC4
- Seeded: 3x10^5^ cells in 15mL

### Ramos-BC5
- Seeded: 3x10^5^ cells in 15mL








